From 95d8071624088152a9a3deaf916012651bddb5a1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:29 -0500
Subject: [PATCH 003/398] B::C patch for TEST

---
 t/CORE/TEST | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/TEST b/t/CORE/TEST
index 96eb6a4..5b27cb1 100755
--- a/t/CORE/TEST
+++ b/t/CORE/TEST
@@ -132,9 +132,7 @@ our $show_elapsed_time = $ENV{HARNESS_TIMER} || 0;
 }
 
 chdir 't' if -f 't/TEST';
-if (-f 'TEST' && -f 'harness' && -d '../lib') {
-    @INC = '../lib';
-}
+unshift @INC, 't/CORE/lib';
 
 die "You need to run \"make test\" first to set things up.\n"
   unless -e 'perl' or -e 'perl.exe' or -e 'perl.pm';
-- 
2.5.0

