From 5377d3aa4f2920ad99ab35291f7d7a15cfa3603f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:33 -0500
Subject: [PATCH 004/398] B::C patch for base/rs.t

---
 t/CORE/base/rs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/base/rs.t b/t/CORE/base/rs.t
index 416696e..57082c4 100644
--- a/t/CORE/base/rs.t
+++ b/t/CORE/base/rs.t
@@ -120,7 +120,7 @@ $/ = "\n";
  # If we do not include the lib directories, we may end up picking up a
  # binary-incompatible previously-installed version. The eval won’t help in
  # intercepting a SIGTRAP.
- local @INC = ("../lib", "lib", @INC);
+ local @INC = ("t/CORE/lib", @INC);
  if (not eval q/use PerlIO::scalar; use PerlIO::via::scalar; 1/) {
   # In-memory files necessitate PerlIO::via::scalar, thus a perl with
   # perlio and dynaloading enabled. miniperl won't be able to run this
-- 
2.5.0

