From 87a8b3dca1093a2ca070fab061403a15e42ea22e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:34 -0500
Subject: [PATCH 005/398] B::C patch for base/term.t

---
 t/CORE/base/term.t | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/t/CORE/base/term.t b/t/CORE/base/term.t
index 7a16ffd..53108ca 100644
--- a/t/CORE/base/term.t
+++ b/t/CORE/base/term.t
@@ -1,9 +1,5 @@
 #!./perl
 
-BEGIN {
-    chdir 't' if -d 't';
-}
-
 print "1..7\n";
 
 # check "" interpretation
@@ -48,5 +44,5 @@ else {
     die "/dev/null IS NOT A CHARACTER SPECIAL FILE!!!!\n" unless -c '/dev/null';
 }
 
-open(try, "harness") || (die "Can't open harness.");
+open( try, "t/CORE/TEST" ) || ( die "Can't open t/CORE/TEST." );
 if (<try> ne '') {print "ok 7\n";} else {print "not ok 7\n";}
-- 
2.5.0

