From 6a4d455aea2f955d587631ca91264dc2947e7bf5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:34 -0500
Subject: [PATCH 006/398] B::C patch for cmd/mod.t

---
 t/CORE/cmd/mod.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/cmd/mod.t b/t/CORE/cmd/mod.t
index 07617f5..70849d5 100644
--- a/t/CORE/cmd/mod.t
+++ b/t/CORE/cmd/mod.t
@@ -32,7 +32,7 @@ if (join(' ',@y) eq '0 2 4 6 8 10 12 14 16 18 20') {
 	print "not ok 7 @y\n";
 }
 
-open(foo,'./TEST') || open(foo,'TEST') || open(foo,'t/TEST');
+open(foo,'./t/CORE/TEST');
 $x = 0;
 $x++ while <foo>;
 print $x > 50 && $x < 1000 ? "ok 8\n" : "not ok 8\n";
-- 
2.5.0

