From 7fbbc7351fdb58a48ec73d9d0f6bd20a509bf07d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:34 -0500
Subject: [PATCH 007/398] B::C patch for comp/bproto.t

---
 t/CORE/comp/bproto.t | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/t/CORE/comp/bproto.t b/t/CORE/comp/bproto.t
index 8d11b91..b1a7c2e 100644
--- a/t/CORE/comp/bproto.t
+++ b/t/CORE/comp/bproto.t
@@ -3,11 +3,6 @@
 # check if builtins behave as prototyped
 #
 
-BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-}
-
 print "1..16\n";
 
 my $i = 1;
-- 
2.5.0

