From aed0a02e101b9aeda00e0086c99117fe6e5db547 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 008/398] B::C patch for comp/line_debug.t

---
 t/CORE/comp/line_debug.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/comp/line_debug.t b/t/CORE/comp/line_debug.t
index 8361194..1b9dd8a 100644
--- a/t/CORE/comp/line_debug.t
+++ b/t/CORE/comp/line_debug.t
@@ -1,6 +1,6 @@
 #!./perl
 
-chdir 't' if -d 't';
+chdir 't/CORE';
 
 sub ok {
     my($test,$ok) = @_;
-- 
2.5.0

