From 8b39bde9bdc71ecd16191e7f1b538075ba7fea19 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 009/398] B::C patch for comp/parser.t

---
 t/CORE/comp/parser.t | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/t/CORE/comp/parser.t b/t/CORE/comp/parser.t
index 584a473..e3513dd 100644
--- a/t/CORE/comp/parser.t
+++ b/t/CORE/comp/parser.t
@@ -3,10 +3,7 @@
 # Checks if the parser behaves correctly in edge cases
 # (including weird syntax errors)
 
-BEGIN {
-    @INC = qw(. ../lib);
-    chdir 't';
-}
+chdir 't/CORE';
 
 print "1..169\n";
 
-- 
2.5.0

