From ca55c4f5018cca754de8db54745f36dae85d105e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 010/398] B::C patch for comp/proto.t

---
 t/CORE/comp/proto.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/comp/proto.t b/t/CORE/comp/proto.t
index 47ebf74..abe8ea7 100644
--- a/t/CORE/comp/proto.t
+++ b/t/CORE/comp/proto.t
@@ -10,8 +10,7 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
 # We need this, as in places we're testing the interaction of prototypes with
-- 
2.5.0

