From 2056e171bdc3bdc0f99609e0ca07bbe56184b7aa Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 011/398] B::C patch for comp/require.t

---
 t/CORE/comp/require.t | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/t/CORE/comp/require.t b/t/CORE/comp/require.t
index 4eafce4..822e404 100644
--- a/t/CORE/comp/require.t
+++ b/t/CORE/comp/require.t
@@ -1,10 +1,7 @@
 #!./perl
 
-BEGIN {
-    chdir 't' if -d 't';
-    @INC = '.';
-    push @INC, '../lib';
-}
+chdir 't/CORE';
+unshift @INC, '.', 't/CORE/lib';
 
 sub do_require {
     %INC = ();
-- 
2.5.0

