From 98c739052291954f73bbbf58c75b935198ae47e4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:35 -0500
Subject: [PATCH 012/398] B::C patch for comp/use.t

---
 t/CORE/comp/use.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/comp/use.t b/t/CORE/comp/use.t
index 12409cf..e900abd 100644
--- a/t/CORE/comp/use.t
+++ b/t/CORE/comp/use.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib', 'lib');
+    unshift @INC, 't/CORE/lib';
     $INC{"feature.pm"} = 1; # so we don't attempt to load feature.pm
 }
 
-- 
2.5.0

