From f6f8026c168a33e805de77e99ae59866aa2c2515 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:36 -0500
Subject: [PATCH 013/398] B::C patch for io/argv.t

---
 t/CORE/io/argv.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/argv.t b/t/CORE/io/argv.t
index a1febaf..edde8e5 100644
--- a/t/CORE/io/argv.t
+++ b/t/CORE/io/argv.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
-BEGIN { require "./test.pl"; }
+BEGIN { require 'test.pl'; }
 
 plan(tests => 24);
 
-- 
2.5.0

