From ee3bbcf386f7f769afdb35915da912aa0b1dce12 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:36 -0500
Subject: [PATCH 014/398] B::C patch for io/binmode.t

---
 t/CORE/io/binmode.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/binmode.t b/t/CORE/io/binmode.t
index 473261e..bc808b7 100644
--- a/t/CORE/io/binmode.t
+++ b/t/CORE/io/binmode.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 'test.pl';
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

