From 13473b89ca3ebfca782ddfe033d72d9a8c0f7e0c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:36 -0500
Subject: [PATCH 015/398] B::C patch for io/bom.t

---
 t/CORE/io/bom.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/bom.t b/t/CORE/io/bom.t
index dd1be65..d1df5c2 100644
--- a/t/CORE/io/bom.t
+++ b/t/CORE/io/bom.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
-BEGIN { require "./test.pl"; }
+BEGIN { require 'test.pl'; }
 
 plan(tests => 1);
 
-- 
2.5.0

