From 4c4f70e611757e9d35fb86f796c79b8e4c2cf8b1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:36 -0500
Subject: [PATCH 016/398] B::C patch for io/closepid.t

---
 t/CORE/io/closepid.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/closepid.t b/t/CORE/io/closepid.t
index a90db68..7caf8f0 100644
--- a/t/CORE/io/closepid.t
+++ b/t/CORE/io/closepid.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
 }
 
 if ($^O eq 'dos') {
-- 
2.5.0

