From c0b7e6da6e9059fbe8787cc4b0cd57d8bf002d71 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:36 -0500
Subject: [PATCH 017/398] B::C patch for io/crlf.t

---
 t/CORE/io/crlf.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/io/crlf.t b/t/CORE/io/crlf.t
index f26ea0d..01457f5 100644
--- a/t/CORE/io/crlf.t
+++ b/t/CORE/io/crlf.t
@@ -1,8 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
     skip_all_without_perlio();
 }
-- 
2.5.0

