From bf978358ec4a6890682dfd86b4134e16f74baadd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 018/398] B::C patch for io/crlf_through.t

---
 t/CORE/io/crlf_through.t | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/t/CORE/io/crlf_through.t b/t/CORE/io/crlf_through.t
index c080995..caf6fad 100644
--- a/t/CORE/io/crlf_through.t
+++ b/t/CORE/io/crlf_through.t
@@ -1,10 +1,6 @@
 #!./perl
 
-BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-}
-
+chdir 't/CORE';
 no warnings 'once';
 $main::use_crlf = 1;
 do './io/through.t' or die "no kid script";
-- 
2.5.0

