From 8aedde826d65a81bfcf5e458e25c1bf6c4387d7a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 020/398] B::C patch for io/defout.t

---
 t/CORE/io/defout.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/defout.t b/t/CORE/io/defout.t
index 2cb7b9f..a8ae788 100644
--- a/t/CORE/io/defout.t
+++ b/t/CORE/io/defout.t
@@ -7,9 +7,8 @@
 # it probably needs expanding at some point to cover other stuff.
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    unshift @INC, 't/CORE', "t/CORE/lib";
+    require 'test.pl';
 }
 
 plan tests => 16;
-- 
2.5.0

