From 28b2b5d49c79877fe6d85eaeca877f1e491add4e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 021/398] B::C patch for io/dup.t

---
 t/CORE/io/dup.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/dup.t b/t/CORE/io/dup.t
index 1328aeb..62e057c 100644
--- a/t/CORE/io/dup.t
+++ b/t/CORE/io/dup.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "./test.pl";
+    push @INC, qw{t/CORE/lib};
+    require 'test.pl';
 }
 
 use Config;
-- 
2.5.0

