From 1158ac002c1759352622cd8e5a1f391db403de89 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 022/398] B::C patch for io/eintr.t

---
 t/CORE/io/eintr.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/eintr.t b/t/CORE/io/eintr.t
index 32f109d..7a36146 100644
--- a/t/CORE/io/eintr.t
+++ b/t/CORE/io/eintr.t
@@ -7,15 +7,14 @@
 # principally, don't segfault).
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
 use warnings;
 use strict;
 use Config;
 
-require './test.pl';
+require 'test.pl';
 
 my $piped;
 eval {
-- 
2.5.0

