From c8a77978fecec297bfaee52eaca28fa84b4ceb6b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 023/398] B::C patch for io/eintr_print.t

---
 t/CORE/io/eintr_print.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/eintr_print.t b/t/CORE/io/eintr_print.t
index 1a0d84d..254dd6e 100644
--- a/t/CORE/io/eintr_print.t
+++ b/t/CORE/io/eintr_print.t
@@ -5,9 +5,8 @@
 # also fails under 5.8.x
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     skip_all_if_miniperl("No XS under miniperl");
 }
 
-- 
2.5.0

