From e1f4340556ddeeff43a86933bf9a5c54eba04c04 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:37 -0500
Subject: [PATCH 024/398] B::C patch for io/errno.t

---
 t/CORE/io/errno.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/io/errno.t b/t/CORE/io/errno.t
index e9a6c09..948ae6b 100644
--- a/t/CORE/io/errno.t
+++ b/t/CORE/io/errno.t
@@ -7,7 +7,7 @@
 use strict;
 use Config;
 
-require './test.pl';
+require 't/CORE/test.pl';
 
 plan( tests => 16 );
 
-- 
2.5.0

