From 191bf37f2e9db852d88bd78ba8b8a34b712d8689 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 025/398] B::C patch for io/errnosig.t

---
 t/CORE/io/errnosig.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/io/errnosig.t b/t/CORE/io/errnosig.t
index 8effcf1..e7570d8 100644
--- a/t/CORE/io/errnosig.t
+++ b/t/CORE/io/errnosig.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 require Config; import Config;
-- 
2.5.0

