From 17fdd9954a3a4e4e6d308be05724f01ed007a2b6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 026/398] B::C patch for io/fflush.t

---
 t/CORE/io/fflush.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/fflush.t b/t/CORE/io/fflush.t
index 0bbfa54..09a57d6 100644
--- a/t/CORE/io/fflush.t
+++ b/t/CORE/io/fflush.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
 }
 
 # Script to test auto flush on fork/exec/system/qx.  The idea is to
-- 
2.5.0

