From e2862dc0893a8d25c526e8f92c7113ef4ff4cf8b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 027/398] B::C patch for io/fs.t

---
 t/CORE/io/fs.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/fs.t b/t/CORE/io/fs.t
index 0d3f435..fe434f5 100644
--- a/t/CORE/io/fs.t
+++ b/t/CORE/io/fs.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require "./test.pl";
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
 }
 
 use Config;
-- 
2.5.0

