From 42057213850bf4ce406105e3bc096456a6ef1d18 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 028/398] B::C patch for io/inplace.t

---
 t/CORE/io/inplace.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/io/inplace.t b/t/CORE/io/inplace.t
index 5cb56b9..181b98d 100644
--- a/t/CORE/io/inplace.t
+++ b/t/CORE/io/inplace.t
@@ -1,6 +1,6 @@
 #!./perl
 use strict;
-require './test.pl';
+require 't/CORE/test.pl';
 
 $^I = $^O eq 'VMS' ? '_bak' : '.bak';
 
-- 
2.5.0

