From 5ae1352a2a18acffb2c2cc2890100ed05c021a6a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 029/398] B::C patch for io/iofile.t

---
 t/CORE/io/iofile.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/iofile.t b/t/CORE/io/iofile.t
index f2ac66f..75e5577 100644
--- a/t/CORE/io/iofile.t
+++ b/t/CORE/io/iofile.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     skip_all_if_miniperl("miniperl can't load IO::File");
 }
 
-- 
2.5.0

