From 4bec3b6929d3657a0c43c62cf4a84bf576b5788e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 030/398] B::C patch for io/iprefix.t

---
 t/CORE/io/iprefix.t | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/t/CORE/io/iprefix.t b/t/CORE/io/iprefix.t
index 9e09ce0..d694aaa 100644
--- a/t/CORE/io/iprefix.t
+++ b/t/CORE/io/iprefix.t
@@ -1,6 +1,7 @@
 #!./perl
 use strict;
-require './test.pl';
+
+require 't/CORE/test.pl';
 
 $^I = 'bak.*';
 
-- 
2.5.0

