From 6f1c1524b2a985c3426563633b9759b6edcc85ad Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 031/398] B::C patch for io/layers.t

---
 t/CORE/io/layers.t | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/t/CORE/io/layers.t b/t/CORE/io/layers.t
index 86d171c..d6dbb65 100644
--- a/t/CORE/io/layers.t
+++ b/t/CORE/io/layers.t
@@ -3,20 +3,20 @@
 my $PERLIO;
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-    skip_all_without_perlio();
-    # FIXME - more of these could be tested without Encode or full perl
-    skip_all_without_dynamic_extension('Encode');
-
-    # Makes testing easier.
-    $ENV{PERLIO} = 'stdio' if exists $ENV{PERLIO} && $ENV{PERLIO} eq '';
-    skip_all("PERLIO='$ENV{PERLIO}' unknown")
-	if exists $ENV{PERLIO} && $ENV{PERLIO} !~ /^(stdio|perlio|mmap)$/;
-    $PERLIO = exists $ENV{PERLIO} ? $ENV{PERLIO} : "(undef)";
+    unshift @INC, 't/CORE', "t/CORE/lib";
+    require 'test.pl';
 }
 
+skip_all_without_perlio();
+# FIXME - more of these could be tested without Encode or full perl
+skip_all_without_dynamic_extension('Encode');
+
+# Makes testing easier.
+$ENV{PERLIO} = 'stdio' if exists $ENV{PERLIO} && $ENV{PERLIO} eq '';
+skip_all("PERLIO='$ENV{PERLIO}' unknown")
+if exists $ENV{PERLIO} && $ENV{PERLIO} !~ /^(stdio|perlio|mmap)$/;
+$PERLIO = exists $ENV{PERLIO} ? $ENV{PERLIO} : "(undef)";
+
 use Config;
 
 my $DOSISH    = $^O =~ /^(?:MSWin32|os2|dos|NetWare)$/ ? 1 : 0;
@@ -248,6 +248,6 @@ EOT
     is $f, 1, '1 fetch on tied string';
 
     # No distinction between nums and strings
-    open "12", "<:crlf", "test.pl" or die "$0 cannot open test.pl: $!";
+    open "12", "<:crlf", "t/CORE/test.pl" or die "$0 cannot open test.pl: $!";
     ok PerlIO::get_layers(12), 'str/num arguments are treated identically';
 }
-- 
2.5.0

