From 5a86f1b3c13488fac865c1a8b94a0eee5ab8ef8e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 035/398] B::C patch for io/perlio.t

---
 t/CORE/io/perlio.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/perlio.t b/t/CORE/io/perlio.t
index 3987a79..39830b1 100644
--- a/t/CORE/io/perlio.t
+++ b/t/CORE/io/perlio.t
@@ -1,8 +1,7 @@
 BEGIN {
-	chdir 't' if -d 't';
-	@INC = '../lib';
+	push @INC, "t/CORE/lib";
 	require Config; import Config;
-	require './test.pl';
+	require 'test.pl';
 	skip_all_without_perlio();
 }
 
-- 
2.5.0

