From cc78c68d550ffb45f41abe7b0c66483a95450d34 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:38 -0500
Subject: [PATCH 036/398] B::C patch for io/perlio_fail.t

---
 t/CORE/io/perlio_fail.t | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/t/CORE/io/perlio_fail.t b/t/CORE/io/perlio_fail.t
index 6bc9b17..688c10b 100644
--- a/t/CORE/io/perlio_fail.t
+++ b/t/CORE/io/perlio_fail.t
@@ -1,12 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require "../t/test.pl";
+    require "t/CORE/test.pl";
     skip_all_without_perlio();
-    plan (15);
 }
+plan (15);
 
 use warnings 'layer';
 my $warn;
-- 
2.5.0

