From a0cdae26affc070254ed68e1cfecc529d0429022 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 038/398] B::C patch for io/perlio_open.t

---
 t/CORE/io/perlio_open.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/perlio_open.t b/t/CORE/io/perlio_open.t
index 702c76c..25c85bf 100644
--- a/t/CORE/io/perlio_open.t
+++ b/t/CORE/io/perlio_open.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     skip_all_without_perlio();
     skip_all_without_dynamic_extension('Fcntl'); # how did you get this far?
 }
-- 
2.5.0

