From dda3cf87590c6c2f9fbb47628044b944503099e1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 039/398] B::C patch for io/pipe.t

---
 t/CORE/io/pipe.t | 18 ++++++++----------
 1 file changed, 8 insertions(+), 10 deletions(-)

diff --git a/t/CORE/io/pipe.t b/t/CORE/io/pipe.t
index fdd8b99..d62db7a 100644
--- a/t/CORE/io/pipe.t
+++ b/t/CORE/io/pipe.t
@@ -1,17 +1,15 @@
 #!./perl
 
+use Config;
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require Config; import Config;
-    require './test.pl';
+    require 't/CORE/test.pl';
+}
 
-    if (!$Config{'d_fork'}) {
-        skip_all("fork required to pipe");
-    }
-    else {
-        plan(tests => 24);
-    }
+if (!$Config{'d_fork'}) {
+    skip_all("fork required to pipe");
+}
+else {
+    plan(tests => 24);
 }
 
 my $Perl = which_perl();
-- 
2.5.0

