From aa8ca4d971be5525028491d3b65d08e5d6f76999 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 042/398] B::C patch for io/read.t

---
 t/CORE/io/read.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/read.t b/t/CORE/io/read.t
index 8cc1640..feda5e5 100644
--- a/t/CORE/io/read.t
+++ b/t/CORE/io/read.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

