From b50d39aa7f6310b6c205d21edeb0b8ed7a9502e4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 043/398] B::C patch for io/say.t

---
 t/CORE/io/say.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/say.t b/t/CORE/io/say.t
index 70f83a7..35caf76 100644
--- a/t/CORE/io/say.t
+++ b/t/CORE/io/say.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

