From f9fd6cc24262a3c0b1479034d0d83c6a67febdc5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 044/398] B::C patch for io/sem.t

---
 t/CORE/io/sem.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/sem.t b/t/CORE/io/sem.t
index 0e309eb..f9c4b32 100644
--- a/t/CORE/io/sem.t
+++ b/t/CORE/io/sem.t
@@ -1,10 +1,9 @@
 #!perl
 
 BEGIN {
-  chdir 't' if -d 't';
-  @INC = '../lib' if -d '../lib' && -d '../ext';
+  push @INC, 't/CORE/lib';
 
-  require "./test.pl";
+  require 'test.pl';
   require Config; import Config;
 
   if ($ENV{'PERL_CORE'} && $Config{'extensions'} !~ m[\bIPC/SysV\b]) {
-- 
2.5.0

