From a157621394517630935e08f69b7f10a5578337cc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:39 -0500
Subject: [PATCH 046/398] B::C patch for io/socket.t

---
 t/CORE/io/socket.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/socket.t b/t/CORE/io/socket.t
index b723e3c..5e2c952 100644
--- a/t/CORE/io/socket.t
+++ b/t/CORE/io/socket.t
@@ -3,10 +3,9 @@
 # sanity tests for socket functions
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib' if -d '../lib' && -d '../ext';
+    push @INC, 't/CORE/lib';
 
-    require "./test.pl";
+    require 'test.pl';
     require Config; import Config;
 
     skip_all_if_miniperl();
-- 
2.5.0

