From f17baa18ca9cb208ad8a63911ffae4725e6ea144 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 049/398] B::C patch for io/utf8.t

---
 t/CORE/io/utf8.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/io/utf8.t b/t/CORE/io/utf8.t
index acce07e..8885df7 100644
--- a/t/CORE/io/utf8.t
+++ b/t/CORE/io/utf8.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 'test.pl';
     skip_all_without_perlio();
 }
 
-- 
2.5.0

