From 3954fbd4291525283b05951e2b240fb5d97f1dd7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 050/398] B::C patch for lib/1_compile.t

---
 t/CORE/lib/1_compile.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/lib/1_compile.t b/t/CORE/lib/1_compile.t
index 2802ae2..dc2a8fd 100644
--- a/t/CORE/lib/1_compile.t
+++ b/t/CORE/lib/1_compile.t
@@ -5,9 +5,8 @@
 # of their own.
 
 BEGIN {
-    chdir 't';
     @INC = '../lib';
-    require './test.pl';
+    require 'test.pl';
 }
 
 use warnings;
-- 
2.5.0

