From e6add0c0986da3e6da593139102323cf7ff4e8f0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 051/398] B::C patch for lib/common.pl

---
 t/CORE/lib/common.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/lib/common.pl b/t/CORE/lib/common.pl
index 4ab00b1..2306567 100644
--- a/t/CORE/lib/common.pl
+++ b/t/CORE/lib/common.pl
@@ -6,7 +6,7 @@
 # to call cur_test() to find out how many this executed
 
 BEGIN {
-    require './test.pl';
+    require 'test.pl';
 }
 
 use Config;
-- 
2.5.0

