From 75c9a86f061460748a800a7efcf2951e52840396 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 053/398] B::C patch for lib/compmod.pl

---
 t/CORE/lib/compmod.pl | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/CORE/lib/compmod.pl b/t/CORE/lib/compmod.pl
index fa032f1..9427353 100644
--- a/t/CORE/lib/compmod.pl
+++ b/t/CORE/lib/compmod.pl
@@ -1,7 +1,6 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
     @INC = '../lib';
 }
 
-- 
2.5.0

