From e959eab0d170ef74da8590aa7954a4b843420916 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 054/398] B::C patch for lib/cygwin.t

---
 t/CORE/lib/cygwin.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/lib/cygwin.t b/t/CORE/lib/cygwin.t
index 9033d3f..c450832 100644
--- a/t/CORE/lib/cygwin.t
+++ b/t/CORE/lib/cygwin.t
@@ -1,9 +1,8 @@
 #!perl
 
 BEGIN {
-    chdir 't' if -d 't';
     @INC = ('../lib');
-    require './test.pl';
+    require 'test.pl';
     skip_all('cygwin specific test') unless $^O eq 'cygwin';
 }
 
-- 
2.5.0

