From 25bd2536c7ae1113ee94039a2384cdb11349e4dc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 055/398] B::C patch for lib/deprecate.t

---
 t/CORE/lib/deprecate.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/lib/deprecate.t b/t/CORE/lib/deprecate.t
index 9e59469..b4119db 100644
--- a/t/CORE/lib/deprecate.t
+++ b/t/CORE/lib/deprecate.t
@@ -2,7 +2,7 @@
 use strict;
 
 BEGIN {
-    require './test.pl';
+    require 'test.pl';
 }
 use File::Copy ();
 use File::Path ();
-- 
2.5.0

