From ff91966e500bf332ff2f3f61e9051d8be7a5e5da Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 056/398] B::C patch for lib/mypragma.t

---
 t/CORE/lib/mypragma.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/lib/mypragma.t b/t/CORE/lib/mypragma.t
index 0464897..5540eb8 100644
--- a/t/CORE/lib/mypragma.t
+++ b/t/CORE/lib/mypragma.t
@@ -5,7 +5,7 @@ use warnings;
 
 BEGIN {
     unshift @INC, 'lib';
-    require './test.pl';
+    require 'test.pl';
     plan(tests => 14);
 }
 
-- 
2.5.0

