From 30c53ad8c8c0ce957d4ffe34bd12252282046ce9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 057/398] B::C patch for lib/no_load.t

---
 t/CORE/lib/no_load.t | 1 -
 1 file changed, 1 deletion(-)

diff --git a/t/CORE/lib/no_load.t b/t/CORE/lib/no_load.t
index 39f0dc6..8b6887e 100644
--- a/t/CORE/lib/no_load.t
+++ b/t/CORE/lib/no_load.t
@@ -4,7 +4,6 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
     @INC = qw(. ../lib);
 }
 
-- 
2.5.0

