From 81d69655fa345a23d2936fcfcedbf8f8d6d3e596 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 058/398] B::C patch for lib/proxy_constant_subs.t

---
 t/CORE/lib/proxy_constant_subs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/lib/proxy_constant_subs.t b/t/CORE/lib/proxy_constant_subs.t
index 9e73006..a554e80 100644
--- a/t/CORE/lib/proxy_constant_subs.t
+++ b/t/CORE/lib/proxy_constant_subs.t
@@ -1,6 +1,6 @@
 my @symbols;
 BEGIN {
-    require './test.pl';
+    require 'test.pl';
     skip_all_without_dynamic_extension($_) foreach qw(B Fcntl);
     # S_IFMT is a real subroutine, and acts as control
     # SEEK_SET is a proxy constant subroutine.
-- 
2.5.0

