From cdf2f97245e8d7eaf71320454a148af3826dd062 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:40 -0500
Subject: [PATCH 059/398] B::C patch for lib/universal.t

---
 t/CORE/lib/universal.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/lib/universal.t b/t/CORE/lib/universal.t
index 19f8f28..17b3337 100644
--- a/t/CORE/lib/universal.t
+++ b/t/CORE/lib/universal.t
@@ -3,9 +3,8 @@
 # Test the Internal::* functions and other tibits in universal.c
 
 BEGIN {
-    chdir 't' if -d 't';
     @INC = '../lib';
-    require './test.pl';
+    require 'test.pl';
     plan( tests => 13 );
 }
 
-- 
2.5.0

