From 5272c1b3b9a9b6e71f410b3e7f89bba7dc5a567d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 060/398] B::C patch for lib/warnings/doio

---
 t/CORE/lib/warnings/doio | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/lib/warnings/doio b/t/CORE/lib/warnings/doio
index 63250e1..e2dfe20 100644
--- a/t/CORE/lib/warnings/doio
+++ b/t/CORE/lib/warnings/doio
@@ -290,9 +290,9 @@ EOM
 }
 use warnings 'io' ;
 close STDOUT;
-open FH1, "../harness"; close FH1;
+open FH1, "t/CORE/TEST"; close FH1;
 no warnings 'io' ;
-open FH2, "../harness"; close FH2;
+open FH2, "t/CORE/TEST"; close FH2;
 EXPECT
 Filehandle STDOUT reopened as FH1 only for input at - line 14.
 ########
@@ -311,9 +311,9 @@ EOM
 }
 use warnings 'io' ;
 close STDOUT;
-open ᶠᚻ1, "../harness"; close ᶠᚻ1;
+open ᶠᚻ1, "t/CORE/TEST"; close ᶠᚻ1;
 no warnings 'io' ;
-open ᶠᚻ2, "../harness"; close ᶠᚻ2;
+open ᶠᚻ2, "t/CORE/TEST"; close ᶠᚻ2;
 EXPECT
 Filehandle STDOUT reopened as ᶠᚻ1 only for input at - line 16.
 ########
-- 
2.5.0

