From 926ec3ebc08ebaa808c7e9fdf7f1f3a6d3fd41b3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 061/398] B::C patch for lib/warnings/pp_sys

---
 t/CORE/lib/warnings/pp_sys | 28 ++++++++++++++--------------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/t/CORE/lib/warnings/pp_sys b/t/CORE/lib/warnings/pp_sys
index 6999327..44333f4 100644
--- a/t/CORE/lib/warnings/pp_sys
+++ b/t/CORE/lib/warnings/pp_sys
@@ -650,7 +650,7 @@ binmode() on unopened filehandle at - line 4.
 ########
 # pp_sys.c [pp_lstat]
 use warnings 'io';
-open FH, "../harness" or die "# $!";
+open FH, "t/CORE/TEST" or die "# $!";
 lstat FH;
 lstat *FH;
 lstat \*FH;
@@ -674,7 +674,7 @@ lstat() on filehandle at - line 9.
 use warnings 'io';
 use utf8;
 use open qw( :utf8 :std );
-open ᶠḨ, "../harness" or die "# $!";
+open ᶠḨ, "t/CORE/TEST" or die "# $!";
 lstat ᶠḨ;
 open my $ｆᚺ, $0 or die "# $!";
 lstat $ｆᚺ;
@@ -754,11 +754,11 @@ chdir() on closed filehandle $fh at - line 23.
 use warnings;
 opendir FOO, ".";
 opendir my $foo, ".";
-open FOO, "../harness";
-open $foo, "../harness";
+open FOO, "t/CORE/TEST";
+open $foo, "t/CORE/TEST";
 no warnings qw(io deprecated);
-open FOO, "../harness";
-open $foo, "../harness";
+open FOO, "t/CORE/TEST";
+open $foo, "t/CORE/TEST";
 EXPECT
 Opening dirhandle FOO also as a file at - line 5.
 Opening dirhandle $foo also as a file at - line 6.
@@ -770,19 +770,19 @@ use open qw( :utf8 :std );
 use warnings;
 opendir ＦＯＯ, ".";
 opendir $ｆｏｏ, ".";
-open ＦＯＯ, "../harness";
-open $ｆｏｏ, "../harness";
+open ＦＯＯ, "t/CORE/TEST";
+open $ｆｏｏ, "t/CORE/TEST";
 no warnings qw(io deprecated);
-open ＦＯＯ, "../harness";
-open $ｆｏｏ, "../harness";
+open ＦＯＯ, "t/CORE/TEST";
+open $ｆｏｏ, "t/CORE/TEST";
 EXPECT
 Opening dirhandle ＦＯＯ also as a file at - line 8.
 Opening dirhandle $ｆｏｏ also as a file at - line 9.
 ########
 # pp_sys.c [pp_open_dir]
 use warnings;
-open FOO, "../harness";
-open my $foo, "../harness";
+open FOO, "t/CORE/TEST";
+open my $foo, "t/CORE/TEST";
 opendir FOO, ".";
 opendir $foo, ".";
 no warnings qw(io deprecated);
@@ -798,8 +798,8 @@ use utf8;
 use open qw( :utf8 :std );
 use warnings;
 use warnings;
-open ＦＯＯ, "../harness";
-open $ｆｏｏ, "../harness";
+open ＦＯＯ, "t/CORE/TEST";
+open $ｆｏｏ, "t/CORE/TEST";
 opendir ＦＯＯ, ".";
 opendir $ｆｏｏ, ".";
 no warnings qw(io deprecated);
-- 
2.5.0

