From 45518410107118ac9694f6f4f23f256d368299ea Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 062/398] B::C patch for mro/basic.t

---
 t/CORE/mro/basic.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/mro/basic.t b/t/CORE/mro/basic.t
index 5625b51..f72bcb7 100644
--- a/t/CORE/mro/basic.t
+++ b/t/CORE/mro/basic.t
@@ -4,9 +4,8 @@ use strict;
 use warnings;
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require q(./test.pl);
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 plan(tests => 61);
 
-- 
2.5.0

