From 81d7b0fdc57541939fbec8c08e0c4f1d068fc494 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 065/398] B::C patch for mro/basic_01_dfs.t

---
 t/CORE/mro/basic_01_dfs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_01_dfs.t b/t/CORE/mro/basic_01_dfs.t
index c4ea2be..29ae8c7 100644
--- a/t/CORE/mro/basic_01_dfs.t
+++ b/t/CORE/mro/basic_01_dfs.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 4);
+require 't/CORE/test.pl'; plan(tests => 4);
 
 =pod
 
-- 
2.5.0

