From 9d2f61a0697bce9504387193a4abc3028bcbd277 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 066/398] B::C patch for mro/basic_01_dfs_utf8.t

---
 t/CORE/mro/basic_01_dfs_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_01_dfs_utf8.t b/t/CORE/mro/basic_01_dfs_utf8.t
index b122aba..c1d7ee1 100644
--- a/t/CORE/mro/basic_01_dfs_utf8.t
+++ b/t/CORE/mro/basic_01_dfs_utf8.t
@@ -5,7 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-require q(./test.pl); plan(tests => 4);
+require 't/CORE/test.pl'; plan(tests => 4);
 
 =pod
 
-- 
2.5.0

