From 0de3f0c8519e26f4ff956d6de18b7a86d9cdaaab Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 067/398] B::C patch for mro/basic_02_c3.t

---
 t/CORE/mro/basic_02_c3.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_02_c3.t b/t/CORE/mro/basic_02_c3.t
index b2c3ce4..aa66942 100644
--- a/t/CORE/mro/basic_02_c3.t
+++ b/t/CORE/mro/basic_02_c3.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 10);
+require 't/CORE/test.pl'; plan(tests => 10);
 
 =pod
 
-- 
2.5.0

