From 0df50b616a8796ca8373ab4930c95475fdf83c45 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 069/398] B::C patch for mro/basic_02_dfs.t

---
 t/CORE/mro/basic_02_dfs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_02_dfs.t b/t/CORE/mro/basic_02_dfs.t
index a4bad46..739d735 100644
--- a/t/CORE/mro/basic_02_dfs.t
+++ b/t/CORE/mro/basic_02_dfs.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 10);
+require 't/CORE/test.pl'; plan(tests => 10);
 
 =pod
 
-- 
2.5.0

