From 898ac8b8c01904f75a34b20c7d685d9b352754ad Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:41 -0500
Subject: [PATCH 070/398] B::C patch for mro/basic_02_dfs_utf8.t

---
 t/CORE/mro/basic_02_dfs_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_02_dfs_utf8.t b/t/CORE/mro/basic_02_dfs_utf8.t
index 77d7d71..5cc569e 100644
--- a/t/CORE/mro/basic_02_dfs_utf8.t
+++ b/t/CORE/mro/basic_02_dfs_utf8.t
@@ -5,7 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-require q(./test.pl); plan(tests => 10);
+require 't/CORE/test.pl'; plan(tests => 10);
 
 =pod
 
-- 
2.5.0

