From 023f2d932728d0a1eac9d7b6bedbb84e43c408d9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:42 -0500
Subject: [PATCH 076/398] B::C patch for mro/basic_04_c3_utf8.t

---
 t/CORE/mro/basic_04_c3_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_04_c3_utf8.t b/t/CORE/mro/basic_04_c3_utf8.t
index 3665ca6..f126440 100644
--- a/t/CORE/mro/basic_04_c3_utf8.t
+++ b/t/CORE/mro/basic_04_c3_utf8.t
@@ -5,7 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-require q(./test.pl); plan(tests => 1);
+require 't/CORE/test.pl'; plan(tests => 1);
 
 =pod 
 
-- 
2.5.0

