From 5d730524829599a9f8ac617c89942f7c12f7db82 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:42 -0500
Subject: [PATCH 079/398] B::C patch for mro/basic_05_c3.t

---
 t/CORE/mro/basic_05_c3.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_05_c3.t b/t/CORE/mro/basic_05_c3.t
index 1558585..50af173 100644
--- a/t/CORE/mro/basic_05_c3.t
+++ b/t/CORE/mro/basic_05_c3.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 2);
+require 't/CORE/test.pl'; plan(tests => 2);
 
 =pod
 
-- 
2.5.0

