From 1ce55da2c3905971390e00d91d67874f3b1eccbe Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:42 -0500
Subject: [PATCH 080/398] B::C patch for mro/basic_05_c3_utf8.t

---
 t/CORE/mro/basic_05_c3_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_05_c3_utf8.t b/t/CORE/mro/basic_05_c3_utf8.t
index a295c96..2a23736 100644
--- a/t/CORE/mro/basic_05_c3_utf8.t
+++ b/t/CORE/mro/basic_05_c3_utf8.t
@@ -5,7 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-require q(./test.pl); plan(tests => 2);
+require 't/CORE/test.pl'; plan(tests => 2);
 
 =pod
 
-- 
2.5.0

