From 9ed6cc7bac165c577dfcb9d6d41a947d958a1ce3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:42 -0500
Subject: [PATCH 083/398] B::C patch for mro/basic_utf8.t

---
 t/CORE/mro/basic_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/basic_utf8.t b/t/CORE/mro/basic_utf8.t
index d0dff50..e66de0a 100644
--- a/t/CORE/mro/basic_utf8.t
+++ b/t/CORE/mro/basic_utf8.t
@@ -5,7 +5,7 @@ use open qw( :utf8 :std );
 use strict;
 use warnings;
 
-BEGIN { require q(./test.pl); } plan(tests => 53);
+BEGIN { require 't/CORE/test.pl'; } plan(tests => 53);
 
 require mro;
 
-- 
2.5.0

