From ce72d6a23acc3feb8e0993ab47b9f027f2e2116c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 085/398] B::C patch for mro/c3_with_overload_utf8.t

---
 t/CORE/mro/c3_with_overload_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/c3_with_overload_utf8.t b/t/CORE/mro/c3_with_overload_utf8.t
index a75c31a..6009e1d 100644
--- a/t/CORE/mro/c3_with_overload_utf8.t
+++ b/t/CORE/mro/c3_with_overload_utf8.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 7);
+require 't/CORE/test.pl'; plan(tests => 7);
 
 {
     package BaseTest;
-- 
2.5.0

