From 9dc3f427b599de9ce2120c896ce74c5706dc9473 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 088/398] B::C patch for mro/complex_dfs.t

---
 t/CORE/mro/complex_dfs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/complex_dfs.t b/t/CORE/mro/complex_dfs.t
index 905ac2d..ed76f9b 100644
--- a/t/CORE/mro/complex_dfs.t
+++ b/t/CORE/mro/complex_dfs.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 11);
+require 't/CORE/test.pl'; plan(tests => 11);
 
 =pod
 
-- 
2.5.0

