From 9b57f64d8951d377eb51ad7220c29a41a0c3260a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 089/398] B::C patch for mro/complex_dfs_utf8.t

---
 t/CORE/mro/complex_dfs_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/complex_dfs_utf8.t b/t/CORE/mro/complex_dfs_utf8.t
index 723af14..aeeb118 100644
--- a/t/CORE/mro/complex_dfs_utf8.t
+++ b/t/CORE/mro/complex_dfs_utf8.t
@@ -5,7 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-require q(./test.pl); plan(tests => 11);
+require 't/CORE/test.pl'; plan(tests => 11);
 
 =pod
 
-- 
2.5.0

