From 58bb186611c6a1cafad6a6d9c4aa0a737dd7b174 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 090/398] B::C patch for mro/dbic_c3.t

---
 t/CORE/mro/dbic_c3.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/dbic_c3.t b/t/CORE/mro/dbic_c3.t
index 51d8784..2d31f83 100644
--- a/t/CORE/mro/dbic_c3.t
+++ b/t/CORE/mro/dbic_c3.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 1);
+require 't/CORE/test.pl'; plan(tests => 1);
 
 =pod
 
-- 
2.5.0

