From 3ea214a2c4e52cd89f2ae24d2a8dba1cce62ac63 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:43 -0500
Subject: [PATCH 094/398] B::C patch for mro/inconsistent_c3.t

---
 t/CORE/mro/inconsistent_c3.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/inconsistent_c3.t b/t/CORE/mro/inconsistent_c3.t
index ae01e9f..3f4e15c 100644
--- a/t/CORE/mro/inconsistent_c3.t
+++ b/t/CORE/mro/inconsistent_c3.t
@@ -2,14 +2,8 @@
 
 use strict;
 use warnings;
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-}
 
-require q(./test.pl); plan(tests => 1);
+require 't/CORE/test.pl'; plan(tests => 1);
 
 require mro;
 
-- 
2.5.0

