From 14d5d12cd700d89bb2d8396a97d630586d0d68a1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 096/398] B::C patch for mro/isa_aliases.t

---
 t/CORE/mro/isa_aliases.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/isa_aliases.t b/t/CORE/mro/isa_aliases.t
index 2073e5e..c5daeb3 100644
--- a/t/CORE/mro/isa_aliases.t
+++ b/t/CORE/mro/isa_aliases.t
@@ -1,6 +1,6 @@
 #!./perl
 
-BEGIN { chdir 't'; @INC = '../lib'; require './test.pl' }
+BEGIN { require 't/CORE/test.pl' }
 
 plan 13;
 
-- 
2.5.0

