From 7c721491cdb817241ca191e69c1a89ae88f810b7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 097/398] B::C patch for mro/isa_aliases_utf8.t

---
 t/CORE/mro/isa_aliases_utf8.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/mro/isa_aliases_utf8.t b/t/CORE/mro/isa_aliases_utf8.t
index ef715a2..7287c25 100644
--- a/t/CORE/mro/isa_aliases_utf8.t
+++ b/t/CORE/mro/isa_aliases_utf8.t
@@ -1,7 +1,6 @@
 #!./perl
 
-BEGIN { chdir 't'; @INC = '../lib'; require './test.pl' }
-
+BEGIN { require 't/CORE/test.pl'; }
 use utf8;
 use open qw( :utf8 :std );
 
-- 
2.5.0

