From 5f3d9ea8b982439214c5e91c0ce1a0f37fc353d8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 098/398] B::C patch for mro/isa_c3.t

---
 t/CORE/mro/isa_c3.t | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/t/CORE/mro/isa_c3.t b/t/CORE/mro/isa_c3.t
index 20ae5f0..ad19465 100644
--- a/t/CORE/mro/isa_c3.t
+++ b/t/CORE/mro/isa_c3.t
@@ -1,10 +1,6 @@
 #!perl -w
 
-BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require "./test.pl";
-}
+BEGIN { require 't/CORE/test.pl' }
 
 use strict;
 
-- 
2.5.0

