From b9acc231e652c69e62ad6225af801dc86fb313ed Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 102/398] B::C patch for mro/isarev.t

---
 t/CORE/mro/isarev.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/isarev.t b/t/CORE/mro/isarev.t
index 3bd543d..c3e662c 100644
--- a/t/CORE/mro/isarev.t
+++ b/t/CORE/mro/isarev.t
@@ -1,12 +1,6 @@
 #!./perl
 
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-    require q(./test.pl);
-}
+BEGIN { require 't/CORE/test.pl' }
 
 use strict;
 use warnings;
-- 
2.5.0

