From 7bc55256343591fe82ca89f54ba0be54876f27b3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 103/398] B::C patch for mro/isarev_utf8.t

---
 t/CORE/mro/isarev_utf8.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/isarev_utf8.t b/t/CORE/mro/isarev_utf8.t
index dff3058..a1e4d71 100644
--- a/t/CORE/mro/isarev_utf8.t
+++ b/t/CORE/mro/isarev_utf8.t
@@ -1,12 +1,6 @@
 #!./perl
 
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-    require q(./test.pl);
-}
+BEGIN { require 't/CORE/test.pl' }
 
 use strict;
 use warnings;
-- 
2.5.0

