From 943612db8a440d0d2ffe5912292936e2037817c4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 104/398] B::C patch for mro/method_caching.t

---
 t/CORE/mro/method_caching.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/method_caching.t b/t/CORE/mro/method_caching.t
index 3f21b1b..6c7f074 100644
--- a/t/CORE/mro/method_caching.t
+++ b/t/CORE/mro/method_caching.t
@@ -6,13 +6,7 @@ use warnings;
 no warnings 'redefine'; # we do a lot of this
 no warnings 'prototype'; # we do a lot of this
 
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-    require './test.pl';
-}
+BEGIN { require 't/CORE/test.pl' }
 
 {
     package MCTest::Base;
-- 
2.5.0

