From c7aaa311687893cac1064b35baf607caf6f456c2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 105/398] B::C patch for mro/method_caching_utf8.t

---
 t/CORE/mro/method_caching_utf8.t | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/t/CORE/mro/method_caching_utf8.t b/t/CORE/mro/method_caching_utf8.t
index b0a451d..b628db6 100644
--- a/t/CORE/mro/method_caching_utf8.t
+++ b/t/CORE/mro/method_caching_utf8.t
@@ -7,14 +7,7 @@ use warnings;
 no warnings 'redefine'; # we do a lot of this
 no warnings 'prototype'; # we do a lot of this
 
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-}
-
-require './test.pl';
+require 't/CORE/test.pl';
 
 {
     package MC텟ᵀ::Bࡎᶓ;
-- 
2.5.0

