From 9633bd5455760d707baa27e6956066d4e81ae249 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:44 -0500
Subject: [PATCH 106/398] B::C patch for mro/next_NEXT.t

---
 t/CORE/mro/next_NEXT.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_NEXT.t b/t/CORE/mro/next_NEXT.t
index e8d85fc..8df4678 100644
--- a/t/CORE/mro/next_NEXT.t
+++ b/t/CORE/mro/next_NEXT.t
@@ -4,7 +4,7 @@ use strict;
 use warnings;
 use NEXT;
 
-require './test.pl';
+BEGIN { require 't/CORE/test.pl' }
 plan(tests => 4);
 
 {
-- 
2.5.0

