From 27ce92d8c682151026e4e49300ec610d67c1c8b8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 107/398] B::C patch for mro/next_NEXT_utf8.t

---
 t/CORE/mro/next_NEXT_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_NEXT_utf8.t b/t/CORE/mro/next_NEXT_utf8.t
index 5961a95..131d471 100644
--- a/t/CORE/mro/next_NEXT_utf8.t
+++ b/t/CORE/mro/next_NEXT_utf8.t
@@ -6,7 +6,7 @@ use NEXT;
 use utf8;
 use open qw( :utf8 :std );
 
-require './test.pl';
+BEGIN { require 't/CORE/test.pl'}
 plan(tests => 4);
 
 {
-- 
2.5.0

