From 16fded86ddd5dcb490692f13323000b13ed5d2db Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 108/398] B::C patch for mro/next_edgecases.t

---
 t/CORE/mro/next_edgecases.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_edgecases.t b/t/CORE/mro/next_edgecases.t
index e177d70..0ce24a5 100644
--- a/t/CORE/mro/next_edgecases.t
+++ b/t/CORE/mro/next_edgecases.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-BEGIN { chdir 't'; require q(./test.pl); @INC = qw "../lib lib" }
+require 't/CORE/test.pl';
 
 plan(tests => 12);
 
-- 
2.5.0

