From 8d96a973ae0f60b88ef1c47678dc144ec9f87d3a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 109/398] B::C patch for mro/next_edgecases_utf8.t

---
 t/CORE/mro/next_edgecases_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_edgecases_utf8.t b/t/CORE/mro/next_edgecases_utf8.t
index ba6ff8b..127235e 100644
--- a/t/CORE/mro/next_edgecases_utf8.t
+++ b/t/CORE/mro/next_edgecases_utf8.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-BEGIN { chdir 't'; require q(./test.pl); @INC = qw "../lib lib" }
+require 't/CORE/test.pl';
 
 use utf8;
 use open qw( :utf8 :std );
-- 
2.5.0

