From 92d7405b27d91fd49d1ce9d369b7422e572e1b11 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 110/398] B::C patch for mro/next_goto.t

---
 t/CORE/mro/next_goto.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_goto.t b/t/CORE/mro/next_goto.t
index b7a33ec..a671677 100644
--- a/t/CORE/mro/next_goto.t
+++ b/t/CORE/mro/next_goto.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 4);
+require 't/CORE/test.pl'; plan(tests => 4);
 
 use mro;
 
-- 
2.5.0

