From 734e05a47f38fadc69b2c58d65270159a7e8b02c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 112/398] B::C patch for mro/next_inanon.t

---
 t/CORE/mro/next_inanon.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_inanon.t b/t/CORE/mro/next_inanon.t
index b6f0451..c967e78 100644
--- a/t/CORE/mro/next_inanon.t
+++ b/t/CORE/mro/next_inanon.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 2);
+require 't/CORE/test.pl'; plan(tests => 2);
 
 =pod
 
-- 
2.5.0

