From 5363de5888ad26fd7d22870d17b904bde711c2e8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 113/398] B::C patch for mro/next_inanon_utf8.t

---
 t/CORE/mro/next_inanon_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_inanon_utf8.t b/t/CORE/mro/next_inanon_utf8.t
index d1dd6e4..fc7fde7 100644
--- a/t/CORE/mro/next_inanon_utf8.t
+++ b/t/CORE/mro/next_inanon_utf8.t
@@ -4,7 +4,7 @@ use strict;
 use warnings;
 use utf8;
 use open qw( :utf8 :std );
-require q(./test.pl); plan(tests => 2);
+require 't/CORE/test.pl'; plan(tests => 2);
 
 =pod
 
-- 
2.5.0

