From 7388578721e1a4e5d08e43e2463eb29f966a972e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:45 -0500
Subject: [PATCH 116/398] B::C patch for mro/next_method.t

---
 t/CORE/mro/next_method.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_method.t b/t/CORE/mro/next_method.t
index b737917..a4c575a 100644
--- a/t/CORE/mro/next_method.t
+++ b/t/CORE/mro/next_method.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 5);
+require 't/CORE/test.pl'; plan(tests => 5);
 
 =pod
 
-- 
2.5.0

