From 0ab1c9e6d4ab53e9ee27903c5324d38b5527b38d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 118/398] B::C patch for mro/next_skip.t

---
 t/CORE/mro/next_skip.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_skip.t b/t/CORE/mro/next_skip.t
index 1614855..a0ac4eb 100644
--- a/t/CORE/mro/next_skip.t
+++ b/t/CORE/mro/next_skip.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 10);
+require 't/CORE/test.pl'; plan(tests => 10);
 
 =pod
 
-- 
2.5.0

