From 73df59405b25c99140f4ad915a1ce6f54fdc75f4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 119/398] B::C patch for mro/next_skip_utf8.t

---
 t/CORE/mro/next_skip_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/mro/next_skip_utf8.t b/t/CORE/mro/next_skip_utf8.t
index 64e7745..41b008c 100644
--- a/t/CORE/mro/next_skip_utf8.t
+++ b/t/CORE/mro/next_skip_utf8.t
@@ -3,7 +3,7 @@
 use strict;
 use warnings;
 
-require q(./test.pl); plan(tests => 10);
+require 't/CORE/test.pl'; plan(tests => 10);
 
 use utf8;
 use open qw( :utf8 :std );
-- 
2.5.0

