From 77766e4cc1fa3e2ce69021ee7238809ae5ff7765 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 122/398] B::C patch for mro/overload_dfs.t

---
 t/CORE/mro/overload_dfs.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/overload_dfs.t b/t/CORE/mro/overload_dfs.t
index 5943c85..1351094 100644
--- a/t/CORE/mro/overload_dfs.t
+++ b/t/CORE/mro/overload_dfs.t
@@ -2,14 +2,8 @@
 
 use strict;
 use warnings;
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-}
 
-require q(./test.pl); plan(tests => 7);
+require 't/CORE/test.pl'; plan(tests => 7);
 
 {
     package BaseTest;
-- 
2.5.0

