From 3515214e13e7ebdbe6c45647f264d6c28ab9c2ad Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 124/398] B::C patch for mro/package_aliases_utf8.t

---
 t/CORE/mro/package_aliases_utf8.t | 24 ++++++------------------
 1 file changed, 6 insertions(+), 18 deletions(-)

diff --git a/t/CORE/mro/package_aliases_utf8.t b/t/CORE/mro/package_aliases_utf8.t
index 09db6b7..ed28c5a 100644
--- a/t/CORE/mro/package_aliases_utf8.t
+++ b/t/CORE/mro/package_aliases_utf8.t
@@ -1,13 +1,6 @@
 #!./perl
 
-BEGIN {
-    $ENV{PERL_UNICODE} = 0;
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-    require q(./test.pl);
-}
+BEGIN { require 't/CORE/test.pl' }
 
 use strict;
 use warnings;
@@ -57,8 +50,7 @@ for(
 my $prog =    q~
      BEGIN {
          unless (-d 'blib') {
-             chdir 't' if -d 't';
-             @INC = '../lib';
+             push @INC, "t/CORE/lib";
          }
      }
      use utf8;
@@ -119,8 +111,7 @@ for(
  my $prog = q~
      BEGIN {
          unless (-d 'blib') {
-             chdir 't' if -d 't';
-             @INC = '../lib';
+             push @INC, "t/CORE/lib";
          }
      }
      use utf8;
@@ -184,8 +175,7 @@ for(
   my $prog =     q~
      BEGIN {
          unless (-d 'blib') {
-             chdir 't' if -d 't';
-             @INC = '../lib';
+             push @INC, "t/CORE/lib";
          }
      }
       use utf8;
@@ -220,8 +210,7 @@ for(
   $prog =     q~
       BEGIN {
          unless (-d 'blib') {
-             chdir 't' if -d 't';
-             @INC = '../lib';
+             push @INC, "t/CORE/lib";
          }
       }
       use utf8;
@@ -304,8 +293,7 @@ no warnings; # temporary; there seems to be a scoping bug, as this does not
 my $prog =    q~#!perl -w
      BEGIN {
          unless (-d 'blib') {
-             chdir 't' if -d 't';
-             @INC = '../lib';
+             push @INC, "t/CORE/lib";
          }
      }
      use utf8;
-- 
2.5.0

