From 07a8395ece891da305af65bf1db38ea58629e905 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 125/398] B::C patch for mro/pkg_gen.t

---
 t/CORE/mro/pkg_gen.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/mro/pkg_gen.t b/t/CORE/mro/pkg_gen.t
index e236ea9..f431bc1 100644
--- a/t/CORE/mro/pkg_gen.t
+++ b/t/CORE/mro/pkg_gen.t
@@ -3,8 +3,7 @@
 use strict;
 use warnings;
 
-chdir 't' if -d 't';
-require q(./test.pl); plan(tests => 7);
+require 't/CORE/test.pl'; plan(tests => 7);
 
 require mro;
 
-- 
2.5.0

