From f98ebedb6b8cb8cb53565706319c018a060448e5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:46 -0500
Subject: [PATCH 126/398] B::C patch for mro/pkg_gen_utf8.t

---
 t/CORE/mro/pkg_gen_utf8.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/mro/pkg_gen_utf8.t b/t/CORE/mro/pkg_gen_utf8.t
index c572c5b..aefb1b9 100644
--- a/t/CORE/mro/pkg_gen_utf8.t
+++ b/t/CORE/mro/pkg_gen_utf8.t
@@ -5,8 +5,7 @@ use warnings;
 use utf8;
 use open qw( :utf8 :std );
 
-chdir 't' if -d 't';
-require q(./test.pl); plan(tests => 7);
+require 't/CORE/test.pl'; plan(tests => 7);
 
 require mro;
 
-- 
2.5.0

