From 27c73018f59376a5b356495bc91a16466a9b1d24 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 129/398] B::C patch for mro/recursion_dfs.t

---
 t/CORE/mro/recursion_dfs.t | 8 +-------
 1 file changed, 1 insertion(+), 7 deletions(-)

diff --git a/t/CORE/mro/recursion_dfs.t b/t/CORE/mro/recursion_dfs.t
index ddb4d31..a950944 100644
--- a/t/CORE/mro/recursion_dfs.t
+++ b/t/CORE/mro/recursion_dfs.t
@@ -2,14 +2,8 @@
 
 use strict;
 use warnings;
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-}
 
-require './test.pl';
+require 't/CORE/test.pl';
 
 plan(skip_all => "Your system has no SIGALRM") if !exists $SIG{ALRM};
 plan(tests => 8);
-- 
2.5.0

