From 0ce7411c7b377a06f7bbc86bcffa9c18500c18e9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 130/398] B::C patch for mro/recursion_dfs_utf8.t

---
 t/CORE/mro/recursion_dfs_utf8.t | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/t/CORE/mro/recursion_dfs_utf8.t b/t/CORE/mro/recursion_dfs_utf8.t
index 6b428e1..e0206a4 100644
--- a/t/CORE/mro/recursion_dfs_utf8.t
+++ b/t/CORE/mro/recursion_dfs_utf8.t
@@ -2,17 +2,11 @@
 
 use strict;
 use warnings;
-BEGIN {
-    unless (-d 'blib') {
-        chdir 't' if -d 't';
-        @INC = '../lib';
-    }
-}
+require 't/CORE/test.pl';
+
 use utf8;
 use open qw( :utf8 :std );
 
-require './test.pl';
-
 plan(skip_all => "Your system has no SIGALRM") if !exists $SIG{ALRM};
 plan(tests => 8);
 
-- 
2.5.0

