From f10f093a39099e2cf164636a636113e55029f770 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 135/398] B::C patch for op/64bitint.t

---
 t/CORE/op/64bitint.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/64bitint.t b/t/CORE/op/64bitint.t
index 168d597..0d0b79c 100644
--- a/t/CORE/op/64bitint.t
+++ b/t/CORE/op/64bitint.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     eval { my $q = pack "q", 0 };
     skip_all('no 64-bit types') if $@;
 }
-- 
2.5.0

