From fd2c8a6f0faf32c458067c4c8e7eeecbe46de6e5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 136/398] B::C patch for op/alarm.t

---
 t/CORE/op/alarm.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/alarm.t b/t/CORE/op/alarm.t
index 82691c5..7c1e3e1 100644
--- a/t/CORE/op/alarm.t
+++ b/t/CORE/op/alarm.t
@@ -1,9 +1,8 @@
 #!./perl 
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 BEGIN {
-- 
2.5.0

