From 25937ca4adb52c63b4d6b04dcdc73fcf4805a718 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 137/398] B::C patch for op/anonsub.t

---
 t/CORE/op/anonsub.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/anonsub.t b/t/CORE/op/anonsub.t
index 6b8745f..c0cf823 100644
--- a/t/CORE/op/anonsub.t
+++ b/t/CORE/op/anonsub.t
@@ -1,8 +1,7 @@
 #!./perl -w
 
-chdir 't' if -d 't';
-@INC = '../lib';
-require './test.pl';
+push @INC, "t/CORE/lib";
+require 't/CORE/test.pl';
 use strict;
 
 $|=1;
-- 
2.5.0

