From 334286ab849a1614ba56668a8788b2ce4f21b4dd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:47 -0500
Subject: [PATCH 138/398] B::C patch for op/append.t

---
 t/CORE/op/append.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/append.t b/t/CORE/op/append.t
index 44847b5..d0bcac2 100644
--- a/t/CORE/op/append.t
+++ b/t/CORE/op/append.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 ##Literal test count since evals below can fail
-- 
2.5.0

