From e30c6420eeedd895851c11ab1d711bc8feb77107 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 140/398] B::C patch for op/array.t

---
 t/CORE/op/array.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/array.t b/t/CORE/op/array.t
index 7486808..d47a32b 100644
--- a/t/CORE/op/array.t
+++ b/t/CORE/op/array.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('.', '../lib');
-    require 'test.pl';
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 plan (171);
-- 
2.5.0

