From 59408e2b1a94a6ce9b384abef13f93dbf9c933fa Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 141/398] B::C patch for op/array_base.t

---
 t/CORE/op/array_base.t | 29 +++++++++++++++++------------
 1 file changed, 17 insertions(+), 12 deletions(-)

diff --git a/t/CORE/op/array_base.t b/t/CORE/op/array_base.t
index a276240..4f24ba8 100644
--- a/t/CORE/op/array_base.t
+++ b/t/CORE/op/array_base.t
@@ -1,22 +1,27 @@
 #!perl -w
 use strict;
 
-BEGIN {
- require './test.pl';
+my %tests_in_begin_block;
 
- plan (tests => my $tests = 11);
-
- # Run these at BEGIN time, before arybase loads
+BEGIN { 
  use v5.15;
- is(eval('$[ = 1; 123'), undef);
- like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
+ # Run these at BEGIN time, before arybase loads
+ $tests_in_begin_block{'eval'} = eval('$[ = 1; 123');
+ $tests_in_begin_block{'error'} = $@;
+}
+
+require 't/CORE/test.pl';
+
+plan (tests => my $tests = 11);
+
+is($tests_in_begin_block{'eval'}, undef);
+like($tests_in_begin_block{'error'}, qr/\AAssigning non-zero to \$\[ is no longer possible/);
 
  if (is_miniperl()) {
    # skip the rest
    SKIP: { skip ("no arybase.xs on miniperl", $tests-2) }
    exit;
  }
-}
 
 no warnings 'deprecated';
 
@@ -24,17 +29,17 @@ is(eval('$['), 0);
 is(eval('$[ = 0; 123'), 123);
 is(eval('$[ = 1; 123'), 123);
 $[ = 1;
-ok $INC{'arybase.pm'};
+ok($INC{'arybase.pm'}, "arybase is in INC");
 
 use v5.15;
 is(eval('$[ = 1; 123'), undef);
 like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
-is $[, 0, '$[ is 0 under 5.16';
+is($[, 0, '$[ is 0 under 5.16');
 $_ = "hello";
 /l/g;
 my $pos = \pos;
-is $$pos, 3;
+is($$pos, 3);
 $$pos = 1;
-is $$pos, 1;
+is($$pos, 1);
 
 1;
-- 
2.5.0

