From 7dda4bf71f4929962a7b28835530e42346b3c266 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 142/398] B::C patch for op/assignwarn.t

---
 t/CORE/op/assignwarn.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/assignwarn.t b/t/CORE/op/assignwarn.t
index ac1ad77..f803ef8 100644
--- a/t/CORE/op/assignwarn.t
+++ b/t/CORE/op/assignwarn.t
@@ -8,9 +8,8 @@
 # Now almost completely rewritten.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

