From cc64e93bbe872b493908f164c01b652b3f28a800 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 143/398] B::C patch for op/attrhand.t

---
 t/CORE/op/attrhand.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/attrhand.t b/t/CORE/op/attrhand.t
index 996d572..7ebc17c 100644
--- a/t/CORE/op/attrhand.t
+++ b/t/CORE/op/attrhand.t
@@ -1,9 +1,8 @@
 #!/usr/bin/perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("miniperl can't load attributes");
 }
 
-- 
2.5.0

