From 17c4e7cab0ea1cef78c0ff6b2750385e195df38d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 145/398] B::C patch for op/attrs.t

---
 t/CORE/op/attrs.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/attrs.t b/t/CORE/op/attrs.t
index ec6768e..74fac2f 100644
--- a/t/CORE/op/attrs.t
+++ b/t/CORE/op/attrs.t
@@ -3,9 +3,8 @@
 # Regression tests for attributes.pm and the C< : attrs> syntax.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("miniperl can't load attributes");
 }
 
-- 
2.5.0

