From 19126cc2218c1cf97e02d743756ec06c5d925cae Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 146/398] B::C patch for op/auto.t

---
 t/CORE/op/auto.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/auto.t b/t/CORE/op/auto.t
index 00f7caa..d06626e 100644
--- a/t/CORE/op/auto.t
+++ b/t/CORE/op/auto.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
 }
 
-- 
2.5.0

