From a205eb8c1b32f7f7a75be2edca2515c5ccd53c63 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 147/398] B::C patch for op/avhv.t

---
 t/CORE/op/avhv.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/avhv.t b/t/CORE/op/avhv.t
index d301fad..b935eee 100644
--- a/t/CORE/op/avhv.t
+++ b/t/CORE/op/avhv.t
@@ -4,8 +4,7 @@
 # they were properly removed in 5.9.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
 require Tie::Array;
@@ -20,7 +19,7 @@ sub STORESIZE { $#{$_[0]} = $_[1]+1 }
 
 package main;
 
-require './test.pl';
+require 't/CORE/test.pl';
 plan(tests => 40);
 
 # Helper function to check the typical error message.
-- 
2.5.0

