From 3f9b5941fb793169c52bff4b8a907e0a5d8be4b3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 148/398] B::C patch for op/bless.t

---
 t/CORE/op/bless.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/bless.t b/t/CORE/op/bless.t
index 69217fd..8f9e68d 100644
--- a/t/CORE/op/bless.t
+++ b/t/CORE/op/bless.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan (114);
-- 
2.5.0

