From 5346e16dcdd65ef44b3e0b820e3903b358aed137 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 149/398] B::C patch for op/blocks.t

---
 t/CORE/op/blocks.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/blocks.t b/t/CORE/op/blocks.t
index fb15eee..57371c8 100644
--- a/t/CORE/op/blocks.t
+++ b/t/CORE/op/blocks.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 7;
-- 
2.5.0

