From 034a0b9a19511267207997901b2b322adfd8ccf6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:48 -0500
Subject: [PATCH 150/398] B::C patch for op/bop.t

---
 t/CORE/op/bop.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/bop.t b/t/CORE/op/bop.t
index fa08e98..dd6144d 100644
--- a/t/CORE/op/bop.t
+++ b/t/CORE/op/bop.t
@@ -5,9 +5,8 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require "./test.pl";
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     require Config;
 }
 
-- 
2.5.0

