From 6af00dc7fafb85eafd01059e26e67bc943dad83c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 152/398] B::C patch for op/chars.t

---
 t/CORE/op/chars.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/chars.t b/t/CORE/op/chars.t
index 3fa9b8f..776b1d6 100644
--- a/t/CORE/op/chars.t
+++ b/t/CORE/op/chars.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 34;
-- 
2.5.0

