From 8536006f0e095d54fb2a3caf345d8f23510c2a5f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 153/398] B::C patch for op/chdir.t

---
 t/CORE/op/chdir.t | 20 +++++++++++++-------
 1 file changed, 13 insertions(+), 7 deletions(-)

diff --git a/t/CORE/op/chdir.t b/t/CORE/op/chdir.t
index 2c6535b..b3c987c 100644
--- a/t/CORE/op/chdir.t
+++ b/t/CORE/op/chdir.t
@@ -1,19 +1,25 @@
 #!./perl -w
 
+my $zero;
+
 BEGIN {
     # We're not going to chdir() into 't' because we don't know if
     # chdir() works!  Instead, we'll hedge our bets and put both
     # possibilities into @INC.
-    @INC = qw(t . lib ../lib);
-    require "test.pl";
+    require "t/CORE/test.pl";
     # Really want to know if chdir is working, as the build process will all go
     # wrong if it is not.
     if (is_miniperl() && !eval {require File::Spec::Functions; 1}) {
 	push @INC, qw(dist/Cwd/lib dist/Cwd ../dist/Cwd/lib ../dist/Cwd);
     }
-    plan(tests => 48);
+    # freeze $0 for compilation
+    $zero = $0;
 }
 
+chdir 't/CORE';
+
+plan(tests => 48);
+
 use Config;
 
 my $IsVMS   = $^O eq 'VMS';
@@ -52,7 +58,7 @@ my $Cwd = abs_path;
 # Let's get to a known position
 SKIP: {
     my ($vol,$dir) = splitpath(abs_path,1);
-    my $test_dir = 't';
+    my $test_dir = 'CORE';
     my $compare_dir = (splitdir($dir))[-1];
 
     # VMS is case insensitive but will preserve case in EFS mode.
@@ -158,8 +164,8 @@ sub check_env {
         ok( chdir(undef),           "chdir(undef) w/ only \$ENV{$key} set" );
         is( abs_path, $ENV{$key},   '  abs_path() agrees' );
         is( $warning,  <<WARNING,   '  got uninit & deprecation warning' );
-Use of uninitialized value in chdir at $0 line 64.
-Use of chdir('') or chdir(undef) as chdir() is deprecated at $0 line 64.
+Use of uninitialized value in chdir at $zero line 64.
+Use of chdir('') or chdir(undef) as chdir() is deprecated at $zero line 64.
 WARNING
 
         chdir($Cwd);
@@ -170,7 +176,7 @@ WARNING
         ok( chdir(''),              "chdir('') w/ only \$ENV{$key} set" );
         is( abs_path, $ENV{$key},   '  abs_path() agrees' );
         is( $warning,  <<WARNING,   '  got deprecation warning' );
-Use of chdir('') or chdir(undef) as chdir() is deprecated at $0 line 76.
+Use of chdir('') or chdir(undef) as chdir() is deprecated at $zero line 76.
 WARNING
 
         chdir($Cwd);
-- 
2.5.0

