From debd681166568624e5b2017c221145e5c847bd54 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 154/398] B::C patch for op/chop.t

---
 t/CORE/op/chop.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/chop.t b/t/CORE/op/chop.t
index 3f2247f..aab0f69 100644
--- a/t/CORE/op/chop.t
+++ b/t/CORE/op/chop.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 143;
-- 
2.5.0

