From c2f3a0562a91f5d40ccca307ade7980c9fe29e0e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 155/398] B::C patch for op/chr.t

---
 t/CORE/op/chr.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/chr.t b/t/CORE/op/chr.t
index 57b4ade..4e38d7d 100644
--- a/t/CORE/op/chr.t
+++ b/t/CORE/op/chr.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib); # ../lib needed for test.deparse
+    push @INC, qw{t/CORE/lib}; # ../lib needed for test.deparse
     require "test.pl";
 }
 
-- 
2.5.0

