From 5cd863e03ca43623a3986255dbb63ec6f60a5abc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 156/398] B::C patch for op/closure.t

---
 t/CORE/op/closure.t | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/closure.t b/t/CORE/op/closure.t
index 82e65c5..8b8e753 100644
--- a/t/CORE/op/closure.t
+++ b/t/CORE/op/closure.t
@@ -7,9 +7,7 @@
 #   Run with -debug for debugging output.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use Config;
@@ -241,7 +239,7 @@ END_MARK_TWO
     print "not ok: got unexpected warning \$msg\\n";
 } }
 
-require './test.pl';
+require 't/CORE/test.pl';
 curr_test($test);
 
 # some of the variables which the closure will access
@@ -731,7 +729,7 @@ BEGIN {
 
 # [perl #113812] Closure prototypes with no CvOUTSIDE (crash caused by the
 #                fix for #89544)
-do "./op/closure_test.pl" or die $@||$!;
+do "./t/CORE/op/closure_test.pl" or die $@||$!;
 is $closure_test::s2->()(), '10 cubes',
   'cloning closure proto with no CvOUTSIDE';
 
-- 
2.5.0

