From ab21caca161121a55a74e2e407982ccecdd58d2f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 157/398] B::C patch for op/concat2.t

---
 t/CORE/op/concat2.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/concat2.t b/t/CORE/op/concat2.t
index 9dfcc5b..23ad167 100644
--- a/t/CORE/op/concat2.t
+++ b/t/CORE/op/concat2.t
@@ -6,9 +6,8 @@
 # its ok() function.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan 3;
-- 
2.5.0

