From 53a54cad4a577e423b802bd65de2f924f09aaad0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 158/398] B::C patch for op/cond.t

---
 t/CORE/op/cond.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/cond.t b/t/CORE/op/cond.t
index acf0704..c4264f5 100644
--- a/t/CORE/op/cond.t
+++ b/t/CORE/op/cond.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 is( 1 ? 1 : 0, 1, 'compile time, true' );
-- 
2.5.0

