From c0883601983a24258cc8b9702533b86644afb8b4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 159/398] B::C patch for op/context.t

---
 t/CORE/op/context.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/context.t b/t/CORE/op/context.t
index 8753d1e..6600f65 100644
--- a/t/CORE/op/context.t
+++ b/t/CORE/op/context.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 require "test.pl";
-- 
2.5.0

