From a0ec7eca7aaa635c5946d107f8ddb4df66740139 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 160/398] B::C patch for op/coreamp.t

---
 t/CORE/op/coreamp.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/coreamp.t b/t/CORE/op/coreamp.t
index addc4bb..7a28d82 100644
--- a/t/CORE/op/coreamp.t
+++ b/t/CORE/op/coreamp.t
@@ -8,12 +8,12 @@
 # Other tests for CORE subs are in coresubs.t
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "test.pl";
+    require "t/CORE/test.pl";
     $^P |= 0x100;
 }
 
+chdir 't/CORE';
+
 no warnings 'experimental::smartmatch';
 
 sub lis($$;$) {
@@ -988,7 +988,7 @@ like $@, qr'^Undefined format "STDOUT" called',
   require File::Spec::Functions;
   my $keywords_file =
    File::Spec::Functions::catfile(
-      File::Spec::Functions::updir,'regen','keywords.pl'
+      'lib','keywords.pl'
    );
   open my $kh, $keywords_file
     or die "$0 cannot open $keywords_file: $!";
-- 
2.5.0

