From 7a287b024349e32cfcac2e8567a12fbd79523114 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 161/398] B::C patch for op/coresubs.t

---
 t/CORE/op/coresubs.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/coresubs.t b/t/CORE/op/coresubs.t
index 2e93c71..c134880 100644
--- a/t/CORE/op/coresubs.t
+++ b/t/CORE/op/coresubs.t
@@ -5,9 +5,8 @@
 # file, too.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require "test.pl";
+    push @INC, qw{t/CORE/lib};
+    require "t/CORE/test.pl";
     skip_all_without_dynamic_extension('B');
     $^P |= 0x100;
 }
@@ -42,7 +41,8 @@ my %desc = (
 );
 
 use File::Spec::Functions;
-my $keywords_file = catfile(updir,'regen','keywords.pl');
+chdir 't/CORE/op';
+my $keywords_file = catfile(updir,'lib','keywords.pl');
 open my $kh, $keywords_file
    or die "$0 cannot open $keywords_file: $!";
 while(<$kh>) {
-- 
2.5.0

