From 582e5af50eda933a1e32f9b1b4cb28841649d553 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 162/398] B::C patch for op/cproto.t

---
 t/CORE/op/cproto.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/cproto.t b/t/CORE/op/cproto.t
index 85b86db..77708b8 100644
--- a/t/CORE/op/cproto.t
+++ b/t/CORE/op/cproto.t
@@ -2,11 +2,10 @@
 # Tests to ensure that we don't unexpectedly change prototypes of builtins
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
-BEGIN { require './test.pl'; }
+BEGIN { require 't/CORE/test.pl'; }
 plan tests => 254;
 
 while (<DATA>) {
-- 
2.5.0

