From c7289b3cda81de5ae08e26bc3a85489da178d6df Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:49 -0500
Subject: [PATCH 163/398] B::C patch for op/crypt.t

---
 t/CORE/op/crypt.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/crypt.t b/t/CORE/op/crypt.t
index 27c878f..9d80175 100644
--- a/t/CORE/op/crypt.t
+++ b/t/CORE/op/crypt.t
@@ -1,14 +1,14 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 BEGIN {
     use Config;
 
     require "test.pl";
+}
 
     if( !$Config{d_crypt} ) {
         skip_all("crypt unimplemented");
@@ -16,7 +16,7 @@ BEGIN {
     else {
         plan(tests => 4);
     }
-}
+
 
 # Can't assume too much about the string returned by crypt(),
 # and about how many bytes of the encrypted (really, hashed)
-- 
2.5.0

