From 12b414546db60a7128f2ea20a831a57461c0ae1f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 164/398] B::C patch for op/current_sub.t

---
 t/CORE/op/current_sub.t | 15 ++-------------
 1 file changed, 2 insertions(+), 13 deletions(-)

diff --git a/t/CORE/op/current_sub.t b/t/CORE/op/current_sub.t
index 8c82d11..920e51f 100644
--- a/t/CORE/op/current_sub.t
+++ b/t/CORE/op/current_sub.t
@@ -1,11 +1,9 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = qw(../lib);
-    require './test.pl';
-    plan (tests => 17);
+    require 't/CORE/test.pl';
 }
+plan (tests => 15);
 
 is __SUB__, "__SUB__", '__SUB__ is a bareword outside of use feature';
 
@@ -37,15 +35,6 @@ is $subsubs[0]()(0), 1, '__SUB__ inside closure (1)';
 is $subsubs[1]()(0), 2, '__SUB__ inside closure (2)';
 is $subsubs[2]()(0), 3, '__SUB__ inside closure (3)';
 
-BEGIN {
-    return "begin 1" if @_;
-    is CORE::__SUB__->(0), "begin 1", 'in BEGIN block'
-}
-BEGIN {
-    return "begin 2" if @_;
-    is &CORE::__SUB__->(0), "begin 2", 'in BEGIN block via & (unoptimised)'
-}
-
 sub bar;
 sub bar {
     () = sort {
-- 
2.5.0

