From 6bf4ac60bb0fda0cfd21f50976627a579e7822cd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 165/398] B::C patch for op/dbm.t

---
 t/CORE/op/dbm.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/dbm.t b/t/CORE/op/dbm.t
index 9e81f59..4ad165a 100644
--- a/t/CORE/op/dbm.t
+++ b/t/CORE/op/dbm.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 
     eval { require AnyDBM_File }; # not all places have dbm* functions
     skip_all("No dbm functions") if $@;
-- 
2.5.0

