From 225e0ec8a6f6f89ca02cf5c7662d69431a2b6721 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 167/398] B::C patch for op/defins.t

---
 t/CORE/op/defins.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/defins.t b/t/CORE/op/defins.t
index 54d3150..4802a2e 100644
--- a/t/CORE/op/defins.t
+++ b/t/CORE/op/defins.t
@@ -5,11 +5,10 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     $SIG{__WARN__} = sub { $warns++; warn $_[0] };
 }
-require 'test.pl';
+require 't/CORE/test.pl';
 plan( tests => 27 );
 
 my $unix_mode = 1;
-- 
2.5.0

