From 3ea222d43a50a0847e126cafedee23f0056896d7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 168/398] B::C patch for op/delete.t

---
 t/CORE/op/delete.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/delete.t b/t/CORE/op/delete.t
index 493717e..dec9003 100644
--- a/t/CORE/op/delete.t
+++ b/t/CORE/op/delete.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 require "test.pl";
-- 
2.5.0

