From 4fe0b49df291373878c2c45274beeb37edca4a92 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 169/398] B::C patch for op/die.t

---
 t/CORE/op/die.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/die.t b/t/CORE/op/die.t
index 8faef6a..6acdbbb 100644
--- a/t/CORE/op/die.t
+++ b/t/CORE/op/die.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 19;
-- 
2.5.0

