From 64e72f48519a01694614d50a26dad514290e085f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 170/398] B::C patch for op/die_except.t

---
 t/CORE/op/die_except.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/die_except.t b/t/CORE/op/die_except.t
index a65bf4a..22d046f 100644
--- a/t/CORE/op/die_except.t
+++ b/t/CORE/op/die_except.t
@@ -1,6 +1,6 @@
 #!./perl -w
 
-require './test.pl';
+require 't/CORE/test.pl';
 use strict;
 
 {
-- 
2.5.0

