From 351a26f850632320b5a0fb2e7972f5f812bf58c9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 171/398] B::C patch for op/die_exit.t

---
 t/CORE/op/die_exit.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/die_exit.t b/t/CORE/op/die_exit.t
index bd9ac28..89fa855 100644
--- a/t/CORE/op/die_exit.t
+++ b/t/CORE/op/die_exit.t
@@ -6,9 +6,8 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

