From 5eeb7027da23be8ecd913c2b86c09cf9be453f28 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 172/398] B::C patch for op/die_keeperr.t

---
 t/CORE/op/die_keeperr.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/die_keeperr.t b/t/CORE/op/die_keeperr.t
index 083bd5d..bb7bd08 100644
--- a/t/CORE/op/die_keeperr.t
+++ b/t/CORE/op/die_keeperr.t
@@ -1,10 +1,9 @@
 #!perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    require 'test.pl';
-    plan(24);
+    require 't/CORE/test.pl';
 }
+plan(24);
 
 sub End::DESTROY { $_[0]->() }
 
-- 
2.5.0

