From 59d6c114902d66117a3be8bf6b808c66a80f0efa Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 173/398] B::C patch for op/die_unwind.t

---
 t/CORE/op/die_unwind.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/die_unwind.t b/t/CORE/op/die_unwind.t
index 29fa6e6..27d088f 100644
--- a/t/CORE/op/die_unwind.t
+++ b/t/CORE/op/die_unwind.t
@@ -1,6 +1,6 @@
 #!./perl -w
 
-require './test.pl';
+require 't/CORE/test.pl';
 use strict;
 
 #
-- 
2.5.0

