From bd8fabe900d70694c51387f9ad82b770b61ba2c0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:50 -0500
Subject: [PATCH 174/398] B::C patch for op/do.t

---
 t/CORE/op/do.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/do.t b/t/CORE/op/do.t
index 012166e..d44693c 100644
--- a/t/CORE/op/do.t
+++ b/t/CORE/op/do.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 use strict;
 no warnings 'void';
-- 
2.5.0

