From 924286b5f1e79b3a05c0f235f1051180da256326 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 175/398] B::C patch for op/dor.t

---
 t/CORE/op/dor.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/dor.t b/t/CORE/op/dor.t
index a0b98f1..d06e981 100644
--- a/t/CORE/op/dor.t
+++ b/t/CORE/op/dor.t
@@ -3,12 +3,11 @@
 # Test // and friends.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
 package main;
-require './test.pl';
+require 't/CORE/test.pl';
 
 plan( tests => 34 );
 
-- 
2.5.0

