From 7b5e0362d37416fefeb1fb7d6169b98a359ce808 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 176/398] B::C patch for op/each.t

---
 t/CORE/op/each.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/each.t b/t/CORE/op/each.t
index be8aa48..e79e5be 100644
--- a/t/CORE/op/each.t
+++ b/t/CORE/op/each.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 59;
-- 
2.5.0

