From 0e2e8465f75b77b044131d0c389ced3d3d931f9b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 177/398] B::C patch for op/each_array.t

---
 t/CORE/op/each_array.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/each_array.t b/t/CORE/op/each_array.t
index 1055d6c..5db1446 100644
--- a/t/CORE/op/each_array.t
+++ b/t/CORE/op/each_array.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 use strict;
 use warnings;
-- 
2.5.0

