From 005bde72a3cc32e589a85ca867b8290d6470677f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 180/398] B::C patch for op/exec.t

---
 t/CORE/op/exec.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/exec.t b/t/CORE/op/exec.t
index 6ec3646..12d2602 100644
--- a/t/CORE/op/exec.t
+++ b/t/CORE/op/exec.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib');
-    require './test.pl';
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 my $vms_exit_mode = 0;
-- 
2.5.0

