From 779e1b73266a36595f8a05482c116ad07d82c820 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 181/398] B::C patch for op/exists_sub.t

---
 t/CORE/op/exists_sub.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/exists_sub.t b/t/CORE/op/exists_sub.t
index a08e0f5..39d3dc9 100644
--- a/t/CORE/op/exists_sub.t
+++ b/t/CORE/op/exists_sub.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 sub t1;
-- 
2.5.0

