From 21a171299afc73e0a91033c00e16ff0c790ca452 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 183/398] B::C patch for op/fh.t

---
 t/CORE/op/fh.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/fh.t b/t/CORE/op/fh.t
index afca57b..bbdff7a 100644
--- a/t/CORE/op/fh.t
+++ b/t/CORE/op/fh.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 8;
-- 
2.5.0

