From 9b2858a5c2a6c599ae108af29b143ecac8518151 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 184/398] B::C patch for op/filehandle.t

---
 t/CORE/op/filehandle.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/filehandle.t b/t/CORE/op/filehandle.t
index a7621db..77def41 100644
--- a/t/CORE/op/filehandle.t
+++ b/t/CORE/op/filehandle.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no IO, hence no FileHandle");
 }
 
-- 
2.5.0

