From 4f3d2ee070d53563e99f684375c7f6c11e62a024 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 186/398] B::C patch for op/filetest_stack_ok.t

---
 t/CORE/op/filetest_stack_ok.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/filetest_stack_ok.t b/t/CORE/op/filetest_stack_ok.t
index 4128612..903f96a 100644
--- a/t/CORE/op/filetest_stack_ok.t
+++ b/t/CORE/op/filetest_stack_ok.t
@@ -5,9 +5,8 @@
 # argument on the stack.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 my @ops = split //, 'rwxoRWXOezsfdlpSbctugkTMBAC';
-- 
2.5.0

