From acbaa5cd1a020c9ce22c9b967c72eb4ece834041 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:51 -0500
Subject: [PATCH 187/398] B::C patch for op/filetest_t.t

---
 t/CORE/op/filetest_t.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/filetest_t.t b/t/CORE/op/filetest_t.t
index eec67ec..fa28a4d 100644
--- a/t/CORE/op/filetest_t.t
+++ b/t/CORE/op/filetest_t.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

