From 6ea8158a5918be91096bfa22d526aaa74003ac50 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 188/398] B::C patch for op/flip.t

---
 t/CORE/op/flip.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/op/flip.t b/t/CORE/op/flip.t
index 8526db7..c1f873c 100644
--- a/t/CORE/op/flip.t
+++ b/t/CORE/op/flip.t
@@ -1,7 +1,7 @@
 #!./perl
 
 BEGIN {
-    require "test.pl";
+    require 't/CORE/test.pl';
 }
 
 plan(11);
@@ -23,7 +23,7 @@ is($y, '12E0123E0');
 {
 local $.;
 
-open(of,'harness') or die "Can't open harness: $!";
+open(of,'t/CORE/TEST') or die "Can't open harness: $!";
 while (<of>) {
     (3 .. 5) && ($foo .= $_);
 }
-- 
2.5.0

