From 1cf3d66d43f77faf6ba4d105b1ef5706f2ae01dd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 189/398] B::C patch for op/for.t

---
 t/CORE/op/for.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/for.t b/t/CORE/op/for.t
index 2ac0fc8..4dc6421 100644
--- a/t/CORE/op/for.t
+++ b/t/CORE/op/for.t
@@ -1,7 +1,7 @@
 #!./perl
 
 BEGIN {
-    require "test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan(106);
-- 
2.5.0

