From 68241b4796f1533a094cebe1c2cda9649e2e34d8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 190/398] B::C patch for op/fork.t

---
 t/CORE/op/fork.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/fork.t b/t/CORE/op/fork.t
index 60c991b..5db377f 100644
--- a/t/CORE/op/fork.t
+++ b/t/CORE/op/fork.t
@@ -3,9 +3,8 @@
 # tests for both real and emulated fork()
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     require Config;
     skip_all('no fork')
 	unless ($Config::Config{d_fork} or $Config::Config{d_pseudofork});
-- 
2.5.0

