From 28ccbd78031d3b5cfc2a5766ee516a584e633580 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 191/398] B::C patch for op/fresh_perl_utf8.t

---
 t/CORE/op/fresh_perl_utf8.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/fresh_perl_utf8.t b/t/CORE/op/fresh_perl_utf8.t
index fcf48e7..57f6811 100644
--- a/t/CORE/op/fresh_perl_utf8.t
+++ b/t/CORE/op/fresh_perl_utf8.t
@@ -3,9 +3,8 @@
 #This file is intentionally written in UTF-8
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan 1;
-- 
2.5.0

