From 6674adc23d15c3a3d4f64f192f31db2166710365 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 192/398] B::C patch for op/getppid.t

---
 t/CORE/op/getppid.t | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/getppid.t b/t/CORE/op/getppid.t
index a8d0f2c..93e6dcc 100644
--- a/t/CORE/op/getppid.t
+++ b/t/CORE/op/getppid.t
@@ -8,18 +8,18 @@
 # the same. (and not ours)
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib);
+    unshift @INC, 't/CORE/lib';
 }
 
 use strict;
 
 BEGIN {
-    require './test.pl';
-    skip_all_without_config(qw(d_pipe d_fork d_waitpid d_getppid));
-    plan (8);
+    require 't/CORE/test.pl';
 }
 
+skip_all_without_config(qw(d_pipe d_fork d_waitpid d_getppid));
+plan (8);
+
 # No, we don't want any zombies. kill 0, $ppid spots zombies :-(
 $SIG{CHLD} = 'IGNORE';
 
-- 
2.5.0

