From 67ce17e2857721c984ff323eae3bf5bd5e38e86b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 193/398] B::C patch for op/glob.t

---
 t/CORE/op/glob.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/glob.t b/t/CORE/op/glob.t
index e6f3680..e49a9f2 100644
--- a/t/CORE/op/glob.t
+++ b/t/CORE/op/glob.t
@@ -1,13 +1,12 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require 'test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan( tests => 18 );
 
+chdir 't/CORE';
 @oops = @ops = <op/*>;
 
 if ($^O eq 'MSWin32') {
-- 
2.5.0

