From 1232830461b727686f407bef5442491939e01b33 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 194/398] B::C patch for op/gmagic.t

---
 t/CORE/op/gmagic.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/gmagic.t b/t/CORE/op/gmagic.t
index bcf1322..e6f988c 100644
--- a/t/CORE/op/gmagic.t
+++ b/t/CORE/op/gmagic.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

