From c2e06bbf884ad29ae187ffb57f96da132729c5dc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 195/398] B::C patch for op/goto.t

---
 t/CORE/op/goto.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/goto.t b/t/CORE/op/goto.t
index 13e6b04..79b71f9 100644
--- a/t/CORE/op/goto.t
+++ b/t/CORE/op/goto.t
@@ -3,8 +3,7 @@
 # "This IS structured code.  It's just randomly structured."
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
 }
 
-- 
2.5.0

