From 3da4c5d37863c85b11a20c65fc2984ab89ab468f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 196/398] B::C patch for op/grent.t

---
 t/CORE/op/grent.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/grent.t b/t/CORE/op/grent.t
index 3b28619..14362a4 100644
--- a/t/CORE/op/grent.t
+++ b/t/CORE/op/grent.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 eval {my @n = getgrgid 0};
-- 
2.5.0

