From fa7ddc7966dd29955171695a0cbf37908d3b371d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 197/398] B::C patch for op/grep.t

---
 t/CORE/op/grep.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/grep.t b/t/CORE/op/grep.t
index d34686d..0c36e67 100644
--- a/t/CORE/op/grep.t
+++ b/t/CORE/op/grep.t
@@ -5,8 +5,7 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
 }
 
-- 
2.5.0

