From 0953cdf1382bc7f295ddb4eb9c23968e180955de Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:52 -0500
Subject: [PATCH 198/398] B::C patch for op/groups.t

---
 t/CORE/op/groups.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/groups.t b/t/CORE/op/groups.t
index dc2631e..fe87388 100644
--- a/t/CORE/op/groups.t
+++ b/t/CORE/op/groups.t
@@ -10,9 +10,8 @@ BEGIN {
     $ENV{LC_ALL} = "C"; # so that external utilities speak English
     $ENV{LANGUAGE} = 'C'; # GNU locale extension
 
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no POSIX");
 }
 use 5.010;
-- 
2.5.0

