From 0cedb01151895eaf183affb4d36a62e7bf39d609 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 199/398] B::C patch for op/gv.t

---
 t/CORE/op/gv.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/gv.t b/t/CORE/op/gv.t
index 6b90864..21634f0 100644
--- a/t/CORE/op/gv.t
+++ b/t/CORE/op/gv.t
@@ -5,11 +5,11 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
+chdir 't/CORE';
+
 use warnings;
 
 plan( tests => 271 );
-- 
2.5.0

