From 6d9c06c4b475670439098abf50e88d9a5363ddf2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 200/398] B::C patch for op/hash-rt85026.t

---
 t/CORE/op/hash-rt85026.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/hash-rt85026.t b/t/CORE/op/hash-rt85026.t
index 91087b8..7222824 100644
--- a/t/CORE/op/hash-rt85026.t
+++ b/t/CORE/op/hash-rt85026.t
@@ -1,9 +1,8 @@
 #!/usr/bin/perl -w
 
 BEGIN {
-  chdir 't';
-  @INC = '../lib';
-  require './test.pl';
+  push @INC, "t/CORE/lib";
+  require 't/CORE/test.pl';
   skip_all_without_dynamic_extension("Devel::Peek");
 }
 
-- 
2.5.0

