From db1042a753e1a68f49e38bc46dff61ac96be38ff Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 201/398] B::C patch for op/hash.t

---
 t/CORE/op/hash.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/hash.t b/t/CORE/op/hash.t
index 5f4c143..f65c471 100644
--- a/t/CORE/op/hash.t
+++ b/t/CORE/op/hash.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

