From 336daa93b0b120fb58a753b074e5859454af4b1f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 203/398] B::C patch for op/hashwarn.t

---
 t/CORE/op/hashwarn.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/hashwarn.t b/t/CORE/op/hashwarn.t
index b14e9c2..cec6ed4 100644
--- a/t/CORE/op/hashwarn.t
+++ b/t/CORE/op/hashwarn.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
-require 'test.pl';
+require 't/CORE/test.pl';
 plan( tests => 16 );
 
 use strict;
-- 
2.5.0

