From cd09bbab48fac92b61b77bfe7ca1dae515e7ea12 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 204/398] B::C patch for op/heredoc.t

---
 t/CORE/op/heredoc.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/heredoc.t b/t/CORE/op/heredoc.t
index a239e92..8bfbe98 100644
--- a/t/CORE/op/heredoc.t
+++ b/t/CORE/op/heredoc.t
@@ -1,9 +1,8 @@
 # tests for heredocs besides what is tested in base/lex.t
 
 BEGIN {
-   chdir 't' if -d 't';
-   @INC = '../lib';
-   require './test.pl';
+   push @INC, "t/CORE/lib";
+   require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

