From b8671f56d22b2d8bccec1435ff837c95d545fb60 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 205/398] B::C patch for op/inc.t

---
 t/CORE/op/inc.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/inc.t b/t/CORE/op/inc.t
index 8db0660..d612e94 100644
--- a/t/CORE/op/inc.t
+++ b/t/CORE/op/inc.t
@@ -1,6 +1,6 @@
 #!./perl -w
 
-require './test.pl';
+require 't/CORE/test.pl';
 use strict;
 
 # Tests of post/pre - increment/decrement operators.
-- 
2.5.0

