From 86566f75f6875045a14045d1f0dc46d0d23e7536 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 206/398] B::C patch for op/inccode-tie.t

---
 t/CORE/op/inccode-tie.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/inccode-tie.t b/t/CORE/op/inccode-tie.t
index 5bbcc10..8adef3f 100644
--- a/t/CORE/op/inccode-tie.t
+++ b/t/CORE/op/inccode-tie.t
@@ -6,7 +6,7 @@ use Tie::Array;
 my @orig_INC = @INC;
 tie @INC, 'Tie::StdArray';
 @INC = @orig_INC;
-for my $file ('./op/inccode.t', './t/op/inccode.t', ':op:inccode.t') {
+for my $file ('./t/CORE/op/inccode.t', './t/op/inccode.t', ':op:inccode.t') {
     if (-r $file) {
 	do $file; die $@ if $@;
 	exit;
-- 
2.5.0

