From 51ad8d9e66f70f0589d1838b244476eb65a84dbc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 207/398] B::C patch for op/inccode.t

---
 t/CORE/op/inccode.t | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/inccode.t b/t/CORE/op/inccode.t
index 1a0b919..e4b12e2 100644
--- a/t/CORE/op/inccode.t
+++ b/t/CORE/op/inccode.t
@@ -3,9 +3,8 @@
 # Tests for the coderef-in-@INC feature
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 use Config;
@@ -197,7 +196,7 @@ is( $ret, 'abc', 'do "abc.pl" sees return value' );
     my $got;
     #local @INC; # local fails on tied @INC
     my @old_INC = @INC; # because local doesn't work on tied arrays
-    @INC = ('lib', 'lib/Devel', sub { $got = $_[1]; return undef; });
+    @INC = ('t/CORE/lib', 'lib/Devel', sub { $got = $_[1]; return undef; });
     foreach my $filename ('/test_require.pm', './test_require.pm',
 			  '../test_require.pm') {
 	local %INC;
-- 
2.5.0

