From fba945ff6b1efadffd0dace80d1abdce62118ac1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 208/398] B::C patch for op/incfilter.t

---
 t/CORE/op/incfilter.t | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/incfilter.t b/t/CORE/op/incfilter.t
index c344558..4f59a5c 100644
--- a/t/CORE/op/incfilter.t
+++ b/t/CORE/op/incfilter.t
@@ -3,16 +3,16 @@
 # Tests for the source filters in coderef-in-@INC
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require 'test.pl';
-    skip_all_if_miniperl('no dynamic loading on miniperl, no Filter::Util::Call');
-    skip_all_without_perlio();
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 use strict;
 use Config;
 use Filter::Util::Call;
 
+skip_all_if_miniperl('no dynamic loading on miniperl, no Filter::Util::Call');
+skip_all_without_perlio();
+
 plan(tests => 153);
 
 unshift @INC, sub {
-- 
2.5.0

