From e06f9fd536694c870ed040b4960ce345d665c261 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 209/398] B::C patch for op/index.t

---
 t/CORE/op/index.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/index.t b/t/CORE/op/index.t
index eaed4b3..634c07f 100644
--- a/t/CORE/op/index.t
+++ b/t/CORE/op/index.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

