From 3411003d87de02035f745def4e3a272426cebf80 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:53 -0500
Subject: [PATCH 210/398] B::C patch for op/int.t

---
 t/CORE/op/int.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/int.t b/t/CORE/op/int.t
index 9aad020..bcea5a1 100644
--- a/t/CORE/op/int.t
+++ b/t/CORE/op/int.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan 15;
-- 
2.5.0

