From e06f2959823c37b35899ac9afb86937327b3fa5f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 212/398] B::C patch for op/kill0.t

---
 t/CORE/op/kill0.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/kill0.t b/t/CORE/op/kill0.t
index 7f6e6ec..14a48bf 100644
--- a/t/CORE/op/kill0.t
+++ b/t/CORE/op/kill0.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 BEGIN {
-- 
2.5.0

