From 79a119d192a8108a27be713d82b1a5a91cf432c1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 213/398] B::C patch for op/kvaslice.t

---
 t/CORE/op/kvaslice.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/kvaslice.t b/t/CORE/op/kvaslice.t
index e8aa14e..2334c1f 100644
--- a/t/CORE/op/kvaslice.t
+++ b/t/CORE/op/kvaslice.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 # use strict;
-- 
2.5.0

