From 732501791c47e7cddd218f48f1dd58becf40e7e6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 214/398] B::C patch for op/kvhslice.t

---
 t/CORE/op/kvhslice.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/kvhslice.t b/t/CORE/op/kvhslice.t
index 321d1c4..5b51bd9 100644
--- a/t/CORE/op/kvhslice.t
+++ b/t/CORE/op/kvhslice.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 # use strict;
-- 
2.5.0

