From 63ed99a20f55d72db9f9f1ff85e883daa70b8fd4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 215/398] B::C patch for op/lc.t

---
 t/CORE/op/lc.t | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/lc.t b/t/CORE/op/lc.t
index 38d2b6b..0cd9f18 100644
--- a/t/CORE/op/lc.t
+++ b/t/CORE/op/lc.t
@@ -3,11 +3,10 @@
 # This file is intentionally encoded in latin-1.
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
+    unshift @INC, 't/CORE/lib';
     require Config; import Config;
-    require './test.pl';
-    require './loc_tools.pl';   # Contains find_utf8_ctype_locale()
+    require 't/CORE/test.pl';
+    require 't/CORE/loc_tools.pl';
 }
 
 use feature qw( fc );
-- 
2.5.0

