From d9c0cd46ddd577d97e08151829cde1f8ef8bda9e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 216/398] B::C patch for op/leaky-magic.t

---
 t/CORE/op/leaky-magic.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/leaky-magic.t b/t/CORE/op/leaky-magic.t
index 371f3ac..7171a4b 100644
--- a/t/CORE/op/leaky-magic.t
+++ b/t/CORE/op/leaky-magic.t
@@ -5,9 +5,8 @@
 # in other packages (%Net::DNS::RR::SIG, ${"'Oh no'!"}, etc.).
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    @INC = '../lib';
+    require 't/CORE/test.pl';
+    push @INC, "t/CORE/lib";
 }
 
 # Hack to allow test counts to be specified piecemeal
-- 
2.5.0

