From 102cac98147cf73087281e6ade4c59236b930a02 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 217/398] B::C patch for op/length.t

---
 t/CORE/op/length.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/length.t b/t/CORE/op/length.t
index b144b09..5e84e51 100644
--- a/t/CORE/op/length.t
+++ b/t/CORE/op/length.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    require './test.pl';
-    @INC = '../lib';
+    require 't/CORE/test.pl';
+    push @INC, "t/CORE/lib";
 }
 
 plan (tests => 41);
-- 
2.5.0

