From dbf3e6a104b80e01329064fec59d5ceb33e72e99 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 218/398] B::C patch for op/lex.t

---
 t/CORE/op/lex.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/lex.t b/t/CORE/op/lex.t
index b33f0ef..79dfe11 100644
--- a/t/CORE/op/lex.t
+++ b/t/CORE/op/lex.t
@@ -2,7 +2,7 @@
 use strict;
 use warnings;
 
-BEGIN { chdir 't'; require './test.pl'; }
+BEGIN { require 't/CORE/test.pl' }
 
 plan(tests => 8);
 
-- 
2.5.0

