From 6330f2c0720e82c1c6e6afdada26a89c0309b7cc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 219/398] B::C patch for op/lex_assign.t

---
 t/CORE/op/lex_assign.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/lex_assign.t b/t/CORE/op/lex_assign.t
index 290023c..dd7dd07 100644
--- a/t/CORE/op/lex_assign.t
+++ b/t/CORE/op/lex_assign.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 $| = 1;
-- 
2.5.0

