From 50f2d557109290178bf9fea0ff975d915f908faa Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 220/398] B::C patch for op/lexsub.t

---
 t/CORE/op/lexsub.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/lexsub.t b/t/CORE/op/lexsub.t
index 8cfbab1..5ea1419 100644
--- a/t/CORE/op/lexsub.t
+++ b/t/CORE/op/lexsub.t
@@ -1,9 +1,8 @@
 #!perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     *bar::is = *is;
     *bar::like = *like;
 }
-- 
2.5.0

