From df0654d1e0508db5233d4c7d853472ed706f3049 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:54 -0500
Subject: [PATCH 222/398] B::C patch for op/list.t

---
 t/CORE/op/list.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/list.t b/t/CORE/op/list.t
index 74f256b..58507a3 100644
--- a/t/CORE/op/list.t
+++ b/t/CORE/op/list.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 require "test.pl";
-- 
2.5.0

