From f49e26860e891f24e4487156afd4a84adfccd20b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 223/398] B::C patch for op/local.t

---
 t/CORE/op/local.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/local.t b/t/CORE/op/local.t
index 03a8310..524764c 100644
--- a/t/CORE/op/local.t
+++ b/t/CORE/op/local.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 plan tests => 310;
 
-- 
2.5.0

