From 69a081c2f76d24949fdaa556970a289e1cc17060 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 224/398] B::C patch for op/localref.t

---
 t/CORE/op/localref.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/localref.t b/t/CORE/op/localref.t
index 3f49344..bcc406a 100644
--- a/t/CORE/op/localref.t
+++ b/t/CORE/op/localref.t
@@ -1,7 +1,6 @@
 #!./perl
 
-chdir 't' if -d 't';
-@INC = qw(. ../lib);
+push @INC, qw{t/CORE/lib};
 require "test.pl";
 plan( tests => 64 );
 
-- 
2.5.0

