From 144c652d2a35f4bb87b5c8ae4c3f0a5262095d8e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 225/398] B::C patch for op/lock.t

---
 t/CORE/op/lock.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/lock.t b/t/CORE/op/lock.t
index c40ec4c..49ec809 100644
--- a/t/CORE/op/lock.t
+++ b/t/CORE/op/lock.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require './test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 plan tests => 5;
 
-- 
2.5.0

