From cb0970f790c2d287419f3eb3c43152c94f03eb86 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 226/398] B::C patch for op/loopctl.t

---
 t/CORE/op/loopctl.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/loopctl.t b/t/CORE/op/loopctl.t
index d520a7f..d237b92 100644
--- a/t/CORE/op/loopctl.t
+++ b/t/CORE/op/loopctl.t
@@ -31,8 +31,7 @@
 #
 #  -- .robin. <robin@kitsite.com>  2001-03-13
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
 }
 
-- 
2.5.0

