From 6ffe383d648bb0bad078ce1c66090186573456ea Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 227/398] B::C patch for op/lop.t

---
 t/CORE/op/lop.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/lop.t b/t/CORE/op/lop.t
index bc4eb85..f60815f 100644
--- a/t/CORE/op/lop.t
+++ b/t/CORE/op/lop.t
@@ -5,9 +5,8 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 17;
-- 
2.5.0

