From 30aeb6fde87d2722f1a2008ad18f2e4d41b3e45a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 228/398] B::C patch for op/magic-27839.t

---
 t/CORE/op/magic-27839.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/op/magic-27839.t b/t/CORE/op/magic-27839.t
index 3e877a4..a6a7737 100644
--- a/t/CORE/op/magic-27839.t
+++ b/t/CORE/op/magic-27839.t
@@ -2,10 +2,10 @@
 
 BEGIN {
     $SIG{__WARN__} = sub { die "Dying on warning: ", @_ };
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no Tie::Hash::NamedCapture");
-    plan(tests => 2);
 }
+plan(tests => 2);
 
 use strict;
 
-- 
2.5.0

