From 7e3525dd249014054d3ae901a8f3c886d7dd49a7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 230/398] B::C patch for op/method.t

---
 t/CORE/op/method.t | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/method.t b/t/CORE/op/method.t
index 059b44f..a0e1b9d 100644
--- a/t/CORE/op/method.t
+++ b/t/CORE/op/method.t
@@ -5,14 +5,15 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib lib);
-    require "test.pl";
+    unshift @INC, 'lib';
+    require "t/CORE/test.pl";
 }
 
 use strict;
 no warnings 'once';
 
+chdir 't/CORE';
+
 plan(tests => 147);
 
 @A::ISA = 'B';
-- 
2.5.0

