From 5d173abc668e2b6bb8b93f7a4462e7122ecd4995 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 231/398] B::C patch for op/mkdir.t

---
 t/CORE/op/mkdir.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/mkdir.t b/t/CORE/op/mkdir.t
index d5c04b3..e772c7b 100644
--- a/t/CORE/op/mkdir.t
+++ b/t/CORE/op/mkdir.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 22;
-- 
2.5.0

