From 25c1236b2098699de990b009900a134496602a96 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 232/398] B::C patch for op/my.t

---
 t/CORE/op/my.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/my.t b/t/CORE/op/my.t
index 2dca46f..640fd98 100644
--- a/t/CORE/op/my.t
+++ b/t/CORE/op/my.t
@@ -1,8 +1,7 @@
 #!./perl
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 sub foo {
-- 
2.5.0

