From 392732d52c75da772eb9b2ee8aa9910322b8abc5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 233/398] B::C patch for op/my_stash.t

---
 t/CORE/op/my_stash.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/my_stash.t b/t/CORE/op/my_stash.t
index 6ec7619..d126ad9 100644
--- a/t/CORE/op/my_stash.t
+++ b/t/CORE/op/my_stash.t
@@ -3,9 +3,8 @@
 package Foo;
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan 9;
-- 
2.5.0

