From e5df82dd827874e0fb099dae46b463c69d9238b4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:55 -0500
Subject: [PATCH 234/398] B::C patch for op/mydef.t

---
 t/CORE/op/mydef.t | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/t/CORE/op/mydef.t b/t/CORE/op/mydef.t
index b993f1b..dfdc4bf 100644
--- a/t/CORE/op/mydef.t
+++ b/t/CORE/op/mydef.t
@@ -1,11 +1,12 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
+plan tests => 73;
+
 use strict;
 no warnings 'misc', 'experimental::lexical_topic';
 
@@ -153,7 +154,7 @@ $_ = "global";
     my $_ = "titi";
     my $r;
     {
-	local $::TODO = 'Marked as todo since test was added in 59f00321bbc2d046';
+    local $::TODO = 'Marked as todo since test was added in 59f00321bbc2d046';
 	$r = $s =~ /to(?{ is($_, 'toto', 'my $_ in code-match' ) })to/;
     }
     ok($r, "\$s=$s should match!");
@@ -179,7 +180,7 @@ my $file = tempfile();
     open my $_, '>', $file or die "Can't open $file: $!";
     print $_ "hello\n";
     close $_;
-    cmp_ok(-s $file, '>', 5, 'writing to filehandle $_ works');
+    ok( -s $file >  5, 'writing to filehandle $_ works');
 }
 {
     open my $_, $file or die "Can't open $file: $!";
@@ -215,4 +216,3 @@ my $file = tempfile();
     ::is($_, 4, '$_ is "special", and always forced to main');
 }
 
-done_testing();
-- 
2.5.0

