From 2ee31c2ba2f01008613754bbf61409217840deef Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 235/398] B::C patch for op/negate.t

---
 t/CORE/op/negate.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/negate.t b/t/CORE/op/negate.t
index 3b02e35..5cac846 100644
--- a/t/CORE/op/negate.t
+++ b/t/CORE/op/negate.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 46;
-- 
2.5.0

