From d5ce218932b414079a64b97aa3e195f5822632df Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 236/398] B::C patch for op/not.t

---
 t/CORE/op/not.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/not.t b/t/CORE/op/not.t
index 17f7a36..adad5f2 100644
--- a/t/CORE/op/not.t
+++ b/t/CORE/op/not.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 24;
-- 
2.5.0

