From 2ee31d5340e120c54b9f9e9d73c5b12c5bd94fc8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 237/398] B::C patch for op/numconvert.t

---
 t/CORE/op/numconvert.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/numconvert.t b/t/CORE/op/numconvert.t
index bfdb488..ff7eaf5 100644
--- a/t/CORE/op/numconvert.t
+++ b/t/CORE/op/numconvert.t
@@ -36,9 +36,8 @@
 # sub a5 { "$_[0]" }		# P
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

