From eb60a20e5185be2303c02153c2ea1e830db4a606 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 238/398] B::C patch for op/oct.t

---
 t/CORE/op/oct.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/oct.t b/t/CORE/op/oct.t
index d8caecd..c689363 100644
--- a/t/CORE/op/oct.t
+++ b/t/CORE/op/oct.t
@@ -2,7 +2,7 @@
 
 # Tests 51 onwards are intentionally not all-warnings-clean
 
-require './test.pl';
+require 't/CORE/test.pl';
 use strict;
 
 plan(tests => 77);
-- 
2.5.0

