From 813ac7c1c47b8d08aeea31300ad66a90f4a0c1c0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 239/398] B::C patch for op/or.t

---
 t/CORE/op/or.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/or.t b/t/CORE/op/or.t
index 9e7de4c..77dcf09 100644
--- a/t/CORE/op/or.t
+++ b/t/CORE/op/or.t
@@ -3,8 +3,7 @@
 # Test || in weird situations.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
 
@@ -23,7 +22,7 @@ sub FETCH {
 
 
 package main;
-require './test.pl';
+require 't/CORE/test.pl';
 
 plan( tests => 14 );
 
-- 
2.5.0

