From 326698bc5d17cac1a1b4e929576758fe77103f60 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 240/398] B::C patch for op/ord.t

---
 t/CORE/op/ord.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/ord.t b/t/CORE/op/ord.t
index 1c82262..2c960eb 100644
--- a/t/CORE/op/ord.t
+++ b/t/CORE/op/ord.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib); # ../lib needed for test.deparse
+    push @INC, qw{t/CORE/lib}; # ../lib needed for test.deparse
     require "test.pl";
 }
 
-- 
2.5.0

