From 758af48f735165e780f67d5febca29a1d7658863 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 241/398] B::C patch for op/overload_integer.t

---
 t/CORE/op/overload_integer.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/overload_integer.t b/t/CORE/op/overload_integer.t
index 2375ab9..8326700 100644
--- a/t/CORE/op/overload_integer.t
+++ b/t/CORE/op/overload_integer.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    push @INC, '../lib';
-    require './test.pl';
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

