From a1fb5efd2d38932cc282a98bca94d47337572e6e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 244/398] B::C patch for op/packagev.t

---
 t/CORE/op/packagev.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/packagev.t b/t/CORE/op/packagev.t
index f4e094c..d6fe815 100644
--- a/t/CORE/op/packagev.t
+++ b/t/CORE/op/packagev.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 # XXX remove this later -- dagolden, 2010-01-13
-- 
2.5.0

