From d2542f90dba12fc41c1368325e2ad5382bbbddfb Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:56 -0500
Subject: [PATCH 246/398] B::C patch for op/postfixderef.t

---
 t/CORE/op/postfixderef.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/postfixderef.t b/t/CORE/op/postfixderef.t
index e60b33b..cf7fdf5 100644
--- a/t/CORE/op/postfixderef.t
+++ b/t/CORE/op/postfixderef.t
@@ -9,9 +9,8 @@ this file contains all dereferencing tests from ref.t but using postfix instead
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require 'test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 
 use strict qw(refs subs);
-- 
2.5.0

