From 94ba1c54d7b7ec5cc08e0937b2fb22341d967a55 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 247/398] B::C patch for op/pow.t

---
 t/CORE/op/pow.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/pow.t b/t/CORE/op/pow.t
index 4cfeed1..277eb55 100644
--- a/t/CORE/op/pow.t
+++ b/t/CORE/op/pow.t
@@ -2,9 +2,8 @@
 # Now they'll be wanting biff! and zap! tests too.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 # This calculation ought to be within 0.001 of the right answer.
-- 
2.5.0

