From 3b291dc8cf61163d523ab3c9ba4ab95beba3d8cd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 248/398] B::C patch for op/print.t

---
 t/CORE/op/print.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/print.t b/t/CORE/op/print.t
index a98b0bf..9bdd0d8 100644
--- a/t/CORE/op/print.t
+++ b/t/CORE/op/print.t
@@ -1,7 +1,7 @@
 #!./perl
 
 BEGIN {
-    require "test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan(3);
-- 
2.5.0

