From e7afc91603683671bab1fa561a8fa4ecd8508af2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 251/398] B::C patch for op/pwent.t

---
 t/CORE/op/pwent.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/pwent.t b/t/CORE/op/pwent.t
index 7562bc0..071984d 100644
--- a/t/CORE/op/pwent.t
+++ b/t/CORE/op/pwent.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

