From e13de43b4f79db22af2cb76fffece504eb04cf73 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 252/398] B::C patch for op/qr.t

---
 t/CORE/op/qr.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/qr.t b/t/CORE/op/qr.t
index ac017eb..6469498 100644
--- a/t/CORE/op/qr.t
+++ b/t/CORE/op/qr.t
@@ -3,8 +3,7 @@
 use strict;
 
 BEGIN {
-    chdir 't';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan(tests => 32);
-- 
2.5.0

