From 5ddb0adb75bb93d2821de377d8e6bdf1cf32b7ec Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 253/398] B::C patch for op/quotemeta.t

---
 t/CORE/op/quotemeta.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/quotemeta.t b/t/CORE/op/quotemeta.t
index 1b8613a..6dbb173 100644
--- a/t/CORE/op/quotemeta.t
+++ b/t/CORE/op/quotemeta.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib .);
+    unshift @INC, 't/CORE/lib';
     require Config; import Config;
     require "test.pl";
 }
-- 
2.5.0

