From 14b2d409db7a622fceab7ae86f35bc0d09d776d5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 254/398] B::C patch for op/rand.t

---
 t/CORE/op/rand.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/rand.t b/t/CORE/op/rand.t
index 90d1c37..c38e3fc 100644
--- a/t/CORE/op/rand.t
+++ b/t/CORE/op/rand.t
@@ -16,8 +16,7 @@
 # chapter 3. ISBN 0-201-03822-6 (v. 2)
 
 BEGIN {
-    chdir "t" if -d "t";
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 use strict;
-- 
2.5.0

