From 1ac8595b3a4634c748d12a18d86c879132c7b370 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 255/398] B::C patch for op/range.t

---
 t/CORE/op/range.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/range.t b/t/CORE/op/range.t
index 903cdee..bc90287 100644
--- a/t/CORE/op/range.t
+++ b/t/CORE/op/range.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib', '.');
+    unshift @INC, 't/CORE/lib';
 }   
 # Avoid using eq_array below as it uses .. internally.
-require 'test.pl';
+require 't/CORE/test.pl';
 
 use Config;
 
-- 
2.5.0

