From 3a249d6d0615048f93d2b37cf482ad411de86d78 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 256/398] B::C patch for op/read.t

---
 t/CORE/op/read.t | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/t/CORE/op/read.t b/t/CORE/op/read.t
index 23f1b51..1597f89 100644
--- a/t/CORE/op/read.t
+++ b/t/CORE/op/read.t
@@ -1,14 +1,14 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 use strict;
+use Config ();
 
 plan tests => 2564;
 
+chdir 't/CORE';
 open(FOO,'op/read.t') || open(FOO,'t/op/read.t') || open(FOO,':op:read.t') || die "Can't open op.read";
 seek(FOO,4,0) or die "Seek failed: $!";
 my $buf;
@@ -23,13 +23,7 @@ $got = read(FOO,$buf,4);
 is ($got, 0);
 is ($buf, "");
 
-# This is true if Config is not built, or if PerlIO is enabled
-# ie assume that PerlIO is present, unless we know for sure otherwise.
-my $has_perlio = !eval {
-    no warnings;
-    require Config;
-    !$Config::Config{useperlio}
-};
+my $has_perlio = 1;
 
 my $tmpfile = tempfile();
 
-- 
2.5.0

