From f52f8953b22585f7633ba075d465c0951c23a31a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 257/398] B::C patch for op/readdir.t

---
 t/CORE/op/readdir.t | 17 ++++-------------
 1 file changed, 4 insertions(+), 13 deletions(-)

diff --git a/t/CORE/op/readdir.t b/t/CORE/op/readdir.t
index 8515d1d..c71bafc 100644
--- a/t/CORE/op/readdir.t
+++ b/t/CORE/op/readdir.t
@@ -1,15 +1,15 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use strict;
 use warnings;
 use vars qw($fh @fh %fh);
 
+chdir 't/CORE';
+
 eval 'opendir(NOSUCH, "no/such/directory");';
 skip_all($@) if $@;
 
@@ -23,16 +23,7 @@ is(opendir(OP, "op"), 1);
 my @D = grep(/^[^\.].*\.t$/i, readdir(OP));
 closedir(OP);
 
-my $expect;
-{
-    open my $man, '<', '../MANIFEST' or die "Can't open ../MANIFEST: $!";
-    while (<$man>) {
-	++$expect if m!^t/op/[^/]+\t!;
-    }
-}
-
-my ($min, $max) = ($expect - 10, $expect + 10);
-within(scalar @D, $expect, 10, 'counting op/*.t');
+within(scalar @D, 180, 200, 'counting op/*.t');
 
 my @R = sort @D;
 my @G = sort <op/*.t>;
-- 
2.5.0

