From ad565a903a3297695023d82873850c163bf9c461 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:57 -0500
Subject: [PATCH 258/398] B::C patch for op/readline.t

---
 t/CORE/op/readline.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/readline.t b/t/CORE/op/readline.t
index 1cfd78c..5ab5564 100644
--- a/t/CORE/op/readline.t
+++ b/t/CORE/op/readline.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 30;
-- 
2.5.0

