From 247b094f4448982952e55689fea5134040e52d74 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 259/398] B::C patch for op/recurse.t

---
 t/CORE/op/recurse.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/recurse.t b/t/CORE/op/recurse.t
index 2a0201a..9955302 100644
--- a/t/CORE/op/recurse.t
+++ b/t/CORE/op/recurse.t
@@ -5,12 +5,11 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
-    plan(tests => 28);
 }
 
+plan(tests => 28);
 use strict;
 
 sub gcd {
-- 
2.5.0

