From 31609606075c510e809fd0a7af7960c033af7e7c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 260/398] B::C patch for op/ref.t

---
 t/CORE/op/ref.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/ref.t b/t/CORE/op/ref.t
index a6564ce..8f053c3 100644
--- a/t/CORE/op/ref.t
+++ b/t/CORE/op/ref.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require 'test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 
 use strict qw(refs subs);
-- 
2.5.0

