From 3d608869af1706abb03fec5f81efe04baf60a5ab Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 261/398] B::C patch for op/repeat.t

---
 t/CORE/op/repeat.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/repeat.t b/t/CORE/op/repeat.t
index bfd142f..8d27dd9 100644
--- a/t/CORE/op/repeat.t
+++ b/t/CORE/op/repeat.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 
-require './test.pl';
+require 't/CORE/test.pl';
 plan(tests => 43);
 
 # compile time
-- 
2.5.0

