From 0facb47e46b8b04490b3d904e6260a89f547db3e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 262/398] B::C patch for op/require_37033.t

---
 t/CORE/op/require_37033.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/op/require_37033.t b/t/CORE/op/require_37033.t
index dac8568..af83de9 100644
--- a/t/CORE/op/require_37033.t
+++ b/t/CORE/op/require_37033.t
@@ -4,8 +4,8 @@ use strict;
 # Check that require doesn't leave the handle it uses open, if it happens that
 # the handle it opens gets file descriptor 0. RT #37033.
 
-require './test.pl';
-@INC = 'lib';
+require 't/CORE/test.pl';
+unshift @INC, 't/CORE/lib';
 
 sub test_require {
     my ($state, $want) = @_;
-- 
2.5.0

