From c3c7884a4b0b388faaab41a4d3aeca1922dcd341 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 264/398] B::C patch for op/reset.t

---
 t/CORE/op/reset.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/reset.t b/t/CORE/op/reset.t
index e592430..a967b51 100644
--- a/t/CORE/op/reset.t
+++ b/t/CORE/op/reset.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 use strict;
 
-- 
2.5.0

