From a6586446dd251e7e328ab53fd4c6ee9cf55f5c17 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 266/398] B::C patch for op/rt119311.t

---
 t/CORE/op/rt119311.t | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/t/CORE/op/rt119311.t b/t/CORE/op/rt119311.t
index 2ad7a9c..7356ef2 100644
--- a/t/CORE/op/rt119311.t
+++ b/t/CORE/op/rt119311.t
@@ -1,5 +1,9 @@
 #!perl
 
+BEGIN {
+    require 't/CORE/test.pl';
+}
+
 # Complicated enough to get its own test file.
 
 # When a subroutine is called recursively, it gets a new pad indexed by its
@@ -22,7 +26,6 @@
 # outer sub.  Then we test a lexical directly inside the sub that DESTROY
 # calls.  Then we repeat with formats.
 
-BEGIN { chdir 't'; require './test.pl' }
 plan 22;
 
 sub foo {
-- 
2.5.0

