From 0eaa0a943ebf5ae05776d93ae9f82cddb60eac2c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 267/398] B::C patch for op/runlevel.t

---
 t/CORE/op/runlevel.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/runlevel.t b/t/CORE/op/runlevel.t
index 3e68a23..3157140 100644
--- a/t/CORE/op/runlevel.t
+++ b/t/CORE/op/runlevel.t
@@ -6,9 +6,8 @@
 ## Adapted and expanded by Gurusamy Sarathy <gsar@activestate.com>
 ##
 
-chdir 't' if -d 't';
-@INC = '../lib';
-require './test.pl';
+push @INC, "t/CORE/lib";
+require 't/CORE/test.pl';
 
 $|=1;
 
-- 
2.5.0

