From 804e681eef9c72bac737404f0743ef0f05ac3887 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 268/398] B::C patch for op/select.t

---
 t/CORE/op/select.t | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/select.t b/t/CORE/op/select.t
index 1994dd2..1595036 100644
--- a/t/CORE/op/select.t
+++ b/t/CORE/op/select.t
@@ -1,14 +1,14 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan reverse 9;
 
+my $test_pl = 't/CORE/test.pl';
 
-open my $fh, "test.pl" or die "$0 unfortunately cannot open test.pl: $!";
+open my $fh, $test_pl or die "$0 unfortunately cannot open $test_pl: $!";
 
 is select, 'main::STDOUT', 'select retval';
 is select($fh), 'main::STDOUT', 'select retval when called with argument';
@@ -18,7 +18,7 @@ is select(STDOUT), $fh, 'select previous ref when setting to bareword';
 is select, 'main::STDOUT', 'switching back to STDOUT';
 is ref\select, 'SCALAR', 'and STDOUT is a plain string';
 
-open foo::bar, "test.pl" or die "$0 sadly cannot open test.pl: $!";
+open foo::bar, "$test_pl" or die "$0 sadly cannot open $test_pl: $!";
 select foo::bar;
 $handle = \*foo::bar;
 $stash = \%foo::;
@@ -26,7 +26,7 @@ $stash = \%foo::;
 is select, $handle,
     'select returns ref for glob whose stash has been detached';
 
-open thwat::snin, "test.pl" or die "$0 is unable to open test.pl: $!";
+open thwat::snin, "$test_pl" or die "$0 is unable to open $test_pl: $!";
 select thwat::snin;
 $handle = \*thwat::snin;
 *thwat:: = *snin::; # gv is now *__ANON__::snin
-- 
2.5.0

