From 121732e0c11c208d32101dcadadef3b11cbfb28d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 269/398] B::C patch for op/setpgrpstack.t

---
 t/CORE/op/setpgrpstack.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/setpgrpstack.t b/t/CORE/op/setpgrpstack.t
index b384138..66985ba 100644
--- a/t/CORE/op/setpgrpstack.t
+++ b/t/CORE/op/setpgrpstack.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_without_config('d_setpgrp');
 }
 
-- 
2.5.0

