From b0ab67f29a5cff07257cfbdf5cb08f406cb4a6b3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:58 -0500
Subject: [PATCH 270/398] B::C patch for op/sigdispatch.t

---
 t/CORE/op/sigdispatch.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/sigdispatch.t b/t/CORE/op/sigdispatch.t
index 12b2d5e..e99a978 100644
--- a/t/CORE/op/sigdispatch.t
+++ b/t/CORE/op/sigdispatch.t
@@ -3,7 +3,7 @@
 # We assume that TestInit has been used.
 
 BEGIN {
-      require './test.pl';
+      require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

