From cdbd58f54fe87c68ba24900dcb34c92558b437e0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 271/398] B::C patch for op/signatures.t

---
 t/CORE/op/signatures.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/signatures.t b/t/CORE/op/signatures.t
index 99f075f..a82a068 100644
--- a/t/CORE/op/signatures.t
+++ b/t/CORE/op/signatures.t
@@ -1,9 +1,8 @@
 #!perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 plan 912;
 
-- 
2.5.0

