From 3524199c6908b01f8196689e1e6c5603f84c334b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 272/398] B::C patch for op/sigsystem.t

---
 t/CORE/op/sigsystem.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/sigsystem.t b/t/CORE/op/sigsystem.t
index ddfebf9..1c612ee 100644
--- a/t/CORE/op/sigsystem.t
+++ b/t/CORE/op/sigsystem.t
@@ -1,7 +1,7 @@
 #!perl -w
 
 BEGIN {
-      require './test.pl';
+      require 't/CORE/test.pl';
       skip_all_if_miniperl();
       skip_all_without_config(qw(d_fork));
 }
-- 
2.5.0

