From 7b53ef0ffa83b8443480eaeb6148123979000cab Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 273/398] B::C patch for op/sleep.t

---
 t/CORE/op/sleep.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/sleep.t b/t/CORE/op/sleep.t
index 3f5bbe0..b3ff5ba 100644
--- a/t/CORE/op/sleep.t
+++ b/t/CORE/op/sleep.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
 }
 
 require "test.pl";
-- 
2.5.0

