From 8b737eccb9a2cedd97be315b3fd223b76404988c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 274/398] B::C patch for op/smartkve.t

---
 t/CORE/op/smartkve.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/smartkve.t b/t/CORE/op/smartkve.t
index 85eb9e8..a7c0010 100644
--- a/t/CORE/op/smartkve.t
+++ b/t/CORE/op/smartkve.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 use strict;
 use warnings;
-- 
2.5.0

