From ee0226ba42743120fad51d9d8fde2ebe30e9afd8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 275/398] B::C patch for op/smartmatch.t

---
 t/CORE/op/smartmatch.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/smartmatch.t b/t/CORE/op/smartmatch.t
index ed4b3ec..e8bbbb5 100644
--- a/t/CORE/op/smartmatch.t
+++ b/t/CORE/op/smartmatch.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 use strict;
 use warnings;
-- 
2.5.0

