From 91b08dee7881eead6758bd242732f04330a9c598 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 276/398] B::C patch for op/sort.t

---
 t/CORE/op/sort.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sort.t b/t/CORE/op/sort.t
index 151e3ea..4164fe7 100644
--- a/t/CORE/op/sort.t
+++ b/t/CORE/op/sort.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
-    require 'test.pl';
+    push @INC, qw{t/CORE/lib};
+    require 't/CORE/test.pl';
 }
 use warnings;
 plan( tests => 183 );
-- 
2.5.0

