From fe984438a3c4a8ba035eab955bd7875ac5e1bd1d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 278/398] B::C patch for op/split.t

---
 t/CORE/op/split.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/split.t b/t/CORE/op/split.t
index 7e0008e..3d1455a 100644
--- a/t/CORE/op/split.t
+++ b/t/CORE/op/split.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 119;
-- 
2.5.0

