From 632f15fd6905fe3835f638142eb93a90a5b7cd2a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 279/398] B::C patch for op/split_unicode.t

---
 t/CORE/op/split_unicode.t | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/split_unicode.t b/t/CORE/op/split_unicode.t
index bb50620..21c5fff 100644
--- a/t/CORE/op/split_unicode.t
+++ b/t/CORE/op/split_unicode.t
@@ -1,11 +1,12 @@
 #!./perl
 
 BEGIN {
-    require './test.pl';
-    skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec (used by charnames)");
-    plan(tests => 145);
+    require 't/CORE/test.pl';
 }
 
+skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec (used by charnames)");
+plan(tests => 145);
+
 {
     # check the special casing of split /\s/ and unicode
     use charnames qw(:full);
-- 
2.5.0

