From af494d9cd48296be5f40db36b0ba3e8e16ab0690 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 280/398] B::C patch for op/sprintf.t

---
 t/CORE/op/sprintf.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sprintf.t b/t/CORE/op/sprintf.t
index 17e269a..cef4148 100644
--- a/t/CORE/op/sprintf.t
+++ b/t/CORE/op/sprintf.t
@@ -9,14 +9,13 @@
 # For tests that do not fit this format, use sprintf2.t.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 use warnings;
 use version;
 use Config;
 use strict;
-require './test.pl';
+require 't/CORE/test.pl';
 
 my @tests = ();
 my ($template, $data, $result, $comment, $w, $x, $evalData, $n, $p);
-- 
2.5.0

