From c6a66301e9ce03c443a3bf678c79024de0b1ec7e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:21:59 -0500
Subject: [PATCH 281/398] B::C patch for op/sprintf2.t

---
 t/CORE/op/sprintf2.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sprintf2.t b/t/CORE/op/sprintf2.t
index d914de0..3a7e546 100644
--- a/t/CORE/op/sprintf2.t
+++ b/t/CORE/op/sprintf2.t
@@ -3,9 +3,8 @@
 # Tests for sprintf that do not fit the format of sprintf.t.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }   
 
 plan tests => 1370;
-- 
2.5.0

