From 9e628639a19f56d2b2b793d317d452345a0cb312 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 283/398] B::C patch for op/sselect.t

---
 t/CORE/op/sselect.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sselect.t b/t/CORE/op/sselect.t
index 879c9d5..76c6823 100644
--- a/t/CORE/op/sselect.t
+++ b/t/CORE/op/sselect.t
@@ -2,12 +2,11 @@
 
 my $hires;
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('.', '../lib');
+    unshift @INC, 't/CORE/lib';
     $hires = eval 'use Time::HiResx "time"; 1';
 }
 
-require 'test.pl';
+require 't/CORE/test.pl';
 
 plan (15);
 
-- 
2.5.0

