From 5236150773e697049989e83367a1db149270e53e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 284/398] B::C patch for op/stash.t

---
 t/CORE/op/stash.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/stash.t b/t/CORE/op/stash.t
index f2e5523..00a4a0f 100644
--- a/t/CORE/op/stash.t
+++ b/t/CORE/op/stash.t
@@ -1,11 +1,10 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib);
+    unshift @INC, 't/CORE/lib';
 }
 
-BEGIN { require "./test.pl"; }
+BEGIN { require 't/CORE/test.pl'; }
 
 plan( tests => 58 );
 
-- 
2.5.0

