From e887d631ea1b86787a358a9e98a9a16ac53a83b9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 286/398] B::C patch for op/state.t

---
 t/CORE/op/state.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/state.t b/t/CORE/op/state.t
index 81e5486..67e8721 100644
--- a/t/CORE/op/state.t
+++ b/t/CORE/op/state.t
@@ -2,9 +2,8 @@
 # tests state variables
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

