From 1da5e9f065515779ed8a9ca4dfc521dfa2ee80c2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 289/398] B::C patch for op/sub.t

---
 t/CORE/op/sub.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sub.t b/t/CORE/op/sub.t
index 7df8f49..d54e7c3 100644
--- a/t/CORE/op/sub.t
+++ b/t/CORE/op/sub.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan( tests => 33 );
-- 
2.5.0

