From 1f87f0dce413a8ed96291c81706d60e16b68ef80 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 291/398] B::C patch for op/substr.t

---
 t/CORE/op/substr.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/substr.t b/t/CORE/op/substr.t
index 8946759..3a16308 100644
--- a/t/CORE/op/substr.t
+++ b/t/CORE/op/substr.t
@@ -3,8 +3,7 @@
 #P = start of string  Q = start of substr  R = end of substr  S = end of string
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    push @INC, "t/CORE/lib";
 }
 use warnings ;
 
@@ -21,7 +20,7 @@ $SIG{__WARN__} = sub {
      }
 };
 
-BEGIN { require './test.pl'; }
+BEGIN { require 't/CORE/test.pl'; }
 
 plan(387);
 
-- 
2.5.0

