From 3f1f9cc069af8ed0c89c31efb1d403e3d68d6347 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 292/398] B::C patch for op/svleak.t

---
 t/CORE/op/svleak.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/svleak.t b/t/CORE/op/svleak.t
index 74a8338..690709c 100644
--- a/t/CORE/op/svleak.t
+++ b/t/CORE/op/svleak.t
@@ -5,9 +5,8 @@
 # see if the count increases.
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 
     eval { require XS::APItest; XS::APItest->import('sv_count'); 1 }
 	or skip_all("XS::APItest not available");
-- 
2.5.0

