From da51ba0da4f8605890bd57b1bc5ce8ec754d26fe Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:00 -0500
Subject: [PATCH 293/398] B::C patch for op/switch.t

---
 t/CORE/op/switch.t | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/switch.t b/t/CORE/op/switch.t
index 204a57a..0e9ea6c 100644
--- a/t/CORE/op/switch.t
+++ b/t/CORE/op/switch.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use strict;
@@ -440,7 +438,7 @@ sub check_outside1 { is($_, "inside", "\$_ is not lexically scoped") }
 
 {
     my ($ok_d, $ok_f, $ok_r);
-    given("op") {
+    given("t/CORE/op") {
 	when(-d)  {$ok_d = 1; continue}
 	when(!-f) {$ok_f = 1; continue}
 	when(-r)  {$ok_r = 1; continue}
-- 
2.5.0

