From 7b287373a09403ceb0eba5004c332d8771e0502a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 294/398] B::C patch for op/symbolcache.t

---
 t/CORE/op/symbolcache.t | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/t/CORE/op/symbolcache.t b/t/CORE/op/symbolcache.t
index b3e567b..0a4320c 100644
--- a/t/CORE/op/symbolcache.t
+++ b/t/CORE/op/symbolcache.t
@@ -1,14 +1,15 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-    plan( tests => 8 );
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
+plan( tests => 8 );
+
 use strict;
 
+eval q|
 # first, with delete
 # simple removal
 sub removed { 23 }
@@ -40,3 +41,5 @@ is( replaced2(), 'meth', 'original function not bound, was replaced' );
 ok( main->replaced2 eq 'meth', 'method is replaced function' );
 BEGIN { undef $main::{replaced2} }
 sub replaced2 { 'meth' }
+
+|;
\ No newline at end of file
-- 
2.5.0

