From ec57905c8b55e944d9db7dc9d85c352033535ede Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 295/398] B::C patch for op/sysio.t

---
 t/CORE/op/sysio.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/sysio.t b/t/CORE/op/sysio.t
index 25d7197..fcf03e9 100644
--- a/t/CORE/op/sysio.t
+++ b/t/CORE/op/sysio.t
@@ -1,13 +1,13 @@
 #!./perl
 
 BEGIN {
-  chdir 't' if -d 't';
-  @INC = '../lib';
-  require './test.pl';
+  push @INC, "t/CORE/lib";
+  require 't/CORE/test.pl';
 }
 
 plan tests => 48;
 
+chdir 't/CORE';
 open(I, 'op/sysio.t') || die "sysio.t: cannot find myself: $!";
 
 $reopen = ($^O eq 'VMS' ||
-- 
2.5.0

