From d251470b36f31066714a630e668a95afa9eb96c5 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 296/398] B::C patch for op/tie.t

---
 t/CORE/op/tie.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/tie.t b/t/CORE/op/tie.t
index e0b2499..148ab6d 100644
--- a/t/CORE/op/tie.t
+++ b/t/CORE/op/tie.t
@@ -9,9 +9,8 @@
 # Warn or die msgs (if any) at - line 1234
 #
 
-chdir 't' if -d 't';
-@INC = '../lib';
-require './test.pl';
+push @INC, "t/CORE/lib";
+require 't/CORE/test.pl';
 
 $|=1;
 
-- 
2.5.0

