From 2b7a37ada82bc33691a5078f7dae186fecc56660 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 297/398] B::C patch for op/tie_fetch_count.t

---
 t/CORE/op/tie_fetch_count.t | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/tie_fetch_count.t b/t/CORE/op/tie_fetch_count.t
index 9f60bb4..e6aa66a 100644
--- a/t/CORE/op/tie_fetch_count.t
+++ b/t/CORE/op/tie_fetch_count.t
@@ -4,12 +4,11 @@
 # See Bugs #76814 and #87708.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-    plan (tests => 312);
+    require 't/CORE/test.pl';
 }
 
+plan (tests => 312);
+
 use strict;
 use warnings;
 
@@ -144,7 +143,7 @@ for (split //, 'rwxoRWXOezsfdpSbctugkTBMAC') {
     check_count "-$_ \\\$tied_glob";
 }
 $dummy  = -l $var       ; check_count '-l';
-$var = "test.pl";
+$var = "t/CORE/test.pl";
 $dummy  = -e -e -e $var ; check_count '-e -e';
 
 # Matching
-- 
2.5.0

