From b12506ed1ec8e526d498592a0b5de061f3581fd6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 298/398] B::C patch for op/tiearray.t

---
 t/CORE/op/tiearray.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/tiearray.t b/t/CORE/op/tiearray.t
index 99b7938..aba8d1c 100644
--- a/t/CORE/op/tiearray.t
+++ b/t/CORE/op/tiearray.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 my %seen;
-- 
2.5.0

