From b6c49a4e7b0370cf891a21c1cba1139a7072ddf9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 300/398] B::C patch for op/time.t

---
 t/CORE/op/time.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/time.t b/t/CORE/op/time.t
index 5cb9191..49a20ab 100644
--- a/t/CORE/op/time.t
+++ b/t/CORE/op/time.t
@@ -1,9 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 72;
-- 
2.5.0

