From 11c30681909a621b8af222c37ca68591e939bd3a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:01 -0500
Subject: [PATCH 305/398] B::C patch for op/unlink.t

---
 t/CORE/op/unlink.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/unlink.t b/t/CORE/op/unlink.t
index 7602149..0e9c4e4 100644
--- a/t/CORE/op/unlink.t
+++ b/t/CORE/op/unlink.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan 6;
-- 
2.5.0

