From 163acc836f36b8bb6c31997fcd382358eb8885d1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 306/398] B::C patch for op/unshift.t

---
 t/CORE/op/unshift.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/op/unshift.t b/t/CORE/op/unshift.t
index a6cd692..4434439 100644
--- a/t/CORE/op/unshift.t
+++ b/t/CORE/op/unshift.t
@@ -1,7 +1,7 @@
 #!./perl
 
 BEGIN {
-    require "test.pl";
+    require "t/CORE/test.pl";
 }
 
 plan(36);
-- 
2.5.0

