From c781899c590ebe1b3380fb8f5095648c7428ef06 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 307/398] B::C patch for op/upgrade.t

---
 t/CORE/op/upgrade.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/upgrade.t b/t/CORE/op/upgrade.t
index 5f2ffa6..df0237c 100644
--- a/t/CORE/op/upgrade.t
+++ b/t/CORE/op/upgrade.t
@@ -5,9 +5,8 @@
 # build or testsuite that upgraded an NV to an RV
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

