From bffcba7c4900e55e5071acefd9352b191562350e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 308/398] B::C patch for op/utf8cache.t

---
 t/CORE/op/utf8cache.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utf8cache.t b/t/CORE/op/utf8cache.t
index 65254b1..b360c1b 100644
--- a/t/CORE/op/utf8cache.t
+++ b/t/CORE/op/utf8cache.t
@@ -2,9 +2,8 @@
 # Test for malfunctions of utf8 cache
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

