From d287f725e8c817b92ac1d23b9802141cbb69311c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 309/398] B::C patch for op/utf8decode.t

---
 t/CORE/op/utf8decode.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utf8decode.t b/t/CORE/op/utf8decode.t
index 92de821..b7920b9 100644
--- a/t/CORE/op/utf8decode.t
+++ b/t/CORE/op/utf8decode.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 {
-- 
2.5.0

