From 27398afac05e29523041dc12e559081bcd71877d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 310/398] B::C patch for op/utf8magic.t

---
 t/CORE/op/utf8magic.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utf8magic.t b/t/CORE/op/utf8magic.t
index 55e921d..986c15c 100644
--- a/t/CORE/op/utf8magic.t
+++ b/t/CORE/op/utf8magic.t
@@ -1,9 +1,8 @@
 #!perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 6;
-- 
2.5.0

