From f69f89e4676bfe29075bee177f5c6633d460d8dd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 311/398] B::C patch for op/utfhash.t

---
 t/CORE/op/utfhash.t | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/t/CORE/op/utfhash.t b/t/CORE/op/utfhash.t
index ebb2934..ac18627 100644
--- a/t/CORE/op/utfhash.t
+++ b/t/CORE/op/utfhash.t
@@ -1,13 +1,11 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-
-    plan(tests => 99);
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';    
 }
 
+plan(tests => 99);
 use strict;
 
 # Two hashes one will all keys 8-bit possible (initially), other
-- 
2.5.0

