From aa9cb8eaf49da1baf1360b21797fc7dbad19fe23 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 312/398] B::C patch for op/utftaint.t

---
 t/CORE/op/utftaint.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/utftaint.t b/t/CORE/op/utftaint.t
index d734927..a078341 100644
--- a/t/CORE/op/utftaint.t
+++ b/t/CORE/op/utftaint.t
@@ -2,8 +2,8 @@
 # tests whether tainting works with UTF-8
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib);
+	unshift @INC, 't/CORE';
+	require 'test.pl';
 }
 
 use strict;
@@ -17,7 +17,6 @@ sub tainted ($) {
     any_tainted @_;
 }
 
-require './test.pl';
 plan(tests => 3*10 + 3*8 + 2*16 + 3);
 
 my $arg = $ENV{PATH}; # a tainted value
-- 
2.5.0

