From fbd1ddb288d0a7503d9ec6c7f472d9468c8085a4 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 314/398] B::C patch for op/ver.t

---
 t/CORE/op/ver.t | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/t/CORE/op/ver.t b/t/CORE/op/ver.t
index 3969d11..16958c9 100644
--- a/t/CORE/op/ver.t
+++ b/t/CORE/op/ver.t
@@ -1,8 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    push @INC, qw{t/CORE/lib};
     $SIG{'__WARN__'} = sub { warn $_[0] if $DOWARN };
     require "test.pl";
 }
-- 
2.5.0

