From ad44f7c4e08fa3c927238d3337447be355f32364 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 315/398] B::C patch for op/waitpid.t

---
 t/CORE/op/waitpid.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/waitpid.t b/t/CORE/op/waitpid.t
index aff2b99..c4c2c5c 100644
--- a/t/CORE/op/waitpid.t
+++ b/t/CORE/op/waitpid.t
@@ -3,9 +3,8 @@
 # tests for (possibly emulated) waitpid
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     require Config;
     skip_all('no Errno')
 	unless eval 'use Errno qw(EINVAL); 1';
-- 
2.5.0

