From bb9ce56db2d6c3e6cdb3dcfd644061e6185219f6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 316/398] B::C patch for op/wantarray.t

---
 t/CORE/op/wantarray.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/wantarray.t b/t/CORE/op/wantarray.t
index 2ee4243..df79821 100644
--- a/t/CORE/op/wantarray.t
+++ b/t/CORE/op/wantarray.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

