From 2883bde1b4f10dfbe519bed62529ad075164afe0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:02 -0500
Subject: [PATCH 318/398] B::C patch for op/while.t

---
 t/CORE/op/while.t | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/t/CORE/op/while.t b/t/CORE/op/while.t
index 1e8eb44..5968503 100644
--- a/t/CORE/op/while.t
+++ b/t/CORE/op/while.t
@@ -1,10 +1,6 @@
 #!./perl
 
-BEGIN {
-    chdir 't';
-    @INC = "../lib";
-    require "./test.pl";
-}
+require 't/CORE/test.pl';
 
 plan(26);
 
@@ -215,9 +211,9 @@ sub save_context { $_[0] = wantarray; $_[1] }
     ok($a[0] ne $a[1]);
 }
 
-fresh_perl_is <<'72406', "foobar\n", {},
+fresh_perl_is(<<'72406', "foobar\n", {}, "[perl #72406] segv with do{}until CONST where const is not folded");
 { package o; use overload bool => sub { die unless $::ok++; return 1 } }
 use constant OK => bless [], o::;
 do{print("foobar\n");}until OK;
 72406
-    "[perl #72406] segv with do{}until CONST where const is not folded";
+    
-- 
2.5.0

