From 35dc2ae7e7864cf195b7daf418e8c81d15dc0498 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 319/398] B::C patch for op/write.t

---
 t/CORE/op/write.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/write.t b/t/CORE/op/write.t
index 7f0e881..74381ae 100644
--- a/t/CORE/op/write.t
+++ b/t/CORE/op/write.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;	# Amazed that this hackery can be made strict ...
-- 
2.5.0

