From dd363f8621c0f9ce4431234b423edce803f7a929 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 320/398] B::C patch for op/yadayada.t

---
 t/CORE/op/yadayada.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/op/yadayada.t b/t/CORE/op/yadayada.t
index a213bec..92984e8 100644
--- a/t/CORE/op/yadayada.t
+++ b/t/CORE/op/yadayada.t
@@ -1,9 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

