From 00548dbe27f1dad4507d454330ccea02c2be7dae Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 321/398] B::C patch for re/charset.t

---
 t/CORE/re/charset.t | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/t/CORE/re/charset.t b/t/CORE/re/charset.t
index ad2e952..302887c 100644
--- a/t/CORE/re/charset.t
+++ b/t/CORE/re/charset.t
@@ -1,10 +1,8 @@
 # Test the /a, /d, etc regex modifiers
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
-    require './loc_tools.pl';
+    require 't/CORE/test.pl';
+    require 't/CORE/loc_tools.pl';
 }
 
 use strict;
-- 
2.5.0

