From 0192eb9aff74a27aff2c4b8d1847f4c539445ddb Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 322/398] B::C patch for re/fold_grind.t

---
 t/CORE/re/fold_grind.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/re/fold_grind.t b/t/CORE/re/fold_grind.t
index 7437295..824a614 100644
--- a/t/CORE/re/fold_grind.t
+++ b/t/CORE/re/fold_grind.t
@@ -3,15 +3,15 @@
 binmode STDOUT, ":utf8";
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     require Config; import Config;
     skip_all_if_miniperl("no dynamic loading on miniperl, no Encode nor POSIX");
     if ($^O eq 'dec_osf') {
       skip_all("$^O cannot handle this test");
     }
-    require './loc_tools.pl';
+    require 't/CORE/loc_tools.pl';
 }
 
 use charnames ":full";
-- 
2.5.0

