From b5006643dc58987b42e19ffc3f74d99d5509281f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 323/398] B::C patch for re/no_utf8_pm.t

---
 t/CORE/re/no_utf8_pm.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/no_utf8_pm.t b/t/CORE/re/no_utf8_pm.t
index 6595a24..a443643 100644
--- a/t/CORE/re/no_utf8_pm.t
+++ b/t/CORE/re/no_utf8_pm.t
@@ -1,9 +1,9 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 1;
-- 
2.5.0

