From 3e04b0418d8c51fae923ed79e1f7905cc0b7f4f8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 324/398] B::C patch for re/overload.t

---
 t/CORE/re/overload.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/overload.t b/t/CORE/re/overload.t
index dba0357..951cf1a 100644
--- a/t/CORE/re/overload.t
+++ b/t/CORE/re/overload.t
@@ -1,9 +1,9 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

