From 070f3480a460797cae15094b769fc2aec8478854 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 325/398] B::C patch for re/pat.t

---
 t/CORE/re/pat.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/pat.t b/t/CORE/re/pat.t
index 7965f4e..4694772 100644
--- a/t/CORE/re/pat.t
+++ b/t/CORE/re/pat.t
@@ -14,10 +14,10 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
+    
+    unshift @INC, 't/CORE/lib';
     require Config; import Config;
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan tests => 726;  # Update this when adding/deleting tests.
-- 
2.5.0

