From 7a2f8856760f3797fe6df129b7c2502a5373eaa6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 327/398] B::C patch for re/pat_psycho.t

---
 t/CORE/re/pat_psycho.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/pat_psycho.t b/t/CORE/re/pat_psycho.t
index fada302..8f58405 100644
--- a/t/CORE/re/pat_psycho.t
+++ b/t/CORE/re/pat_psycho.t
@@ -18,9 +18,9 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
     if ($^O eq 'dec_osf') {
         skip_all("$^O cannot handle this test");
     }
-- 
2.5.0

