From ee40249718c2fa69f086e6d7c57be95766c13613 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 329/398] B::C patch for re/pat_rt_report.t

---
 t/CORE/re/pat_rt_report.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/pat_rt_report.t b/t/CORE/re/pat_rt_report.t
index 94100d1..9ea1e09 100644
--- a/t/CORE/re/pat_rt_report.t
+++ b/t/CORE/re/pat_rt_report.t
@@ -15,9 +15,9 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("miniperl can't load Tie::Hash::NamedCapture, need for %+ and %-");
 }
 
-- 
2.5.0

