From 534e43a2c15b08070b90585031dadb298e83c3d3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:03 -0500
Subject: [PATCH 330/398] B::C patch for re/pat_special_cc.t

---
 t/CORE/re/pat_special_cc.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/pat_special_cc.t b/t/CORE/re/pat_special_cc.t
index 9f20449..102b607 100644
--- a/t/CORE/re/pat_special_cc.t
+++ b/t/CORE/re/pat_special_cc.t
@@ -15,9 +15,9 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 
-- 
2.5.0

