From c0bb1364718713f336931a23a0b4f1c19a397c8e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 334/398] B::C patch for re/qr_gc.t

---
 t/CORE/re/qr_gc.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/qr_gc.t b/t/CORE/re/qr_gc.t
index ca82f42..6e6aeab 100644
--- a/t/CORE/re/qr_gc.t
+++ b/t/CORE/re/qr_gc.t
@@ -1,9 +1,9 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     undef &Regexp::DESTROY;
 }
 
-- 
2.5.0

