From 5c88ba2f396a775e362ae6fd91e9e44d118e0bfe Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 335/398] B::C patch for re/qrstack.t

---
 t/CORE/re/qrstack.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/qrstack.t b/t/CORE/re/qrstack.t
index 47d190d..06afb74 100644
--- a/t/CORE/re/qrstack.t
+++ b/t/CORE/re/qrstack.t
@@ -1,9 +1,9 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 plan tests => 1;
-- 
2.5.0

