From 682d161123836de7f6cebfc90e93a9050eeac768 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 336/398] B::C patch for re/recompile.t

---
 t/CORE/re/recompile.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/recompile.t b/t/CORE/re/recompile.t
index 2ae310d..dea27c7 100644
--- a/t/CORE/re/recompile.t
+++ b/t/CORE/re/recompile.t
@@ -15,9 +15,9 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no re");
 }
 
-- 
2.5.0

