From 6b8f42bb8b061b029775b9cd6a3aa127b35ae21e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 337/398] B::C patch for re/reg_60508.t

---
 t/CORE/re/reg_60508.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_60508.t b/t/CORE/re/reg_60508.t
index e370ca0..2e57786 100644
--- a/t/CORE/re/reg_60508.t
+++ b/t/CORE/re/reg_60508.t
@@ -5,9 +5,9 @@
 # --Schwern
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

