From a7ed758a27b6ea97dbd89af0be3111cf97125cf7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 338/398] B::C patch for re/reg_email.t

---
 t/CORE/re/reg_email.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_email.t b/t/CORE/re/reg_email.t
index a498585..9520464 100644
--- a/t/CORE/re/reg_email.t
+++ b/t/CORE/re/reg_email.t
@@ -4,9 +4,9 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

