From 5ee49eee39501236fb0a84208d8e09eb9ebfc139 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:04 -0500
Subject: [PATCH 341/398] B::C patch for re/reg_fold.t

---
 t/CORE/re/reg_fold.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_fold.t b/t/CORE/re/reg_fold.t
index 9e97ddd..176d567 100644
--- a/t/CORE/re/reg_fold.t
+++ b/t/CORE/re/reg_fold.t
@@ -1,9 +1,9 @@
 #!perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec");
 }
 
-- 
2.5.0

