From 4d7fd4943682e625aa34b46df573244b97a047fc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 342/398] B::C patch for re/reg_mesg.t

---
 t/CORE/re/reg_mesg.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_mesg.t b/t/CORE/re/reg_mesg.t
index 55eda18..f246bd5 100644
--- a/t/CORE/re/reg_mesg.t
+++ b/t/CORE/re/reg_mesg.t
@@ -1,9 +1,9 @@
 #!./perl -w
 
 BEGIN {
-	chdir 't' if -d 't';
-	@INC = '../lib';
-	require './test.pl';
+	
+	push @INC, "t/CORE/lib";
+	require 't/CORE/test.pl';
 	eval 'require Config'; # assume defaults if this fails
 }
 
-- 
2.5.0

