From 0756b9e85cd23452f4ff1e705e2e030e776bd302 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 343/398] B::C patch for re/reg_namedcapture.t

---
 t/CORE/re/reg_namedcapture.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/re/reg_namedcapture.t b/t/CORE/re/reg_namedcapture.t
index 03034b0..fc02c62 100644
--- a/t/CORE/re/reg_namedcapture.t
+++ b/t/CORE/re/reg_namedcapture.t
@@ -1,8 +1,8 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    
+    push @INC, "t/CORE/lib";
     unless (defined &DynaLoader::boot_DynaLoader) {
       print "1..0 # Skip: no dynamic loading on miniperl, no Tie::Hash::NamedCapture\n";
       exit 0;
-- 
2.5.0

