From cdc119a2a66b94e2f1d38261808ce8e513ebb1a1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 344/398] B::C patch for re/reg_nc_tie.t

---
 t/CORE/re/reg_nc_tie.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_nc_tie.t b/t/CORE/re/reg_nc_tie.t
index 3380b25..1a9c105 100644
--- a/t/CORE/re/reg_nc_tie.t
+++ b/t/CORE/re/reg_nc_tie.t
@@ -1,9 +1,9 @@
 #!./perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no Tie::Hash::NamedCapture");
 }
 
-- 
2.5.0

