From fc223dd5d363e6ec8e538e75f0908869bf4a9d32 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 346/398] B::C patch for re/reg_posixcc.t

---
 t/CORE/re/reg_posixcc.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/re/reg_posixcc.t b/t/CORE/re/reg_posixcc.t
index 7473eea..f12d97d 100644
--- a/t/CORE/re/reg_posixcc.t
+++ b/t/CORE/re/reg_posixcc.t
@@ -1,9 +1,7 @@
 #!perl
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use strict;
-- 
2.5.0

