From 55a69e3e948e2475834e920218a1f7a5aa5ff1cf Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 347/398] B::C patch for re/regex_sets.t

---
 t/CORE/re/regex_sets.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/regex_sets.t b/t/CORE/re/regex_sets.t
index 9ddceae..b41bfba 100644
--- a/t/CORE/re/regex_sets.t
+++ b/t/CORE/re/regex_sets.t
@@ -11,9 +11,9 @@ use warnings;
 $| = 1;
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.');
-    require './test.pl';
+    
+    unshift @INC, 't/CORE/lib';
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

