From 7b813eb58fa5ed30365af944f771aab286317c60 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 349/398] B::C patch for re/regexp.t

---
 t/CORE/re/regexp.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/regexp.t b/t/CORE/re/regexp.t
index ac211ca..8bd79e9 100644
--- a/t/CORE/re/regexp.t
+++ b/t/CORE/re/regexp.t
@@ -58,8 +58,8 @@ BEGIN {
 	open TESTS, $file or die "Can't open $file";
     }
 
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    
+    push @INC, "t/CORE/lib";
 
 }
 
@@ -76,7 +76,7 @@ use vars qw($qr $skip_amp $qr_embed $qr_embed_thr $regex_sets); # set by our cal
 
 
 if (!defined $file) {
-    open TESTS, 're/re_tests' or die "Can't open re/re_tests: $!";
+    open TESTS, 't/CORE/re/re_tests' or die "Can't open re/re_tests: $!";
 }
 
 my @tests = <TESTS>;
-- 
2.5.0

