From fb8389bc0cba519304bfc8dd7e40dbeee0d6228f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 351/398] B::C patch for re/regexp_notrie.t

---
 t/CORE/re/regexp_notrie.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/regexp_notrie.t b/t/CORE/re/regexp_notrie.t
index 9ace6ed..d1a80cc 100644
--- a/t/CORE/re/regexp_notrie.t
+++ b/t/CORE/re/regexp_notrie.t
@@ -6,7 +6,7 @@ BEGIN {
 }
 
 $qr = 1;
-for $file ('./re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
+for $file ('t/CORE/re/regexp.t', './re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
     if (-r $file) {
 	do $file or die $@;
 	exit;
-- 
2.5.0

