From 8877a4fdfaf535843786ef5499d45757b8846d9c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 352/398] B::C patch for re/regexp_qr.t

---
 t/CORE/re/regexp_qr.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/regexp_qr.t b/t/CORE/re/regexp_qr.t
index 18b1cf8..a2863eb 100644
--- a/t/CORE/re/regexp_qr.t
+++ b/t/CORE/re/regexp_qr.t
@@ -1,7 +1,7 @@
 #!./perl
 
 $qr = 1;
-for $file ('./re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
+for $file ('t/CORE/re/regexp.t', './re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
     if (-r $file) {
 	do $file or die $@;
 	exit;
-- 
2.5.0

