From 06b46a03abbdaae37f43ad1292fde0ab747b0b0d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:05 -0500
Subject: [PATCH 353/398] B::C patch for re/regexp_qr_embed.t

---
 t/CORE/re/regexp_qr_embed.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/regexp_qr_embed.t b/t/CORE/re/regexp_qr_embed.t
index f0366ca..d641baa 100644
--- a/t/CORE/re/regexp_qr_embed.t
+++ b/t/CORE/re/regexp_qr_embed.t
@@ -2,7 +2,7 @@
 
 $qr = 1;
 $qr_embed = 1;
-for $file ('./re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
+for $file ('t/CORE/re/regexp.t', './re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
     if (-r $file) {
 	do $file or die $@;
 	exit;
-- 
2.5.0

