From 2bac8611a545c908ec3afa62c3662a356d667153 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 354/398] B::C patch for re/regexp_trielist.t

---
 t/CORE/re/regexp_trielist.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/regexp_trielist.t b/t/CORE/re/regexp_trielist.t
index 55379e8..bbdeb1b 100644
--- a/t/CORE/re/regexp_trielist.t
+++ b/t/CORE/re/regexp_trielist.t
@@ -6,7 +6,7 @@ BEGIN {
       }
 
 $qr = 1;
-for $file ('./re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
+for $file ('t/CORE/re/regexp.t', './re/regexp.t', './t/re/regexp.t', ':re:regexp.t') {
     if (-r $file) {
 	do $file or die $@;
 	exit;
-- 
2.5.0

