From a5f52de92fa11ca7274d19076a27c28ebb8e1353 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 355/398] B::C patch for re/regexp_unicode_prop.t

---
 t/CORE/re/regexp_unicode_prop.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/regexp_unicode_prop.t b/t/CORE/re/regexp_unicode_prop.t
index fb1b154..8bc4d7d 100644
--- a/t/CORE/re/regexp_unicode_prop.t
+++ b/t/CORE/re/regexp_unicode_prop.t
@@ -9,7 +9,7 @@ use warnings;
 use 5.010;
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no File::Spec (used by charnames)");
 }
 
-- 
2.5.0

