From 08873d69d3a24ccc9c6555932c4fa193d461880c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 356/398] B::C patch for re/rt122747.t

---
 t/CORE/re/rt122747.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/re/rt122747.t b/t/CORE/re/rt122747.t
index b839edd..958fa2c 100644
--- a/t/CORE/re/rt122747.t
+++ b/t/CORE/re/rt122747.t
@@ -6,9 +6,8 @@ $| = 1;
 
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = ('../lib','.','../ext/re');
-    require './test.pl';
+    unshift @INC, 't/CORE/lib', 't/CORE/ext/re';
+    require 't/CORE/test.pl';
 }
 
 plan tests => 3;
-- 
2.5.0

