From 61762f6304400b5c822a879df8875dc72b1cd662 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 359/398] B::C patch for re/substT.t

---
 t/CORE/re/substT.t | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/t/CORE/re/substT.t b/t/CORE/re/substT.t
index 22c3c50..c5f25ec 100644
--- a/t/CORE/re/substT.t
+++ b/t/CORE/re/substT.t
@@ -1,8 +1,11 @@
 #!perl -wT
 
-for $file ('re/subst.t', 't/re/subst.t', ':re:subst.t') {
+require './t/CORE/test.pl';
+
+for $file ( 't/CORE/re/subst.t', 're/subst.t', 't/re/subst.t', ':re:subst.t') {
   if (-r $file) {
-    do "./$file";
+    my ($tf) = $file =~ qr{(.*)};
+    require "./$tf";
     exit;
   }
 }
-- 
2.5.0

