From 865975b3c8277b9d301912ce2a4bae1a57225cec Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 361/398] B::C patch for re/subst_wamp.t

---
 t/CORE/re/subst_wamp.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/re/subst_wamp.t b/t/CORE/re/subst_wamp.t
index 286141c..03c8401 100644
--- a/t/CORE/re/subst_wamp.t
+++ b/t/CORE/re/subst_wamp.t
@@ -1,7 +1,7 @@
 #!./perl
 
 $dummy = defined $&;		# Now we have it...
-for $file ('re/subst.t', 't/re/subst.t', ':re:subst.t') {
+for $file ( 't/CORE/re/subst.t', 're/subst.t', 't/re/subst.t', ':re:subst.t') {
   if (-r $file) {
     do "./$file";
     exit;
-- 
2.5.0

