From eff808a64cc7efac2fa3cf9e68fbc5f790e3af75 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 362/398] B::C patch for re/uniprops.t

---
 t/CORE/re/uniprops.t | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/t/CORE/re/uniprops.t b/t/CORE/re/uniprops.t
index 2f1cb44..0eeda0f 100644
--- a/t/CORE/re/uniprops.t
+++ b/t/CORE/re/uniprops.t
@@ -2,7 +2,8 @@ use strict;
 use warnings;
 no warnings 'once';
 
-if ($^O eq 'dec_osf') {
+# disabled by cPanel
+if (1 || $^O eq 'dec_osf') {
    print "1..0 # $^O cannot handle this test\n";
    exit(0);
 }
-- 
2.5.0

