From 526938163d7b58c0902d7af5dccac77909e87e7e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 363/398] B::C patch for uni/attrs.t

---
 t/CORE/uni/attrs.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/attrs.t b/t/CORE/uni/attrs.t
index 3ea2f68..2263063 100644
--- a/t/CORE/uni/attrs.t
+++ b/t/CORE/uni/attrs.t
@@ -3,9 +3,9 @@
 # Regression tests for attributes.pm and the C< : attrs> syntax.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
     skip_all_if_miniperl("miniperl can't load attributes");
 }
 
-- 
2.5.0

