From fa6f142e7edf1d6ca8b760b98bb1bdfa5bc9e841 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:06 -0500
Subject: [PATCH 365/398] B::C patch for uni/cache.t

---
 t/CORE/uni/cache.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/uni/cache.t b/t/CORE/uni/cache.t
index 7b6e31e..7788aa9 100644
--- a/t/CORE/uni/cache.t
+++ b/t/CORE/uni/cache.t
@@ -1,6 +1,6 @@
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib .);
+    
+    unshift @INC, 't/CORE/lib';
     require "test.pl";
 }
 
-- 
2.5.0

