From 30b5647d670d269ae2b743ce10f798c1834fc95d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 367/398] B::C patch for uni/case.pl

---
 t/CORE/uni/case.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/case.pl b/t/CORE/uni/case.pl
index 08df670..afe1493 100644
--- a/t/CORE/uni/case.pl
+++ b/t/CORE/uni/case.pl
@@ -1,4 +1,4 @@
-require "test.pl";
+require "t/CORE/test.pl";
 use strict;
 use warnings;
 
-- 
2.5.0

