From b757cc82c60c2aea847de291359102a4a295ac50 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 369/398] B::C patch for uni/chr.t

---
 t/CORE/uni/chr.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/chr.t b/t/CORE/uni/chr.t
index 9445d32..d79b18a 100644
--- a/t/CORE/uni/chr.t
+++ b/t/CORE/uni/chr.t
@@ -1,7 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

