From 33fa5a4aa9dbb252db8e4fe62524a574e4c52bef Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 370/398] B::C patch for uni/class.t

---
 t/CORE/uni/class.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/uni/class.t b/t/CORE/uni/class.t
index 144ae43..d6f3573 100644
--- a/t/CORE/uni/class.t
+++ b/t/CORE/uni/class.t
@@ -1,6 +1,6 @@
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib .);
+    
+    unshift @INC, 't/CORE/lib';
     require "test.pl";
 }
 
-- 
2.5.0

