From a4554ef12ea0e1913979b7e36f24d083dff7cef3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 372/398] B::C patch for uni/fold.t

---
 t/CORE/uni/fold.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/CORE/uni/fold.t b/t/CORE/uni/fold.t
index 94090e9..634f481 100644
--- a/t/CORE/uni/fold.t
+++ b/t/CORE/uni/fold.t
@@ -5,11 +5,11 @@ use warnings;
 # This file also tests the fc() keyword.
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    
+    push @INC, "t/CORE/lib";
     require Config; import Config;
-    require './test.pl';
-    require './loc_tools.pl';   # Contains find_utf8_ctype_locale()
+    require 't/CORE/test.pl';
+    require 't/CORE/loc_tools.pl';   # Contains find_utf8_ctype_locale()
 }
 
 use feature 'unicode_strings';
-- 
2.5.0

