From fa11eb7ccff2291204aa26d32500cb4258ec3f2a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 373/398] B::C patch for uni/goto.t

---
 t/CORE/uni/goto.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/goto.t b/t/CORE/uni/goto.t
index 922ddc1..f11a809 100644
--- a/t/CORE/uni/goto.t
+++ b/t/CORE/uni/goto.t
@@ -1,7 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan tests => 4;
-- 
2.5.0

