From cc32c21799dc21f5987ac758395990b4cb8d7fca Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 374/398] B::C patch for uni/greek.t

---
 t/CORE/uni/greek.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/greek.t b/t/CORE/uni/greek.t
index 5326ab9..cb6906c 100644
--- a/t/CORE/uni/greek.t
+++ b/t/CORE/uni/greek.t
@@ -1,7 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

