From c7eef214fe9c19266f6c791247ea6194105d927d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 375/398] B::C patch for uni/gv.t

---
 t/CORE/uni/gv.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/gv.t b/t/CORE/uni/gv.t
index 90a6332..b3f8de8 100644
--- a/t/CORE/uni/gv.t
+++ b/t/CORE/uni/gv.t
@@ -5,9 +5,9 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

