From 4fee6050cfcc847d402837c3ba47fad0fdf1d6cc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 376/398] B::C patch for uni/labels.t

---
 t/CORE/uni/labels.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/labels.t b/t/CORE/uni/labels.t
index 3fa9d38..5d3d427 100644
--- a/t/CORE/uni/labels.t
+++ b/t/CORE/uni/labels.t
@@ -3,9 +3,9 @@
 # Tests for labels in UTF-8
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

