From 6c6ba28a46a7e18a4bf32c0c67a42ba8868a0386 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:07 -0500
Subject: [PATCH 377/398] B::C patch for uni/latin2.t

---
 t/CORE/uni/latin2.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/latin2.t b/t/CORE/uni/latin2.t
index 6e7d980..63c2c2c 100644
--- a/t/CORE/uni/latin2.t
+++ b/t/CORE/uni/latin2.t
@@ -1,7 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

