From 129aba77375e55a6bffae80c85f1a6f4efd93208 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 379/398] B::C patch for uni/lower.t

---
 t/CORE/uni/lower.t | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/lower.t b/t/CORE/uni/lower.t
index 5b706af..2f0c236 100644
--- a/t/CORE/uni/lower.t
+++ b/t/CORE/uni/lower.t
@@ -1,7 +1,6 @@
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib uni .);
-    require "case.pl";
+    
+    require 't/CORE/uni/case.pl';
 }
 
 casetest(0, # No extra tests run here,
-- 
2.5.0

