From 4417be73db1c76e4bc35a899b2d6761f202da5ef Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 380/398] B::C patch for uni/method.t

---
 t/CORE/uni/method.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/uni/method.t b/t/CORE/uni/method.t
index 4f9d72d..3882918 100644
--- a/t/CORE/uni/method.t
+++ b/t/CORE/uni/method.t
@@ -5,8 +5,8 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    
+    push @INC, qw{t/CORE/lib};
     require "test.pl";
 }
 
-- 
2.5.0

