From 357dff158d47628a1a1ba52e470e9cd7a6a4ad50 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 381/398] B::C patch for uni/opcroak.t

---
 t/CORE/uni/opcroak.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/opcroak.t b/t/CORE/uni/opcroak.t
index 29909d7..c9ba6b6 100644
--- a/t/CORE/uni/opcroak.t
+++ b/t/CORE/uni/opcroak.t
@@ -5,9 +5,9 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

