From 9a79ff1108d0ce75cf78c4781f704099f0a40d67 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 382/398] B::C patch for uni/overload.t

---
 t/CORE/uni/overload.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/overload.t b/t/CORE/uni/overload.t
index bd87b66..449f35f 100644
--- a/t/CORE/uni/overload.t
+++ b/t/CORE/uni/overload.t
@@ -1,10 +1,8 @@
 #!perl -w
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
     require Config; import Config;
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan(tests => 215);
-- 
2.5.0

