From 3ed6e04beaf8c0c2d9c9709562baf6d16c858649 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 384/398] B::C patch for uni/parser.t

---
 t/CORE/uni/parser.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/parser.t b/t/CORE/uni/parser.t
index b71ca88..9640156 100644
--- a/t/CORE/uni/parser.t
+++ b/t/CORE/uni/parser.t
@@ -4,7 +4,7 @@
 # (including weird syntax errors)
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan (tests => 52);
-- 
2.5.0

