From 359ce4da4b2936137db0440b5f2dc195ee55379b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 385/398] B::C patch for uni/readline.t

---
 t/CORE/uni/readline.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/readline.t b/t/CORE/uni/readline.t
index a83558e..6d8b437 100644
--- a/t/CORE/uni/readline.t
+++ b/t/CORE/uni/readline.t
@@ -1,9 +1,7 @@
 #!./perl
 
 BEGIN {
-    chdir 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan tests => 7;
-- 
2.5.0

