From 2e535d7252f66a15340de69ba0f4bcd1f0d8859f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 386/398] B::C patch for uni/select.t

---
 t/CORE/uni/select.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/select.t b/t/CORE/uni/select.t
index 0e00105..993fda7 100644
--- a/t/CORE/uni/select.t
+++ b/t/CORE/uni/select.t
@@ -5,9 +5,9 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    
+    push @INC, "t/CORE/lib";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

