From 6829c00476c0d98aaf7fdf052abf5284dcfabbe9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 387/398] B::C patch for uni/sprintf.t

---
 t/CORE/uni/sprintf.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/CORE/uni/sprintf.t b/t/CORE/uni/sprintf.t
index 0862649..290c563 100644
--- a/t/CORE/uni/sprintf.t
+++ b/t/CORE/uni/sprintf.t
@@ -1,8 +1,8 @@
 #!./perl -w
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib .);
+    
+    unshift @INC, 't/CORE/lib';
     require "test.pl";
 }
 
-- 
2.5.0

