From 8bb4c83d8cab8becda07d0ec8b8e7199f7c6984c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 388/398] B::C patch for uni/stash.t

---
 t/CORE/uni/stash.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/stash.t b/t/CORE/uni/stash.t
index 7d24e51..006e6a8 100644
--- a/t/CORE/uni/stash.t
+++ b/t/CORE/uni/stash.t
@@ -5,9 +5,7 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

