From 90dad5e10179f15ff9d43b59da990de558bebffa Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:08 -0500
Subject: [PATCH 389/398] B::C patch for uni/tie.t

---
 t/CORE/uni/tie.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/tie.t b/t/CORE/uni/tie.t
index b9da823..7fd35c9 100644
--- a/t/CORE/uni/tie.t
+++ b/t/CORE/uni/tie.t
@@ -1,7 +1,7 @@
 #!perl -w
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 plan (tests => 10);
-- 
2.5.0

