From 302953e61cfaf8d41e8335419555342707968ea2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 390/398] B::C patch for uni/title.t

---
 t/CORE/uni/title.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/title.t b/t/CORE/uni/title.t
index 2d6dcb7..717ec30 100644
--- a/t/CORE/uni/title.t
+++ b/t/CORE/uni/title.t
@@ -1,7 +1,7 @@
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib uni .);
-    require "case.pl";
+    
+    push @INC, qw{t/CORE/lib t/CORE/uni};
+    require 't/CORE/uni/case.pl';
 }
 
 casetest(0, # No extra tests run here,
-- 
2.5.0

