From 20406c63a5e46f071a5026efdbb6434150940d1b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 391/398] B::C patch for uni/tr_7jis.t

---
 t/CORE/uni/tr_7jis.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/tr_7jis.t b/t/CORE/uni/tr_7jis.t
index 2108319..46444da 100644
--- a/t/CORE/uni/tr_7jis.t
+++ b/t/CORE/uni/tr_7jis.t
@@ -5,7 +5,7 @@
 # -- dankogai
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

