From 41b91037fab3bbb9b39ac8fde4c6beda77b546a0 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 392/398] B::C patch for uni/tr_eucjp.t

---
 t/CORE/uni/tr_eucjp.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/tr_eucjp.t b/t/CORE/uni/tr_eucjp.t
index a317bb1..75d55e3 100644
--- a/t/CORE/uni/tr_eucjp.t
+++ b/t/CORE/uni/tr_eucjp.t
@@ -4,7 +4,7 @@
 # -- dankogai
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

