From 9687e5f9dc31790d66215956ecf6fb90aa240f88 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 393/398] B::C patch for uni/tr_sjis.t

---
 t/CORE/uni/tr_sjis.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/tr_sjis.t b/t/CORE/uni/tr_sjis.t
index a82b35c..518c825 100644
--- a/t/CORE/uni/tr_sjis.t
+++ b/t/CORE/uni/tr_sjis.t
@@ -4,7 +4,7 @@
 # -- dankogai
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

