From ee363fe88b63c64b2b3413724399af04c432e363 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 394/398] B::C patch for uni/tr_utf8.t

---
 t/CORE/uni/tr_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/tr_utf8.t b/t/CORE/uni/tr_utf8.t
index 2a566d9..8c15467 100644
--- a/t/CORE/uni/tr_utf8.t
+++ b/t/CORE/uni/tr_utf8.t
@@ -5,7 +5,7 @@
 # -- dankogai
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all_without_dynamic_extension('Encode');
     skip_all("EBCDIC") if $::IS_EBCDIC;
     skip_all_without_perlio();
-- 
2.5.0

