From ccb824b2587b7fdb48d9110f25f78a93da898a90 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 395/398] B::C patch for uni/universal.t

---
 t/CORE/uni/universal.t | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/t/CORE/uni/universal.t b/t/CORE/uni/universal.t
index 626c30f..f326b09 100644
--- a/t/CORE/uni/universal.t
+++ b/t/CORE/uni/universal.t
@@ -4,10 +4,10 @@
 #
 
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    
+    push @INC, "t/CORE/lib";
     $| = 1;
-    require "./test.pl";
+    require 't/CORE/test.pl';
 }
 
 use utf8;
-- 
2.5.0

