From bc42721c2db67eb06a7f5b78584b2ea8f8973fff Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 396/398] B::C patch for uni/upper.t

---
 t/CORE/uni/upper.t | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/t/CORE/uni/upper.t b/t/CORE/uni/upper.t
index 315680c..31e2470 100644
--- a/t/CORE/uni/upper.t
+++ b/t/CORE/uni/upper.t
@@ -1,7 +1,5 @@
 BEGIN {
-    chdir 't' if -d 't';
-    @INC = qw(../lib uni .);
-    require "case.pl";
+    require 't/CORE/uni/case.pl';
 }
 
 is(uc("\x{3B1}\x{345}\x{301}"), "\x{391}\x{301}\x{399}", 'Verify moves YPOGEGRAMMENI');
-- 
2.5.0

