From 9035b4d37d9dd92dfbce3350f2be4cbf18fb1f81 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 397/398] B::C patch for uni/variables.t

---
 t/CORE/uni/variables.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/variables.t b/t/CORE/uni/variables.t
index d802a0a..c865a06 100644
--- a/t/CORE/uni/variables.t
+++ b/t/CORE/uni/variables.t
@@ -4,7 +4,7 @@
 # (including weird syntax errors)
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
 }
 
 use 5.016;
-- 
2.5.0

