From 2478420f7483fc929bec57856b52c88d387253af Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Fri, 25 Sep 2015 12:22:09 -0500
Subject: [PATCH 398/398] B::C patch for uni/write.t

---
 t/CORE/uni/write.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/CORE/uni/write.t b/t/CORE/uni/write.t
index bfc1ddb..851a834 100644
--- a/t/CORE/uni/write.t
+++ b/t/CORE/uni/write.t
@@ -2,7 +2,7 @@
 use strict;
 
 BEGIN {
-    require './test.pl';
+    require 't/CORE/test.pl';
     skip_all("EBCDIC porting needed") if $::IS_EBCDIC;
     skip_all_without_perlio();
 }
-- 
2.5.0

