From 44924cbb27110d40d35eefc3b4386690736cd0c3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:23 -0500
Subject: [PATCH 013/352] B::C patch for io/binmode.t

---
 t/io/binmode.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/binmode.t b/t/io/binmode.t
index 473261e..7bb12ba 100644
--- a/t/io/binmode.t
+++ b/t/io/binmode.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require './test.pl';
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
-- 
2.5.0

