From e2638e1723350824970b5d8fb1a7325e1c37c7f8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:23 -0500
Subject: [PATCH 020/352] B::C patch for io/dup.t

---
 t/io/dup.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/dup.t b/t/io/dup.t
index 1328aeb..bf55dbd 100644
--- a/t/io/dup.t
+++ b/t/io/dup.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require "./test.pl";
 }
 
-- 
2.5.0

