From 50c8ed4ff6f6dd4be7ceadbeeddace947e3e69de Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:24 -0500
Subject: [PATCH 034/352] B::C patch for io/perlio_open.t

---
 t/io/perlio_open.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/perlio_open.t b/t/io/perlio_open.t
index 702c76c..99d7e51 100644
--- a/t/io/perlio_open.t
+++ b/t/io/perlio_open.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     skip_all_without_perlio();
     skip_all_without_dynamic_extension('Fcntl'); # how did you get this far?
 }
-- 
2.5.0

