From 5f7fc48dabeec57886daf16fa5a3769eafce6521 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:24 -0500
Subject: [PATCH 036/352] B::C patch for io/print.t

---
 t/io/print.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/print.t b/t/io/print.t
index 4336090..f8f3646 100644
--- a/t/io/print.t
+++ b/t/io/print.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

