From 1531f239c004f94ce8f9d2f25063d0ebf90c19e8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:24 -0500
Subject: [PATCH 038/352] B::C patch for io/read.t

---
 t/io/read.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/read.t b/t/io/read.t
index 8cc1640..601b9c1 100644
--- a/t/io/read.t
+++ b/t/io/read.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

