From 459bc9475a99c65745aad511a271d8cf83a07dd1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:25 -0500
Subject: [PATCH 046/352] B::C patch for japh/abigail.t

---
 t/japh/abigail.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/japh/abigail.t b/t/japh/abigail.t
index f5115bc..8e979a3 100644
--- a/t/japh/abigail.t
+++ b/t/japh/abigail.t
@@ -25,8 +25,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require "./test.pl";
+    set_up_inc('../lib');
     skip_all('EBCDIC') if $::IS_EBCDIC;
     undef &skip;
 }
-- 
2.5.0

