From 68ba09483e49e97f961c9d37ecb8f91112e0e8ba Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:25 -0500
Subject: [PATCH 049/352] B::C patch for lib/commonsense.t

---
 t/lib/commonsense.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/lib/commonsense.t b/t/lib/commonsense.t
index 9f0f13f..8580341 100644
--- a/t/lib/commonsense.t
+++ b/t/lib/commonsense.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan( tests => 1);
-- 
2.5.0

