From ad9d5d48f4e8a306b67e4279eb9981c59b2ede0e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:25 -0500
Subject: [PATCH 052/352] B::C patch for lib/cygwin.t

---
 t/lib/cygwin.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/lib/cygwin.t b/t/lib/cygwin.t
index 9033d3f..d40c73d 100644
--- a/t/lib/cygwin.t
+++ b/t/lib/cygwin.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = ('../lib');
+    unshift @INC,  ('../lib');
     require './test.pl';
     skip_all('cygwin specific test') unless $^O eq 'cygwin';
 }
-- 
2.5.0

