From b501d6e4f1aca18e64ba4712d2c5bf5d5f008960 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:25 -0500
Subject: [PATCH 065/352] B::C patch for mro/isarev.t

---
 t/mro/isarev.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/mro/isarev.t b/t/mro/isarev.t
index 3bd543d..2681e7f 100644
--- a/t/mro/isarev.t
+++ b/t/mro/isarev.t
@@ -3,9 +3,9 @@
 BEGIN {
     unless (-d 'blib') {
         chdir 't' if -d 't';
-        @INC = '../lib';
     }
     require q(./test.pl);
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

