From 6a05488f80a76403089885f7d2d1c8bb92d10c3c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:25 -0500
Subject: [PATCH 066/352] B::C patch for mro/isarev_utf8.t

---
 t/mro/isarev_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/mro/isarev_utf8.t b/t/mro/isarev_utf8.t
index dff3058..61f7cf3 100644
--- a/t/mro/isarev_utf8.t
+++ b/t/mro/isarev_utf8.t
@@ -3,9 +3,9 @@
 BEGIN {
     unless (-d 'blib') {
         chdir 't' if -d 't';
-        @INC = '../lib';
     }
     require q(./test.pl);
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

