From 92e8766a2de130f38cc3ff91aa794de2d198f2c1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:26 -0500
Subject: [PATCH 086/352] B::C patch for op/array_base.t

---
 t/op/array_base.t | 26 +++++++++++++++-----------
 1 file changed, 15 insertions(+), 11 deletions(-)

diff --git a/t/op/array_base.t b/t/op/array_base.t
index a276240..d566bc4 100644
--- a/t/op/array_base.t
+++ b/t/op/array_base.t
@@ -1,22 +1,26 @@
 #!perl -w
 use strict;
 
-BEGIN {
+my %tests_in_begin_block;
+
+BEGIN { 
+ use v5.15;
+ # Run these at BEGIN time, before arybase loads
+ $tests_in_begin_block{'eval'} = eval('$[ = 1; 123');
+ $tests_in_begin_block{'error'} = $@;
  require './test.pl';
+}
 
- plan (tests => my $tests = 11);
+plan (tests => my $tests = 11);
 
- # Run these at BEGIN time, before arybase loads
- use v5.15;
- is(eval('$[ = 1; 123'), undef);
- like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
+is($tests_in_begin_block{'eval'}, undef);
+like($tests_in_begin_block{'error'}, qr/\AAssigning non-zero to \$\[ is no longer possible/);
 
  if (is_miniperl()) {
    # skip the rest
    SKIP: { skip ("no arybase.xs on miniperl", $tests-2) }
    exit;
  }
-}
 
 no warnings 'deprecated';
 
@@ -24,17 +28,17 @@ is(eval('$['), 0);
 is(eval('$[ = 0; 123'), 123);
 is(eval('$[ = 1; 123'), 123);
 $[ = 1;
-ok $INC{'arybase.pm'};
+ok($INC{'arybase.pm'}, "arybase is in INC");
 
 use v5.15;
 is(eval('$[ = 1; 123'), undef);
 like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
-is $[, 0, '$[ is 0 under 5.16';
+is($[, 0, '$[ is 0 under 5.16');
 $_ = "hello";
 /l/g;
 my $pos = \pos;
-is $$pos, 3;
+is($$pos, 3);
 $$pos = 1;
-is $$pos, 1;
+is($$pos, 1);
 
 1;
-- 
2.5.0

