From 60d193825701886fe98ba9c188454159bb123b95 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:26 -0500
Subject: [PATCH 087/352] B::C patch for op/assignwarn.t

---
 t/op/assignwarn.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/assignwarn.t b/t/op/assignwarn.t
index ac1ad77..8101943 100644
--- a/t/op/assignwarn.t
+++ b/t/op/assignwarn.t
@@ -9,8 +9,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

