From c3315117caa4f04ba41639d1e81884f1e57d7bc3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:26 -0500
Subject: [PATCH 091/352] B::C patch for op/auto.t

---
 t/op/auto.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/auto.t b/t/op/auto.t
index 00f7caa..fb5ad9d 100644
--- a/t/op/auto.t
+++ b/t/op/auto.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require "test.pl";
 }
 
-- 
2.5.0

