From 2fae6ea3611d60b373912dcc1c6ca97182414aaf Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:27 -0500
Subject: [PATCH 092/352] B::C patch for op/avhv.t

---
 t/op/avhv.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/op/avhv.t b/t/op/avhv.t
index d301fad..39a54dc 100644
--- a/t/op/avhv.t
+++ b/t/op/avhv.t
@@ -5,7 +5,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
+    require './test.pl';
+    set_up_inc('../lib');
 }
 
 require Tie::Array;
@@ -20,7 +21,6 @@ sub STORESIZE { $#{$_[0]} = $_[1]+1 }
 
 package main;
 
-require './test.pl';
 plan(tests => 40);
 
 # Helper function to check the typical error message.
-- 
2.5.0

