From 0d86f266c67ca8c02bbf71a1019da6fda041b933 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:27 -0500
Subject: [PATCH 106/352] B::C patch for op/coresubs.t

---
 t/op/coresubs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/coresubs.t b/t/op/coresubs.t
index 2e93c71..b76b42a 100644
--- a/t/op/coresubs.t
+++ b/t/op/coresubs.t
@@ -6,7 +6,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require "test.pl";
     skip_all_without_dynamic_extension('B');
     $^P |= 0x100;
-- 
2.5.0

