From 98018e582c1b9f879c700adcd7a306a6d7c73b62 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:28 -0500
Subject: [PATCH 127/352] B::C patch for op/filetest.t

---
 t/op/filetest.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/filetest.t b/t/op/filetest.t
index 7316442..3c9ee7e 100644
--- a/t/op/filetest.t
+++ b/t/op/filetest.t
@@ -5,8 +5,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan(tests => 53 + 27*14);
-- 
2.5.0

