From 69a412e94c258dd638c1b3ab711a2b634ae75046 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:28 -0500
Subject: [PATCH 129/352] B::C patch for op/filetest_t.t

---
 t/op/filetest_t.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/filetest_t.t b/t/op/filetest_t.t
index eec67ec..a7819ea 100644
--- a/t/op/filetest_t.t
+++ b/t/op/filetest_t.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

