From 641433865f65e0edab14d20defdc5499116b4136 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:28 -0500
Subject: [PATCH 131/352] B::C patch for op/fresh_perl_utf8.t

---
 t/op/fresh_perl_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/fresh_perl_utf8.t b/t/op/fresh_perl_utf8.t
index fcf48e7..b540a80 100644
--- a/t/op/fresh_perl_utf8.t
+++ b/t/op/fresh_perl_utf8.t
@@ -4,8 +4,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan 1;
-- 
2.5.0

