From f1d054569f4b1a5950d294972eae88cebfc6e5da Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:28 -0500
Subject: [PATCH 139/352] B::C patch for op/grep.t

---
 t/op/grep.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/grep.t b/t/op/grep.t
index d34686d..37f594e 100644
--- a/t/op/grep.t
+++ b/t/op/grep.t
@@ -6,7 +6,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require "test.pl";
 }
 
-- 
2.5.0

