From f49e9063285213bbc25dd085d3884e381bc28620 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:29 -0500
Subject: [PATCH 140/352] B::C patch for op/groups.t

---
 t/op/groups.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/groups.t b/t/op/groups.t
index dc2631e..1849977 100644
--- a/t/op/groups.t
+++ b/t/op/groups.t
@@ -11,8 +11,8 @@ BEGIN {
     $ENV{LANGUAGE} = 'C'; # GNU locale extension
 
     chdir 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     skip_all_if_miniperl("no dynamic loading on miniperl, no POSIX");
 }
 use 5.010;
-- 
2.5.0

