From 7207742a2f0ebfe194f8d3c149b51503c53228cc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:29 -0500
Subject: [PATCH 143/352] B::C patch for op/hash.t

---
 t/op/hash.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/hash.t b/t/op/hash.t
index 5f4c143..429eb38 100644
--- a/t/op/hash.t
+++ b/t/op/hash.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

