From d84948cb51ef795629e784a8e36e40e6f1cc9072 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:29 -0500
Subject: [PATCH 154/352] B::C patch for op/kvaslice.t

---
 t/op/kvaslice.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/kvaslice.t b/t/op/kvaslice.t
index e8aa14e..16ee446 100644
--- a/t/op/kvaslice.t
+++ b/t/op/kvaslice.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 # use strict;
-- 
2.5.0

