From c171e301de3ab3914c2a5a050981dc8afcf22f71 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:29 -0500
Subject: [PATCH 159/352] B::C patch for op/lex_assign.t

---
 t/op/lex_assign.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/lex_assign.t b/t/op/lex_assign.t
index 290023c..68ec101 100644
--- a/t/op/lex_assign.t
+++ b/t/op/lex_assign.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 $| = 1;
-- 
2.5.0

