From 1c48a122b1c6e93ea5f642fe78559d20222759ec Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:29 -0500
Subject: [PATCH 162/352] B::C patch for op/list.t

---
 t/op/list.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/list.t b/t/op/list.t
index 74f256b..2596d8a 100644
--- a/t/op/list.t
+++ b/t/op/list.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
 }
 
 require "test.pl";
-- 
2.5.0

