From 6237f5129db87eb1f1d73a6727411bf3130b9ffe Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 166/352] B::C patch for op/loopctl.t

---
 t/op/loopctl.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/loopctl.t b/t/op/loopctl.t
index d520a7f..2443674 100644
--- a/t/op/loopctl.t
+++ b/t/op/loopctl.t
@@ -32,7 +32,7 @@
 #  -- .robin. <robin@kitsite.com>  2001-03-13
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require "test.pl";
 }
 
-- 
2.5.0

