From ba51a0c99d039a27fa949471578a39a6a75ac81c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 167/352] B::C patch for op/lop.t

---
 t/op/lop.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/lop.t b/t/op/lop.t
index bc4eb85..09f40ed 100644
--- a/t/op/lop.t
+++ b/t/op/lop.t
@@ -6,8 +6,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 17;
-- 
2.5.0

