From 8a55a03907045c047eb9b3ab3340e42586f9302c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 169/352] B::C patch for op/magic.t

---
 t/op/magic.t | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/t/op/magic.t b/t/op/magic.t
index e651ad5..11ab90d 100644
--- a/t/op/magic.t
+++ b/t/op/magic.t
@@ -3,17 +3,21 @@
 BEGIN {
     $| = 1;
     chdir 't' if -d 't';
-    @INC = '../lib';
+
+    eval q/use TestInit; 1/;
+
     require './test.pl';
-    plan (tests => 187);
+    set_up_inc('../lib');
 }
 
+plan (tests => 187);
+
 # Test that defined() returns true for magic variables created on the fly,
 # even before they have been created.
 # This must come first, even before turning on warnings or setting up
 # $SIG{__WARN__}, to avoid invalidating the tests.  warnings.pm currently
 # does not mention any special variables, but that could easily change.
-BEGIN {
+{
     # not available in miniperl
     my %non_mini = map { $_ => 1 } qw(+ - [);
     for (qw(
@@ -62,6 +66,8 @@ $PERL =
     $Is_MSWin32 ? '.\perl' :
                   './perl');
 
+$PERL = $^X;
+
 sub env_is {
     my ($key, $val, $desc) = @_;
 
-- 
2.5.0

