From ffc2f2158f72ee1b0db32f7eb5b710935eece418 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 183/352] B::C patch for op/pos.t

---
 t/op/pos.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/pos.t b/t/op/pos.t
index 04b5272..2ccae0d 100644
--- a/t/op/pos.t
+++ b/t/op/pos.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 29;
-- 
2.5.0

