From 365763a9a36312f9bae774529a8b6a4afc440ed2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 184/352] B::C patch for op/postfixderef.t

---
 t/op/postfixderef.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/postfixderef.t b/t/op/postfixderef.t
index e60b33b..9fc9bc1 100644
--- a/t/op/postfixderef.t
+++ b/t/op/postfixderef.t
@@ -10,7 +10,7 @@ this file contains all dereferencing tests from ref.t but using postfix instead
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
     require 'test.pl';
 }
 
-- 
2.5.0

