From 1acaf80c972dca609159ffb9cc04c57625a54983 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 186/352] B::C patch for op/protowarn.t

---
 t/op/protowarn.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/protowarn.t b/t/op/protowarn.t
index 0cf946a..eb0c5e0 100644
--- a/t/op/protowarn.t
+++ b/t/op/protowarn.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
 }
 
 use strict;
-- 
2.5.0

