From a598672594346fe6f0854f269f39bf5ed044dfcf Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 187/352] B::C patch for op/push.t

---
 t/op/push.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/push.t b/t/op/push.t
index b473322..0a36e77 100644
--- a/t/op/push.t
+++ b/t/op/push.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 @tests = split(/\n/, <<EOF);
-- 
2.5.0

