From c667e541da3e01de32129021fea7f78c5af03ab1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:30 -0500
Subject: [PATCH 188/352] B::C patch for op/pwent.t

---
 t/op/pwent.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/pwent.t b/t/op/pwent.t
index 7562bc0..10d0693 100644
--- a/t/op/pwent.t
+++ b/t/op/pwent.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

