From 8688ff5cfcfbd501052d07a296a3dec4b67b33c3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:31 -0500
Subject: [PATCH 200/352] B::C patch for op/reset.t

---
 t/op/reset.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/reset.t b/t/op/reset.t
index e592430..7474da9 100644
--- a/t/op/reset.t
+++ b/t/op/reset.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 use strict;
 
-- 
2.5.0

