From d530cb5157dd10a85990c3c0c3ed3239a2f55aea Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:31 -0500
Subject: [PATCH 201/352] B::C patch for op/reverse.t

---
 t/op/reverse.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/reverse.t b/t/op/reverse.t
index 0796614..059ece2 100644
--- a/t/op/reverse.t
+++ b/t/op/reverse.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 26;
-- 
2.5.0

