From 956986c8eb078722c2b363affac26f864435cbf2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:31 -0500
Subject: [PATCH 203/352] B::C patch for op/setpgrpstack.t

---
 t/op/setpgrpstack.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/setpgrpstack.t b/t/op/setpgrpstack.t
index b384138..588c8b9 100644
--- a/t/op/setpgrpstack.t
+++ b/t/op/setpgrpstack.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     skip_all_without_config('d_setpgrp');
 }
 
-- 
2.5.0

