From ba9b3ae8304a015e8fabd9552337403e2231bac6 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:31 -0500
Subject: [PATCH 207/352] B::C patch for op/smartmatch.t

---
 t/op/smartmatch.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/smartmatch.t b/t/op/smartmatch.t
index ed4b3ec..bbd6372 100644
--- a/t/op/smartmatch.t
+++ b/t/op/smartmatch.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 use strict;
 use warnings;
-- 
2.5.0

