From 509454ebdc6e6ad9fc8b2f16b61a91f51e6818b7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:31 -0500
Subject: [PATCH 214/352] B::C patch for op/srand.t

---
 t/op/srand.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/srand.t b/t/op/srand.t
index 5321cde..e7ce1a7 100644
--- a/t/op/srand.t
+++ b/t/op/srand.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir "t" if -d "t";
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
 }
 
 # Test srand.
-- 
2.5.0

