From 4a6cc71806869237637585433fdf86877e3d3db7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:32 -0500
Subject: [PATCH 218/352] B::C patch for op/state.t

---
 t/op/state.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/state.t b/t/op/state.t
index 81e5486..d7c7a38 100644
--- a/t/op/state.t
+++ b/t/op/state.t
@@ -3,8 +3,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

