From 2a08eede8b1ef5631d9ab576195142657718eb74 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:32 -0500
Subject: [PATCH 221/352] B::C patch for op/sub.t

---
 t/op/sub.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/sub.t b/t/op/sub.t
index 7df8f49..6a6b051 100644
--- a/t/op/sub.t
+++ b/t/op/sub.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan( tests => 33 );
-- 
2.5.0

