From 2a1cb65d0e4a283e2aa48eabbeaa1845d46cb572 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:32 -0500
Subject: [PATCH 225/352] B::C patch for op/switch.t

---
 t/op/switch.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/switch.t b/t/op/switch.t
index 204a57a..bd7b50c 100644
--- a/t/op/switch.t
+++ b/t/op/switch.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

