From 3a95f5555b623aca3999c60a2a27bf4037ccd67f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:32 -0500
Subject: [PATCH 237/352] B::C patch for op/undef.t

---
 t/op/undef.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/undef.t b/t/op/undef.t
index d7d78c0..39d236f 100644
--- a/t/op/undef.t
+++ b/t/op/undef.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

