From 6104cd703c64835a497961b73dad9ae48325bac3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:33 -0500
Subject: [PATCH 249/352] B::C patch for op/wantarray.t

---
 t/op/wantarray.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/wantarray.t b/t/op/wantarray.t
index 2ee4243..cba27e5 100644
--- a/t/op/wantarray.t
+++ b/t/op/wantarray.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

