From bba99d11f70bb3183a1cd3ddd07aeb317d4b1a5e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:34 -0500
Subject: [PATCH 268/352] B::C patch for porting/manifest.t

---
 t/porting/manifest.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/porting/manifest.t b/t/porting/manifest.t
index a88f364..30cee9b 100644
--- a/t/porting/manifest.t
+++ b/t/porting/manifest.t
@@ -26,7 +26,7 @@
 
 use Config;
 BEGIN {
-    @INC = '..' if -f '../TestInit.pm';
+    unshift @INC,  '..' if -f '../TestInit.pm';
 }
 use TestInit qw(T); # T is chdir to the top level
 
-- 
2.5.0

