From 834d2c2250a318bcb42b85dd1ed9735826c4199e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:34 -0500
Subject: [PATCH 271/352] B::C patch for porting/readme.t

---
 t/porting/readme.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/porting/readme.t b/t/porting/readme.t
index cbb2fe8..904e1cf 100644
--- a/t/porting/readme.t
+++ b/t/porting/readme.t
@@ -4,7 +4,7 @@
 # vice versa.
 
 BEGIN {
-    @INC = '..' if -f '../TestInit.pm';
+    unshift @INC,  '..' if -f '../TestInit.pm';
 }
 
 use TestInit qw(T); # T is chdir to the top level
-- 
2.5.0

