From 4d1d18640e9bac03f76ff8d37ba92ef22a4d3c6d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:34 -0500
Subject: [PATCH 273/352] B::C patch for porting/utils.t

---
 t/porting/utils.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/porting/utils.t b/t/porting/utils.t
index 7ff13cf..feb3a08 100644
--- a/t/porting/utils.t
+++ b/t/porting/utils.t
@@ -17,7 +17,7 @@
 # script.
 
 BEGIN {
-    @INC = '..' if -f '../TestInit.pm';
+    unshift @INC,  '..' if -f '../TestInit.pm';
 }
 use TestInit qw(T); # T is chdir to the top level
 use strict;
-- 
2.5.0

