From f28eb866079f0551114b25350d076ac01eea2700 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:34 -0500
Subject: [PATCH 277/352] B::C patch for re/overload.t

---
 t/re/overload.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/overload.t b/t/re/overload.t
index dba0357..840d71a 100644
--- a/t/re/overload.t
+++ b/t/re/overload.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

