From b4e7310922b6fdf572151aae2c7bdb7b6bf57d72 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:35 -0500
Subject: [PATCH 310/352] B::C patch for run/mad.t

---
 t/run/mad.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/run/mad.t b/t/run/mad.t
index 83023c5..bb6bfbe 100644
--- a/t/run/mad.t
+++ b/t/run/mad.t
@@ -6,9 +6,9 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require Config; import Config;
     require './test.pl';
+    set_up_inc('../lib');
     skip_all_without_config('mad');
 }
 
-- 
2.5.0

