From f4bd31c92b1f1bb452184dc88bcd8672a47fb377 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:36 -0500
Subject: [PATCH 318/352] B::C patch for run/switchM.t

---
 t/run/switchM.t | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/t/run/switchM.t b/t/run/switchM.t
index d2b5994..cfa5757 100644
--- a/t/run/switchM.t
+++ b/t/run/switchM.t
@@ -2,15 +2,13 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require Config;
     import Config;
-
+    require './test.pl';
+    set_up_inc('../lib');
 }
 use strict;
 
-require './test.pl';
-
 plan(4);
 
 like(runperl(switches => ['-Irun/flib', '-Mbroken'], stderr => 1),
-- 
2.5.0

