From ee9095af37c99189751fc915ad26ce1b3d5fb078 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:36 -0500
Subject: [PATCH 321/352] B::C patch for run/switchd.t

---
 t/run/switchd.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/run/switchd.t b/t/run/switchd.t
index d24d98b..a12c054 100644
--- a/t/run/switchd.t
+++ b/t/run/switchd.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(../lib lib);
+    unshift @INC,  qw(../lib lib);
 }
 
 BEGIN { require "./test.pl"; }
-- 
2.5.0

