From b9b2097c50e2278071d266979c751d4b7ef3b06a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:36 -0500
Subject: [PATCH 323/352] B::C patch for run/switchn.t

---
 t/run/switchn.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/run/switchn.t b/t/run/switchn.t
index 6ad4a72..91495d3 100644
--- a/t/run/switchn.t
+++ b/t/run/switchn.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib'); 
     *ARGV = *DATA;
     plan(tests => 3);
 }
-- 
2.5.0

