From 790229387431660196b3cb57558328f71838fd95 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:36 -0500
Subject: [PATCH 327/352] B::C patch for uni/bless.t

---
 t/uni/bless.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/uni/bless.t b/t/uni/bless.t
index 5475f3e..fe81485 100644
--- a/t/uni/bless.t
+++ b/t/uni/bless.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use utf8;
-- 
2.5.0

