From 97f4ce1c6940f23aac515bb5ddb731b63fac27a7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:36 -0500
Subject: [PATCH 339/352] B::C patch for uni/readline.t

---
 t/uni/readline.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/uni/readline.t b/t/uni/readline.t
index a83558e..17d9895 100644
--- a/t/uni/readline.t
+++ b/t/uni/readline.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 7;
-- 
2.5.0

