From 286ff7eba69e030006e3e2d8a2bec010eeba6144 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:37 -0500
Subject: [PATCH 341/352] B::C patch for uni/sprintf.t

---
 t/uni/sprintf.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/uni/sprintf.t b/t/uni/sprintf.t
index 0862649..c2c9ca0 100644
--- a/t/uni/sprintf.t
+++ b/t/uni/sprintf.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(../lib .);
+    unshift @INC,  qw(../lib .);
     require "test.pl";
 }
 
-- 
2.5.0

