From 34fa120ad501d6cee5cf4fe117d76588ec6060f7 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Mon, 28 Sep 2015 17:45:37 -0500
Subject: [PATCH 346/352] B::C patch for win32/fs.t

---
 t/win32/fs.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/win32/fs.t b/t/win32/fs.t
index b770311..102599a 100644
--- a/t/win32/fs.t
+++ b/t/win32/fs.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require "./test.pl";
+    set_up_inc('../lib');
     eval 'use Errno';
     die $@ if $@ and !is_miniperl();
 }
-- 
2.5.0

