From 250de30c04f9d7f0234f2e18e60629c9ec0e16de Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:44 -0500
Subject: [PATCH 024/211] B::C patch for io/perlio_fail.t

---
 t/io/perlio_fail.t | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/t/io/perlio_fail.t b/t/io/perlio_fail.t
index 6bc9b17..f7aa709 100644
--- a/t/io/perlio_fail.t
+++ b/t/io/perlio_fail.t
@@ -2,12 +2,13 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require "../t/test.pl";
+    set_up_inc('../lib');
     skip_all_without_perlio();
-    plan (15);
 }
 
+plan (15);
+
 use warnings 'layer';
 my $warn;
 my $file = "fail$$";
-- 
2.5.0

