From 2781cda51b770960f320eaf352ad35434552b04c Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:45 -0500
Subject: [PATCH 046/211] B::C patch for mro/isa_dfs_utf8.t

---
 t/mro/isa_dfs_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/mro/isa_dfs_utf8.t b/t/mro/isa_dfs_utf8.t
index 1c95eaa..6fa0b59 100644
--- a/t/mro/isa_dfs_utf8.t
+++ b/t/mro/isa_dfs_utf8.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require "./test.pl";
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

