From c3e3c2ccc1be7dea1579ce85639f7c5a64322e90 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:45 -0500
Subject: [PATCH 048/211] B::C patch for mro/isarev_utf8.t

---
 t/mro/isarev_utf8.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/mro/isarev_utf8.t b/t/mro/isarev_utf8.t
index dff3058..61f7cf3 100644
--- a/t/mro/isarev_utf8.t
+++ b/t/mro/isarev_utf8.t
@@ -3,9 +3,9 @@
 BEGIN {
     unless (-d 'blib') {
         chdir 't' if -d 't';
-        @INC = '../lib';
     }
     require q(./test.pl);
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

