From 6d5774ab41d783e2a05eb6a513eb46b3d64f19dc Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:45 -0500
Subject: [PATCH 065/211] B::C patch for op/array.t

---
 t/op/array.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/array.t b/t/op/array.t
index 7239d48..e7c48e7 100644
--- a/t/op/array.t
+++ b/t/op/array.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = ('.', '../lib');
     require './test.pl';
+    set_up_inc('.', '../lib');
 }
 
 plan (172);
-- 
2.5.0

