From ee0cc7f4487afea98b33fd99d9b160acd81f7e70 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:45 -0500
Subject: [PATCH 066/211] B::C patch for op/array_base.t

---
 t/op/array_base.t | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/t/op/array_base.t b/t/op/array_base.t
index a30236d..5d5d79d 100644
--- a/t/op/array_base.t
+++ b/t/op/array_base.t
@@ -25,17 +25,17 @@ is(eval('$['), 0);
 is(eval('$[ = 0; 123'), 123);
 is(eval('$[ = 1; 123'), 123);
 $[ = 1;
-ok $INC{'arybase.pm'};
+ok($INC{'arybase.pm'}, "arybase is in INC");
 
 use v5.15;
 is(eval('$[ = 1; 123'), undef);
 like($@, qr/\AAssigning non-zero to \$\[ is no longer possible/);
-is $[, 0, '$[ is 0 under 5.16';
+is($[, 0, '$[ is 0 under 5.16');
 $_ = "hello";
 /l/g;
 my $pos = \pos;
-is $$pos, 3;
+is($$pos, 3);
 $$pos = 1;
-is $$pos, 1;
+is($$pos, 1);
 
 1;
-- 
2.5.0

