From c9c9ad1a2082bf44b5c3dbc14f4581e39bc08523 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 111/211] B::C patch for op/heredoc.t

---
 t/op/heredoc.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/heredoc.t b/t/op/heredoc.t
index a239e92..e9668d0 100644
--- a/t/op/heredoc.t
+++ b/t/op/heredoc.t
@@ -2,8 +2,8 @@
 
 BEGIN {
    chdir 't' if -d 't';
-   @INC = '../lib';
    require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

