From 63dc55409911a7089177047d32d8f916a77cc718 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 117/211] B::C patch for op/kill0.t

---
 t/op/kill0.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/kill0.t b/t/op/kill0.t
index 7f6e6ec..402916e 100644
--- a/t/op/kill0.t
+++ b/t/op/kill0.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 BEGIN {
-- 
2.5.0

