From f6c1eab18b809d1299b1fac060675594b6b73c25 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 127/211] B::C patch for op/mkdir.t

---
 t/op/mkdir.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/mkdir.t b/t/op/mkdir.t
index 2631526..11ff8c0 100644
--- a/t/op/mkdir.t
+++ b/t/op/mkdir.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 22;
-- 
2.5.0

