From 062f34df461f2e4ac841b6b0df87e0465f7202c2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 131/211] B::C patch for op/not.t

---
 t/op/not.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/not.t b/t/op/not.t
index 17f7a36..48a0168 100644
--- a/t/op/not.t
+++ b/t/op/not.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 24;
-- 
2.5.0

