From 61ccd0d9747512dd17e88f12daddacd709bb5c7b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 143/211] B::C patch for op/require_37033.t

---
 t/op/require_37033.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/require_37033.t b/t/op/require_37033.t
index b107636..45f2bfa 100644
--- a/t/op/require_37033.t
+++ b/t/op/require_37033.t
@@ -5,7 +5,7 @@
 
 chdir 't' if -d 't';
 require './test.pl';
-@INC = 'lib';
+unshift @INC,  'lib';
 
 use strict;
 
-- 
2.5.0

