From 44f2b65cddc13ba3760d2893972b6ed7f94fbe89 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 144/211] B::C patch for op/require_errors.t

---
 t/op/require_errors.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/op/require_errors.t b/t/op/require_errors.t
index 3d3d027..6a460e7 100644
--- a/t/op/require_errors.t
+++ b/t/op/require_errors.t
@@ -12,7 +12,7 @@ plan(tests => 17);
 
 my $nonfile = tempfile();
 
-@INC = qw(Perl Rules);
+unshift @INC,  qw(Perl Rules);
 
 # The tests for ' ' and '.h' never did fail, but previously the error reporting
 # code would read memory before the start of the SV's buffer
@@ -127,7 +127,7 @@ like $@, qr/^Can't locate strict\.pm\\0invalid: /, 'do nul check';
 
   $WARN = '';
   local @INC = @INC;
-  unshift @INC, "lib\0invalid";
+  @INC = "lib\0invalid";
   eval { require "unknown.pm" };
   like $WARN, qr{^Invalid \\0 character in \@INC entry for require: lib\\0invalid at }, 'nul warning';
 }
-- 
2.5.0

