From 64279e7a7b4f61f217357c32c441babb6242cbce Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 155/211] B::C patch for op/stat.t

---
 t/op/stat.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/stat.t b/t/op/stat.t
index ad94b3f..bf81a9a 100644
--- a/t/op/stat.t
+++ b/t/op/stat.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';	# for which_perl() etc
+    set_up_inc('../lib');
 }
 
 use Config;
-- 
2.5.0

