From 73bc97180df3a8809a334cafbd111f6e9f642d5f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 160/211] B::C patch for op/switch.t

---
 t/op/switch.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/switch.t b/t/op/switch.t
index 204a57a..bd7b50c 100644
--- a/t/op/switch.t
+++ b/t/op/switch.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

