From 5fc456796c2944e3e098dd9722825bddfcfa980a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 162/211] B::C patch for op/sysio.t

---
 t/op/sysio.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/sysio.t b/t/op/sysio.t
index 25d7197..b95def0 100644
--- a/t/op/sysio.t
+++ b/t/op/sysio.t
@@ -2,8 +2,8 @@
 
 BEGIN {
   chdir 't' if -d 't';
-  @INC = '../lib';
   require './test.pl';
+  set_up_inc('../lib');
 }
 
 plan tests => 48;
-- 
2.5.0

