From c99f7ef6284f901a9daf4c5dc980689eb8be6069 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 174/211] B::C patch for op/wantarray.t

---
 t/op/wantarray.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/wantarray.t b/t/op/wantarray.t
index 433a839..81a2155 100644
--- a/t/op/wantarray.t
+++ b/t/op/wantarray.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

