From 8741b8233745f62e80864bfbfdf94c63b6a72640 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 192/211] B::C patch for re/reg_60508.t

---
 t/re/reg_60508.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/reg_60508.t b/t/re/reg_60508.t
index e370ca0..ed74032 100644
--- a/t/re/reg_60508.t
+++ b/t/re/reg_60508.t
@@ -6,8 +6,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use utf8;
-- 
2.5.0

