From 52c58d7ca387f491d1c6a3d6b758396094aabfd1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 205/211] B::C patch for test.pl

---
 t/test.pl | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/t/test.pl b/t/test.pl
index cda3840..73b1a21 100644
--- a/t/test.pl
+++ b/t/test.pl
@@ -19,6 +19,11 @@
 # In this file, we use the latter "Baby Perl" approach, and increment
 # will be worked over by t/op/inc.t
 
+
+BEGIN {
+    unshift @INC, '..', './lib';
+}
+
 $Level = 1;
 my $test = 1;
 my $planned;
@@ -108,7 +113,7 @@ sub is_miniperl {
 
 sub set_up_inc {
     # Don’t clobber @INC under miniperl
-    @INC = () unless is_miniperl;
+    #@INC = () unless is_miniperl;
     unshift @INC, @_;
 }
 
-- 
2.5.0

