From 1faddce6bbcef5cde006634b4ebf8ca91fe1e6b9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 211/211] B::C patch for uni/select.t

---
 t/uni/select.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/uni/select.t b/t/uni/select.t
index 0e00105..de0cb5e 100644
--- a/t/uni/select.t
+++ b/t/uni/select.t
@@ -6,8 +6,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use utf8;
-- 
2.5.0

