From 524af7d25e153744824d1501b3a1a5c1cbd876ff Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:44 -0500
Subject: [PATCH 021/211] B::C patch for io/open.t

---
 t/io/open.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/open.t b/t/io/open.t
index 3817bdd..bd94791 100644
--- a/t/io/open.t
+++ b/t/io/open.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 $|  = 1;
-- 
2.5.0

