From 0c78aea6af253a9ace034a5353d51cf707f01480 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:44 -0500
Subject: [PATCH 022/211] B::C patch for io/openpid.t

---
 t/io/openpid.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/io/openpid.t b/t/io/openpid.t
index 946fa5e..d3fcf78 100644
--- a/t/io/openpid.t
+++ b/t/io/openpid.t
@@ -9,8 +9,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 if ($^O eq 'dos') {
-- 
2.5.0

