From 275c6d5bede95b59a44faca74295107254ec1da3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:44 -0500
Subject: [PATCH 023/211] B::C patch for io/perlio.t

---
 t/io/perlio.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/io/perlio.t b/t/io/perlio.t
index 6b03e19..91652ab 100644
--- a/t/io/perlio.t
+++ b/t/io/perlio.t
@@ -1,8 +1,8 @@
 BEGIN {
 	chdir 't' if -d 't';
-	@INC = '../lib';
 	require Config; import Config;
-	require './test.pl';
+    require './test.pl';
+    set_up_inc('../lib');
 	skip_all_without_perlio();
 }
 
-- 
2.5.0

