From 2c3939de49818f68e7f3da961d18ef8d984c6dd8 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:45 -0500
Subject: [PATCH 064/211] B::C patch for op/args.t

---
 t/op/args.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/t/op/args.t b/t/op/args.t
index bfa015e..93b266e 100644
--- a/t/op/args.t
+++ b/t/op/args.t
@@ -2,10 +2,10 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
+    require './test.pl';
+    set_up_inc('../lib');
 }
 
-require './test.pl';
 plan( tests => 23 );
 
 # test various operations on @_
-- 
2.5.0

