From a49309a6da78d54bbcd30fab37c71d0c5ea97bed Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:46 -0500
Subject: [PATCH 092/211] B::C patch for op/exec.t

---
 t/op/exec.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/exec.t b/t/op/exec.t
index 6ec3646..2dc3a07 100644
--- a/t/op/exec.t
+++ b/t/op/exec.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = ('../lib');
+    unshift @INC,  ('../lib');
     require './test.pl';
 }
 
-- 
2.5.0

