From e0e3db143bae9f6f4139112fe6a5eeda800c687a Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 107/211] B::C patch for op/groups.t

---
 t/op/groups.t | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/t/op/groups.t b/t/op/groups.t
index 754690c..552be9e 100644
--- a/t/op/groups.t
+++ b/t/op/groups.t
@@ -10,11 +10,12 @@ BEGIN {
     $ENV{LC_ALL} = "C"; # so that external utilities speak English
     $ENV{LANGUAGE} = 'C'; # GNU locale extension
 
-    chdir 't' if -d 't';
-    @INC = '../lib';
+    chdir 't' if -d 't';    
     require './test.pl';
-    skip_all_if_miniperl("no dynamic loading on miniperl, no POSIX");
+    set_up_inc( '../lib' );    
 }
+skip_all_if_miniperl("no dynamic loading on miniperl, no POSIX");
+
 use 5.010;
 use strict;
 use Config ();
-- 
2.5.0

