From 8f51c5263009e1c27209e92939c7d28c2a82e1e2 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 115/211] B::C patch for op/int.t

---
 t/op/int.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/int.t b/t/op/int.t
index 9aad020..5434ab4 100644
--- a/t/op/int.t
+++ b/t/op/int.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan 15;
-- 
2.5.0

