From f04e34277f7ef638dbbb754735c907076d7109fd Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:47 -0500
Subject: [PATCH 128/211] B::C patch for op/my.t

---
 t/op/my.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/my.t b/t/op/my.t
index 2dca46f..0f4c603 100644
--- a/t/op/my.t
+++ b/t/op/my.t
@@ -1,8 +1,8 @@
 #!./perl
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 sub foo {
-- 
2.5.0

