From af1a262e79348a3a30094480c5710ac57bf4d57f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 136/211] B::C patch for op/protowarn.t

---
 t/op/protowarn.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/protowarn.t b/t/op/protowarn.t
index eb046e5..1572bb6 100644
--- a/t/op/protowarn.t
+++ b/t/op/protowarn.t
@@ -2,7 +2,7 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
 }
 
 use strict;
-- 
2.5.0

