From 545869719feccecf19692cb5712bf8948f6c68b9 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 139/211] B::C patch for op/rand.t

---
 t/op/rand.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/rand.t b/t/op/rand.t
index b3bfdd4..2ba0ba2 100644
--- a/t/op/rand.t
+++ b/t/op/rand.t
@@ -17,7 +17,7 @@
 
 BEGIN {
     chdir "t" if -d "t";
-    @INC = qw(. ../lib);
+    unshift @INC,  qw(. ../lib);
 }
 
 use strict;
-- 
2.5.0

