From 5c68377b1585498f085ec07dd23aeb0efa90967d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 149/211] B::C patch for op/splice.t

---
 t/op/splice.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/splice.t b/t/op/splice.t
index dde11f5..b61e4a0 100644
--- a/t/op/splice.t
+++ b/t/op/splice.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 $|  = 1;
-- 
2.5.0

