From ab4b1496996359e9d50ba31da3cf7d8dc4f28c7d Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:48 -0500
Subject: [PATCH 151/211] B::C patch for op/sprintf2.t

---
 t/op/sprintf2.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/sprintf2.t b/t/op/sprintf2.t
index 1a4dd30..7cdb479 100644
--- a/t/op/sprintf2.t
+++ b/t/op/sprintf2.t
@@ -4,8 +4,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }   
 
 # We'll run 12 extra tests (see below) if $Q is false.
-- 
2.5.0

