From fe6c1917e22eb86a194c26977fc87677c8d12452 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 157/211] B::C patch for op/study.t

---
 t/op/study.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/study.t b/t/op/study.t
index 906aba9..866e7c8 100644
--- a/t/op/study.t
+++ b/t/op/study.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 watchdog(10);
-- 
2.5.0

