From 107f0ef74c2710869e53a619b44ba35de2b1d9f3 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 158/211] B::C patch for op/studytied.t

---
 t/op/studytied.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/studytied.t b/t/op/studytied.t
index 62f1ac9..00cf8b0 100644
--- a/t/op/studytied.t
+++ b/t/op/studytied.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

