From 1ba620f2085890c2cbbc29b13288005b1098459b Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 168/211] B::C patch for op/undef.t

---
 t/op/undef.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/undef.t b/t/op/undef.t
index ddef596..cf47115 100644
--- a/t/op/undef.t
+++ b/t/op/undef.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

