From c95e12420dff4bb5ea1b04db03b621e1102a6722 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 173/211] B::C patch for op/waitpid.t

---
 t/op/waitpid.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/op/waitpid.t b/t/op/waitpid.t
index aff2b99..497fc26 100644
--- a/t/op/waitpid.t
+++ b/t/op/waitpid.t
@@ -4,8 +4,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
     require Config;
     skip_all('no Errno')
 	unless eval 'use Errno qw(EINVAL); 1';
-- 
2.5.0

