From 42976d382e0ee03e0014c4d5fe8e1991c35d95c1 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 179/211] B::C patch for re/no_utf8_pm.t

---
 t/re/no_utf8_pm.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/no_utf8_pm.t b/t/re/no_utf8_pm.t
index 6595a24..15b1cb3 100644
--- a/t/re/no_utf8_pm.t
+++ b/t/re/no_utf8_pm.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 plan tests => 1;
-- 
2.5.0

