From dcaf082515de4b4fd0ce524360fe4180bd9cae7e Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:49 -0500
Subject: [PATCH 180/211] B::C patch for re/overload.t

---
 t/re/overload.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/overload.t b/t/re/overload.t
index ee821f1..a11d4f5 100644
--- a/t/re/overload.t
+++ b/t/re/overload.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = qw '../lib ../ext/re';
     require './test.pl';
+    set_up_inc(qw '../lib ../ext/re');
 }
 
 use strict;
-- 
2.5.0

