From f4eb2344b65732aebd336ead5c9a02ddb06b56bb Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 191/211] B::C patch for re/recompile.t

---
 t/re/recompile.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/recompile.t b/t/re/recompile.t
index 20f5a74..fae69a9 100644
--- a/t/re/recompile.t
+++ b/t/re/recompile.t
@@ -12,7 +12,7 @@ $| = 1;
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = ('../lib','.');
+    unshift @INC,  ('../lib','.');
     require './test.pl';
     skip_all_if_miniperl("no dynamic loading on miniperl, no re");
 }
-- 
2.5.0

