From 07ccdcbce01ccd56cbedc7014eb1fc9e18571137 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 193/211] B::C patch for re/reg_email.t

---
 t/re/reg_email.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/reg_email.t b/t/re/reg_email.t
index 4fc9dfe..7a57b66 100644
--- a/t/re/reg_email.t
+++ b/t/re/reg_email.t
@@ -5,8 +5,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

