From c410f927fa470782619fa29b5168adda3032cc27 Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 198/211] B::C patch for re/reg_nocapture.t

---
 t/re/reg_nocapture.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/reg_nocapture.t b/t/re/reg_nocapture.t
index f354cea..21ffe86 100644
--- a/t/re/reg_nocapture.t
+++ b/t/re/reg_nocapture.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

