From 686adac5c0e85566cc02298ec58fe892f0c5c65f Mon Sep 17 00:00:00 2001
From: Nicolas Rochelemagne <rochelemagne@cpanel.net>
Date: Tue, 29 Sep 2015 14:35:50 -0500
Subject: [PATCH 199/211] B::C patch for re/reg_pmod.t

---
 t/re/reg_pmod.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/re/reg_pmod.t b/t/re/reg_pmod.t
index 1db0bed..a106e96 100644
--- a/t/re/reg_pmod.t
+++ b/t/re/reg_pmod.t
@@ -2,8 +2,8 @@
 
 BEGIN {
     chdir 't' if -d 't';
-    @INC = '../lib';
     require './test.pl';
+    set_up_inc('../lib');
 }
 
 use strict;
-- 
2.5.0

